==5591== Memcheck, a memory error detector
==5591== Copyright (C) 2002-2010, and GNU GPL'd, by Julian Seward et al.
==5591== Using Valgrind-3.6.1 and LibVEX; rerun with -h for copyright info
==5591== Command: python trip_to_the_estimator.py
==5591== 
==5591== Invalid read of size 4
==5591==    at 0x4542F3: PyObject_Free (obmalloc.c:931)
==5591==    by 0x52D63A: code_dealloc (codeobject.c:260)
==5591==    by 0x4BB4BD: load_source_module (import.c:1018)
==5591==    by 0x4BC2D2: import_submodule (import.c:2592)
==5591==    by 0x4BC54E: load_next (import.c:2412)
==5591==    by 0x4BCC10: import_module_level (import.c:2133)
==5591==    by 0x4BD133: PyImport_ImportModuleLevel (import.c:2185)
==5591==    by 0x49EBFA: builtin___import__ (bltinmodule.c:48)
==5591==    by 0x41C9D6: PyObject_Call (abstract.c:2492)
==5591==    by 0x49F7BE: T.420 (ceval.c:3619)
==5591==    by 0x4A2B4B: PyEval_EvalFrameEx (ceval.c:2159)
==5591==    by 0x4A6BD0: PyEval_EvalCodeEx (ceval.c:3000)
==5591==  Address 0x636c020 is 352 bytes inside a block of size 568 free'd
==5591==    at 0x4C283A2: free (vg_replace_malloc.c:366)
==5591==    by 0x5F339EC: fclose@@GLIBC_2.2.5 (iofclose.c:88)
==5591==    by 0x4BB5D8: load_source_module (import.c:994)
==5591==    by 0x4BC2D2: import_submodule (import.c:2592)
==5591==    by 0x4BC54E: load_next (import.c:2412)
==5591==    by 0x4BCC10: import_module_level (import.c:2133)
==5591==    by 0x4BD133: PyImport_ImportModuleLevel (import.c:2185)
==5591==    by 0x49EBFA: builtin___import__ (bltinmodule.c:48)
==5591==    by 0x41C9D6: PyObject_Call (abstract.c:2492)
==5591==    by 0x49F7BE: T.420 (ceval.c:3619)
==5591==    by 0x4A2B4B: PyEval_EvalFrameEx (ceval.c:2159)
==5591==    by 0x4A6BD0: PyEval_EvalCodeEx (ceval.c:3000)
==5591== 
==5591== Invalid read of size 4
==5591==    at 0x4542F3: PyObject_Free (obmalloc.c:931)
==5591==    by 0x467E45: tupledealloc (tupleobject.c:183)
==5591==    by 0x52D6F2: code_dealloc (codeobject.c:261)
==5591==    by 0x4BB4BD: load_source_module (import.c:1018)
==5591==    by 0x4BC2D2: import_submodule (import.c:2592)
==5591==    by 0x4BC54E: load_next (import.c:2412)
==5591==    by 0x4BCC10: import_module_level (import.c:2133)
==5591==    by 0x4BD133: PyImport_ImportModuleLevel (import.c:2185)
==5591==    by 0x49EBFA: builtin___import__ (bltinmodule.c:48)
==5591==    by 0x41C9D6: PyObject_Call (abstract.c:2492)
==5591==    by 0x49F7BE: T.420 (ceval.c:3619)
==5591==    by 0x4A2B4B: PyEval_EvalFrameEx (ceval.c:2159)
==5591==  Address 0x636c020 is 352 bytes inside a block of size 568 free'd
==5591==    at 0x4C283A2: free (vg_replace_malloc.c:366)
==5591==    by 0x5F339EC: fclose@@GLIBC_2.2.5 (iofclose.c:88)
==5591==    by 0x4BB5D8: load_source_module (import.c:994)
==5591==    by 0x4BC2D2: import_submodule (import.c:2592)
==5591==    by 0x4BC54E: load_next (import.c:2412)
==5591==    by 0x4BCC10: import_module_level (import.c:2133)
==5591==    by 0x4BD133: PyImport_ImportModuleLevel (import.c:2185)
==5591==    by 0x49EBFA: builtin___import__ (bltinmodule.c:48)
==5591==    by 0x41C9D6: PyObject_Call (abstract.c:2492)
==5591==    by 0x49F7BE: T.420 (ceval.c:3619)
==5591==    by 0x4A2B4B: PyEval_EvalFrameEx (ceval.c:2159)
==5591==    by 0x4A6BD0: PyEval_EvalCodeEx (ceval.c:3000)
==5591== 
==5591== Invalid read of size 4
==5591==    at 0x4542F3: PyObject_Free (obmalloc.c:931)
==5591==    by 0x467E45: tupledealloc (tupleobject.c:183)
==5591==    by 0x52D6DA: code_dealloc (codeobject.c:262)
==5591==    by 0x4BB4BD: load_source_module (import.c:1018)
==5591==    by 0x4BC2D2: import_submodule (import.c:2592)
==5591==    by 0x4BC54E: load_next (import.c:2412)
==5591==    by 0x4BCC10: import_module_level (import.c:2133)
==5591==    by 0x4BD133: PyImport_ImportModuleLevel (import.c:2185)
==5591==    by 0x49EBFA: builtin___import__ (bltinmodule.c:48)
==5591==    by 0x41C9D6: PyObject_Call (abstract.c:2492)
==5591==    by 0x49F7BE: T.420 (ceval.c:3619)
==5591==    by 0x4A2B4B: PyEval_EvalFrameEx (ceval.c:2159)
==5591==  Address 0x636c020 is 352 bytes inside a block of size 568 free'd
==5591==    at 0x4C283A2: free (vg_replace_malloc.c:366)
==5591==    by 0x5F339EC: fclose@@GLIBC_2.2.5 (iofclose.c:88)
==5591==    by 0x4BB5D8: load_source_module (import.c:994)
==5591==    by 0x4BC2D2: import_submodule (import.c:2592)
==5591==    by 0x4BC54E: load_next (import.c:2412)
==5591==    by 0x4BCC10: import_module_level (import.c:2133)
==5591==    by 0x4BD133: PyImport_ImportModuleLevel (import.c:2185)
==5591==    by 0x49EBFA: builtin___import__ (bltinmodule.c:48)
==5591==    by 0x41C9D6: PyObject_Call (abstract.c:2492)
==5591==    by 0x49F7BE: T.420 (ceval.c:3619)
==5591==    by 0x4A2B4B: PyEval_EvalFrameEx (ceval.c:2159)
==5591==    by 0x4A6BD0: PyEval_EvalCodeEx (ceval.c:3000)
==5591== 
==5591== Invalid read of size 4
==5591==    at 0x4542F3: PyObject_Free (obmalloc.c:931)
==5591==    by 0x52D625: code_dealloc (codeobject.c:270)
==5591==    by 0x4BB4BD: load_source_module (import.c:1018)
==5591==    by 0x4BC2D2: import_submodule (import.c:2592)
==5591==    by 0x4BC54E: load_next (import.c:2412)
==5591==    by 0x4BCC10: import_module_level (import.c:2133)
==5591==    by 0x4BD133: PyImport_ImportModuleLevel (import.c:2185)
==5591==    by 0x49EBFA: builtin___import__ (bltinmodule.c:48)
==5591==    by 0x41C9D6: PyObject_Call (abstract.c:2492)
==5591==    by 0x49F7BE: T.420 (ceval.c:3619)
==5591==    by 0x4A2B4B: PyEval_EvalFrameEx (ceval.c:2159)
==5591==    by 0x4A6BD0: PyEval_EvalCodeEx (ceval.c:3000)
==5591==  Address 0x636d020 is 192 bytes inside a block of size 368 free'd
==5591==    at 0x4C29159: realloc (vg_replace_malloc.c:525)
==5591==    by 0x440C20: PyList_Append (listobject.c:62)
==5591==    by 0x4BEFA5: r_object (marshal.c:731)
==5591==    by 0x4BE6DC: r_object (marshal.c:794)
==5591==    by 0x4BE99B: r_object (marshal.c:930)
==5591==    by 0x4C0D44: PyMarshal_ReadLastObjectFromFile (marshal.c:1107)
==5591==    by 0x4BB5B6: load_source_module (import.c:769)
==5591==    by 0x4BC2D2: import_submodule (import.c:2592)
==5591==    by 0x4BC54E: load_next (import.c:2412)
==5591==    by 0x4BCC10: import_module_level (import.c:2133)
==5591==    by 0x4BD133: PyImport_ImportModuleLevel (import.c:2185)
==5591==    by 0x49EBFA: builtin___import__ (bltinmodule.c:48)
==5591== 
==5591== Conditional jump or move depends on uninitialised value(s)
==5591==    at 0x4542FC: PyObject_Free (obmalloc.c:931)
==5591==    by 0x52D63A: code_dealloc (codeobject.c:260)
==5591==    by 0x4BB4BD: load_source_module (import.c:1018)
==5591==    by 0x4BC2D2: import_submodule (import.c:2592)
==5591==    by 0x4BC54E: load_next (import.c:2412)
==5591==    by 0x4BCC10: import_module_level (import.c:2133)
==5591==    by 0x4BD133: PyImport_ImportModuleLevel (import.c:2185)
==5591==    by 0x49EBFA: builtin___import__ (bltinmodule.c:48)
==5591==    by 0x41C9D6: PyObject_Call (abstract.c:2492)
==5591==    by 0x49F7BE: T.420 (ceval.c:3619)
==5591==    by 0x4A2B4B: PyEval_EvalFrameEx (ceval.c:2159)
==5591==    by 0x4A6BD0: PyEval_EvalCodeEx (ceval.c:3000)
==5591== 
==5591== Use of uninitialised value of size 8
==5591==    at 0x454316: PyObject_Free (obmalloc.c:931)
==5591==    by 0x52D63A: code_dealloc (codeobject.c:260)
==5591==    by 0x4BB4BD: load_source_module (import.c:1018)
==5591==    by 0x4BC2D2: import_submodule (import.c:2592)
==5591==    by 0x4BC54E: load_next (import.c:2412)
==5591==    by 0x4BCC10: import_module_level (import.c:2133)
==5591==    by 0x4BD133: PyImport_ImportModuleLevel (import.c:2185)
==5591==    by 0x49EBFA: builtin___import__ (bltinmodule.c:48)
==5591==    by 0x41C9D6: PyObject_Call (abstract.c:2492)
==5591==    by 0x49F7BE: T.420 (ceval.c:3619)
==5591==    by 0x4A2B4B: PyEval_EvalFrameEx (ceval.c:2159)
==5591==    by 0x4A6BD0: PyEval_EvalCodeEx (ceval.c:3000)
==5591== 
==5591== Invalid read of size 4
==5591==    at 0x4542F3: PyObject_Free (obmalloc.c:931)
==5591==    by 0x52D63A: code_dealloc (codeobject.c:260)
==5591==    by 0x467E2C: tupledealloc (tupleobject.c:170)
==5591==    by 0x52D6F2: code_dealloc (codeobject.c:261)
==5591==    by 0x4BB4BD: load_source_module (import.c:1018)
==5591==    by 0x4BC2D2: import_submodule (import.c:2592)
==5591==    by 0x4BC54E: load_next (import.c:2412)
==5591==    by 0x4BCC10: import_module_level (import.c:2133)
==5591==    by 0x4BD133: PyImport_ImportModuleLevel (import.c:2185)
==5591==    by 0x49EBFA: builtin___import__ (bltinmodule.c:48)
==5591==    by 0x41C9D6: PyObject_Call (abstract.c:2492)
==5591==    by 0x49F7BE: T.420 (ceval.c:3619)
==5591==  Address 0x6375020 is 416 bytes inside a block of size 464 free'd
==5591==    at 0x4C29159: realloc (vg_replace_malloc.c:525)
==5591==    by 0x440C20: PyList_Append (listobject.c:62)
==5591==    by 0x4BEFA5: r_object (marshal.c:731)
==5591==    by 0x4BE6DC: r_object (marshal.c:794)
==5591==    by 0x4BE9B1: r_object (marshal.c:933)
==5591==    by 0x4BE6DC: r_object (marshal.c:794)
==5591==    by 0x4BE985: r_object (marshal.c:927)
==5591==    by 0x4C0D44: PyMarshal_ReadLastObjectFromFile (marshal.c:1107)
==5591==    by 0x4BB5B6: load_source_module (import.c:769)
==5591==    by 0x4BC2D2: import_submodule (import.c:2592)
==5591==    by 0x4BC54E: load_next (import.c:2412)
==5591==    by 0x4BCC10: import_module_level (import.c:2133)
==5591== 
==5591== Invalid read of size 4
==5591==    at 0x4542F3: PyObject_Free (obmalloc.c:931)
==5591==    by 0x467E45: tupledealloc (tupleobject.c:183)
==5591==    by 0x52D6DA: code_dealloc (codeobject.c:262)
==5591==    by 0x467E2C: tupledealloc (tupleobject.c:170)
==5591==    by 0x52D6F2: code_dealloc (codeobject.c:261)
==5591==    by 0x4BB4BD: load_source_module (import.c:1018)
==5591==    by 0x4BC2D2: import_submodule (import.c:2592)
==5591==    by 0x4BC54E: load_next (import.c:2412)
==5591==    by 0x4BCC10: import_module_level (import.c:2133)
==5591==    by 0x4BD133: PyImport_ImportModuleLevel (import.c:2185)
==5591==    by 0x49EBFA: builtin___import__ (bltinmodule.c:48)
==5591==    by 0x41C9D6: PyObject_Call (abstract.c:2492)
==5591==  Address 0x6376020 is 256 bytes inside a block of size 872 free'd
==5591==    at 0x4C283A2: free (vg_replace_malloc.c:366)
==5591==    by 0x52D63A: code_dealloc (codeobject.c:260)
==5591==    by 0x467E2C: tupledealloc (tupleobject.c:170)
==5591==    by 0x52D6F2: code_dealloc (codeobject.c:261)
==5591==    by 0x4BB4BD: load_source_module (import.c:1018)
==5591==    by 0x4BC2D2: import_submodule (import.c:2592)
==5591==    by 0x4BC54E: load_next (import.c:2412)
==5591==    by 0x4BCC10: import_module_level (import.c:2133)
==5591==    by 0x4BD133: PyImport_ImportModuleLevel (import.c:2185)
==5591==    by 0x49EBFA: builtin___import__ (bltinmodule.c:48)
==5591==    by 0x41C9D6: PyObject_Call (abstract.c:2492)
==5591==    by 0x49F7BE: T.420 (ceval.c:3619)
==5591== 
==5591== Invalid read of size 4
==5591==    at 0x4542F3: PyObject_Free (obmalloc.c:931)
==5591==    by 0x52D625: code_dealloc (codeobject.c:270)
==5591==    by 0x467E2C: tupledealloc (tupleobject.c:170)
==5591==    by 0x52D6F2: code_dealloc (codeobject.c:261)
==5591==    by 0x4BB4BD: load_source_module (import.c:1018)
==5591==    by 0x4BC2D2: import_submodule (import.c:2592)
==5591==    by 0x4BC54E: load_next (import.c:2412)
==5591==    by 0x4BCC10: import_module_level (import.c:2133)
==5591==    by 0x4BD133: PyImport_ImportModuleLevel (import.c:2185)
==5591==    by 0x49EBFA: builtin___import__ (bltinmodule.c:48)
==5591==    by 0x41C9D6: PyObject_Call (abstract.c:2492)
==5591==    by 0x49F7BE: T.420 (ceval.c:3619)
==5591==  Address 0x63b1020 is 8 bytes after a block of size 280 alloc'd
==5591==    at 0x4C29061: malloc (vg_replace_malloc.c:236)
==5591==    by 0x4D4F38: _PyObject_GC_New (gcmodule.c:1351)
==5591==    by 0x44D2D1: PyDict_New (dictobject.c:256)
==5591==    by 0x534BE9: PyFrame_New (frameobject.c:685)
==5591==    by 0x4A6333: PyEval_EvalCodeEx (ceval.c:2778)
==5591==    by 0x4A4D58: PyEval_EvalFrameEx (ceval.c:3846)
==5591==    by 0x4A6BD0: PyEval_EvalCodeEx (ceval.c:3000)
==5591==    by 0x4A6CA1: PyEval_EvalCode (ceval.c:541)
==5591==    by 0x4B952D: PyImport_ExecCodeModuleEx (import.c:677)
==5591==    by 0x4BB49D: load_source_module (import.c:1017)
==5591==    by 0x4BC2D2: import_submodule (import.c:2592)
==5591==    by 0x4BC54E: load_next (import.c:2412)
==5591== 
==5591== Conditional jump or move depends on uninitialised value(s)
==5591==    at 0x4542FC: PyObject_Free (obmalloc.c:931)
==5591==    by 0x467E45: tupledealloc (tupleobject.c:183)
==5591==    by 0x52D6F2: code_dealloc (codeobject.c:261)
==5591==    by 0x4BB4BD: load_source_module (import.c:1018)
==5591==    by 0x4BC2D2: import_submodule (import.c:2592)
==5591==    by 0x4BC54E: load_next (import.c:2412)
==5591==    by 0x4BCC10: import_module_level (import.c:2133)
==5591==    by 0x4BD133: PyImport_ImportModuleLevel (import.c:2185)
==5591==    by 0x49EBFA: builtin___import__ (bltinmodule.c:48)
==5591==    by 0x41C9D6: PyObject_Call (abstract.c:2492)
==5591==    by 0x49F7BE: T.420 (ceval.c:3619)
==5591==    by 0x4A2B4B: PyEval_EvalFrameEx (ceval.c:2159)
==5591== 
==5591== Use of uninitialised value of size 8
==5591==    at 0x454316: PyObject_Free (obmalloc.c:931)
==5591==    by 0x467E45: tupledealloc (tupleobject.c:183)
==5591==    by 0x52D6F2: code_dealloc (codeobject.c:261)
==5591==    by 0x4BB4BD: load_source_module (import.c:1018)
==5591==    by 0x4BC2D2: import_submodule (import.c:2592)
==5591==    by 0x4BC54E: load_next (import.c:2412)
==5591==    by 0x4BCC10: import_module_level (import.c:2133)
==5591==    by 0x4BD133: PyImport_ImportModuleLevel (import.c:2185)
==5591==    by 0x49EBFA: builtin___import__ (bltinmodule.c:48)
==5591==    by 0x41C9D6: PyObject_Call (abstract.c:2492)
==5591==    by 0x49F7BE: T.420 (ceval.c:3619)
==5591==    by 0x4A2B4B: PyEval_EvalFrameEx (ceval.c:2159)
==5591== 
==5591== Invalid read of size 4
==5591==    at 0x4542F3: PyObject_Free (obmalloc.c:931)
==5591==    by 0x5228C9: PyGrammar_AddAccelerators (acceler.c:124)
==5591==    by 0x5237B4: PyParser_New (parser.c:77)
==5591==    by 0x418292: parsetok (parsetok.c:136)
==5591==    by 0x4C7A41: PyRun_StringFlags (pythonrun.c:1445)
==5591==    by 0x49E3CE: builtin_eval (bltinmodule.c:686)
==5591==    by 0x4A51AD: PyEval_EvalFrameEx (ceval.c:3750)
==5591==    by 0x4A5AAF: PyEval_EvalFrameEx (ceval.c:3836)
==5591==    by 0x4A6BD0: PyEval_EvalCodeEx (ceval.c:3000)
==5591==    by 0x4A6CA1: PyEval_EvalCode (ceval.c:541)
==5591==    by 0x4B952D: PyImport_ExecCodeModuleEx (import.c:677)
==5591==    by 0x4BB49D: load_source_module (import.c:1017)
==5591==  Address 0x6425020 is not stack'd, malloc'd or (recently) free'd
==5591== 
==5591== Invalid read of size 4
==5591==    at 0x4542F3: PyObject_Free (obmalloc.c:931)
==5591==    by 0x4A70BD: compiler_unit_free (compile.c:423)
==5591==    by 0x4B04E5: compiler_mod (compile.c:521)
==5591==    by 0x4B0711: PyAST_Compile (compile.c:288)
==5591==    by 0x4C7A90: PyRun_StringFlags (pythonrun.c:1348)
==5591==    by 0x49E3CE: builtin_eval (bltinmodule.c:686)
==5591==    by 0x4A51AD: PyEval_EvalFrameEx (ceval.c:3750)
==5591==    by 0x4A5AAF: PyEval_EvalFrameEx (ceval.c:3836)
==5591==    by 0x4A6BD0: PyEval_EvalCodeEx (ceval.c:3000)
==5591==    by 0x4A6CA1: PyEval_EvalCode (ceval.c:541)
==5591==    by 0x4B952D: PyImport_ExecCodeModuleEx (import.c:677)
==5591==    by 0x4BB49D: load_source_module (import.c:1017)
==5591==  Address 0x6483020 is not stack'd, malloc'd or (recently) free'd
==5591== 
==5591== Invalid read of size 4
==5591==    at 0x4542F3: PyObject_Free (obmalloc.c:931)
==5591==    by 0x4B04E5: compiler_mod (compile.c:521)
==5591==    by 0x4B0711: PyAST_Compile (compile.c:288)
==5591==    by 0x4C7A90: PyRun_StringFlags (pythonrun.c:1348)
==5591==    by 0x49E3CE: builtin_eval (bltinmodule.c:686)
==5591==    by 0x4A51AD: PyEval_EvalFrameEx (ceval.c:3750)
==5591==    by 0x4A5AAF: PyEval_EvalFrameEx (ceval.c:3836)
==5591==    by 0x4A6BD0: PyEval_EvalCodeEx (ceval.c:3000)
==5591==    by 0x4A6CA1: PyEval_EvalCode (ceval.c:541)
==5591==    by 0x4B952D: PyImport_ExecCodeModuleEx (import.c:677)
==5591==    by 0x4BB49D: load_source_module (import.c:1017)
==5591==    by 0x4BC2D2: import_submodule (import.c:2592)
==5591==  Address 0x6483020 is not stack'd, malloc'd or (recently) free'd
==5591== 
==5591== Invalid read of size 4
==5591==    at 0x4542F3: PyObject_Free (obmalloc.c:931)
==5591==    by 0x52D625: code_dealloc (codeobject.c:270)
==5591==    by 0x4C7B05: PyRun_StringFlags (pythonrun.c:1352)
==5591==    by 0x49E3CE: builtin_eval (bltinmodule.c:686)
==5591==    by 0x4A51AD: PyEval_EvalFrameEx (ceval.c:3750)
==5591==    by 0x4A5AAF: PyEval_EvalFrameEx (ceval.c:3836)
==5591==    by 0x4A6BD0: PyEval_EvalCodeEx (ceval.c:3000)
==5591==    by 0x4A6CA1: PyEval_EvalCode (ceval.c:541)
==5591==    by 0x4B952D: PyImport_ExecCodeModuleEx (import.c:677)
==5591==    by 0x4BB49D: load_source_module (import.c:1017)
==5591==    by 0x4BC2D2: import_submodule (import.c:2592)
==5591==    by 0x4BC54E: load_next (import.c:2412)
==5591==  Address 0x6483020 is not stack'd, malloc'd or (recently) free'd
==5591== 
==5591== Invalid read of size 4
==5591==    at 0x4542F3: PyObject_Free (obmalloc.c:931)
==5591==    by 0x52D625: code_dealloc (codeobject.c:270)
==5591==    by 0x534419: frame_dealloc (frameobject.c:448)
==5591==    by 0x4CE912: tb_dealloc (traceback.c:28)
==5591==    by 0x4CE922: tb_dealloc (traceback.c:27)
==5591==    by 0x44D7F6: PyDict_DelItem (dictobject.c:755)
==5591==    by 0x44D860: PyDict_DelItemString (dictobject.c:2314)
==5591==    by 0x4A075D: PyEval_EvalFrameEx (ceval.c:3166)
==5591==    by 0x4A5AAF: PyEval_EvalFrameEx (ceval.c:3836)
==5591==    by 0x4A6BD0: PyEval_EvalCodeEx (ceval.c:3000)
==5591==    by 0x4A6CA1: PyEval_EvalCode (ceval.c:541)
==5591==    by 0x4B952D: PyImport_ExecCodeModuleEx (import.c:677)
==5591==  Address 0x648f020 is 35,424 bytes inside a block of size 36,032 free'd
==5591==    at 0x4C283A2: free (vg_replace_malloc.c:366)
==5591==    by 0x4184A7: parsetok (parsetok.c:219)
==5591==    by 0x4C7A41: PyRun_StringFlags (pythonrun.c:1445)
==5591==    by 0x49E3CE: builtin_eval (bltinmodule.c:686)
==5591==    by 0x4A51AD: PyEval_EvalFrameEx (ceval.c:3750)
==5591==    by 0x4A5AAF: PyEval_EvalFrameEx (ceval.c:3836)
==5591==    by 0x4A6BD0: PyEval_EvalCodeEx (ceval.c:3000)
==5591==    by 0x4A6CA1: PyEval_EvalCode (ceval.c:541)
==5591==    by 0x4B952D: PyImport_ExecCodeModuleEx (import.c:677)
==5591==    by 0x4BB49D: load_source_module (import.c:1017)
==5591==    by 0x4BC2D2: import_submodule (import.c:2592)
==5591==    by 0x4BC54E: load_next (import.c:2412)
==5591== 
==5591== Invalid read of size 4
==5591==    at 0x4542F3: PyObject_Free (obmalloc.c:931)
==5591==    by 0x52D64A: code_dealloc (codeobject.c:268)
==5591==    by 0x4BB4BD: load_source_module (import.c:1018)
==5591==    by 0x4BC2D2: import_submodule (import.c:2592)
==5591==    by 0x4BC54E: load_next (import.c:2412)
==5591==    by 0x4BCC10: import_module_level (import.c:2133)
==5591==    by 0x4BD133: PyImport_ImportModuleLevel (import.c:2185)
==5591==    by 0x49EBFA: builtin___import__ (bltinmodule.c:48)
==5591==    by 0x41C9D6: PyObject_Call (abstract.c:2492)
==5591==    by 0x49F7BE: T.420 (ceval.c:3619)
==5591==    by 0x4A2B4B: PyEval_EvalFrameEx (ceval.c:2159)
==5591==    by 0x4A6BD0: PyEval_EvalCodeEx (ceval.c:3000)
==5591==  Address 0x6342020 is 480 bytes inside a block of size 768 free'd
==5591==    at 0x4C283A2: free (vg_replace_malloc.c:366)
==5591==    by 0x467E45: tupledealloc (tupleobject.c:183)
==5591==    by 0x52D6DA: code_dealloc (codeobject.c:262)
==5591==    by 0x4BB4BD: load_source_module (import.c:1018)
==5591==    by 0x4BC2D2: import_submodule (import.c:2592)
==5591==    by 0x4BC54E: load_next (import.c:2412)
==5591==    by 0x4BCC10: import_module_level (import.c:2133)
==5591==    by 0x4BD133: PyImport_ImportModuleLevel (import.c:2185)
==5591==    by 0x49EBFA: builtin___import__ (bltinmodule.c:48)
==5591==    by 0x41C9D6: PyObject_Call (abstract.c:2492)
==5591==    by 0x49F7BE: T.420 (ceval.c:3619)
==5591==    by 0x4A2B4B: PyEval_EvalFrameEx (ceval.c:2159)
==5591== 
==5591== Invalid read of size 4
==5591==    at 0x4542F3: PyObject_Free (obmalloc.c:931)
==5591==    by 0x52D625: code_dealloc (codeobject.c:270)
==5591==    by 0x4C7B05: PyRun_StringFlags (pythonrun.c:1352)
==5591==    by 0x4A3C18: PyEval_EvalFrameEx (ceval.c:4481)
==5591==    by 0x4A6BD0: PyEval_EvalCodeEx (ceval.c:3000)
==5591==    by 0x4A4D58: PyEval_EvalFrameEx (ceval.c:3846)
==5591==    by 0x4A6BD0: PyEval_EvalCodeEx (ceval.c:3000)
==5591==    by 0x4A4D58: PyEval_EvalFrameEx (ceval.c:3846)
==5591==    by 0x4A5AAF: PyEval_EvalFrameEx (ceval.c:3836)
==5591==    by 0x4A5AAF: PyEval_EvalFrameEx (ceval.c:3836)
==5591==    by 0x4A6BD0: PyEval_EvalCodeEx (ceval.c:3000)
==5591==    by 0x4A6CA1: PyEval_EvalCode (ceval.c:541)
==5591==  Address 0x6513020 is 320 bytes inside a block of size 432 free'd
==5591==    at 0x4C283A2: free (vg_replace_malloc.c:366)
==5591==    by 0x4B04E5: compiler_mod (compile.c:521)
==5591==    by 0x4B0711: PyAST_Compile (compile.c:288)
==5591==    by 0x4C7A90: PyRun_StringFlags (pythonrun.c:1348)
==5591==    by 0x4A3C18: PyEval_EvalFrameEx (ceval.c:4481)
==5591==    by 0x4A6BD0: PyEval_EvalCodeEx (ceval.c:3000)
==5591==    by 0x4A4D58: PyEval_EvalFrameEx (ceval.c:3846)
==5591==    by 0x4A6BD0: PyEval_EvalCodeEx (ceval.c:3000)
==5591==    by 0x4A4D58: PyEval_EvalFrameEx (ceval.c:3846)
==5591==    by 0x4A5AAF: PyEval_EvalFrameEx (ceval.c:3836)
==5591==    by 0x4A5AAF: PyEval_EvalFrameEx (ceval.c:3836)
==5591==    by 0x4A6BD0: PyEval_EvalCodeEx (ceval.c:3000)
==5591== 
==5591== Conditional jump or move depends on uninitialised value(s)
==5591==    at 0x4548C7: PyObject_Realloc (obmalloc.c:1154)
==5591==    by 0x522C49: PyNode_AddChild (node.c:98)
==5591==    by 0x5233D6: PyParser_AddToken (parser.c:126)
==5591==    by 0x41830B: parsetok (parsetok.c:199)
==5591==    by 0x4C7B39: PyRun_StringFlags (pythonrun.c:1445)
==5591==    by 0x4A3C18: PyEval_EvalFrameEx (ceval.c:4481)
==5591==    by 0x4A6BD0: PyEval_EvalCodeEx (ceval.c:3000)
==5591==    by 0x4A4D58: PyEval_EvalFrameEx (ceval.c:3846)
==5591==    by 0x4A6BD0: PyEval_EvalCodeEx (ceval.c:3000)
==5591==    by 0x4A4D58: PyEval_EvalFrameEx (ceval.c:3846)
==5591==    by 0x4A5AAF: PyEval_EvalFrameEx (ceval.c:3836)
==5591==    by 0x4A5AAF: PyEval_EvalFrameEx (ceval.c:3836)
==5591== 
==5591== Use of uninitialised value of size 8
==5591==    at 0x4548DA: PyObject_Realloc (obmalloc.c:1154)
==5591==    by 0x522C49: PyNode_AddChild (node.c:98)
==5591==    by 0x5233D6: PyParser_AddToken (parser.c:126)
==5591==    by 0x41830B: parsetok (parsetok.c:199)
==5591==    by 0x4C7B39: PyRun_StringFlags (pythonrun.c:1445)
==5591==    by 0x4A3C18: PyEval_EvalFrameEx (ceval.c:4481)
==5591==    by 0x4A6BD0: PyEval_EvalCodeEx (ceval.c:3000)
==5591==    by 0x4A4D58: PyEval_EvalFrameEx (ceval.c:3846)
==5591==    by 0x4A6BD0: PyEval_EvalCodeEx (ceval.c:3000)
==5591==    by 0x4A4D58: PyEval_EvalFrameEx (ceval.c:3846)
==5591==    by 0x4A5AAF: PyEval_EvalFrameEx (ceval.c:3836)
==5591==    by 0x4A5AAF: PyEval_EvalFrameEx (ceval.c:3836)
==5591== 
==5591== Invalid read of size 4
==5591==    at 0x4542F3: PyObject_Free (obmalloc.c:931)
==5591==    by 0x523055: freechildren (node.c:135)
==5591==    by 0x522F3B: freechildren (node.c:133)
==5591==    by 0x522F3B: freechildren (node.c:133)
==5591==    by 0x5230FD: PyNode_Free (node.c:123)
==5591==    by 0x4C7A76: PyRun_StringFlags (pythonrun.c:1455)
==5591==    by 0x4A3C18: PyEval_EvalFrameEx (ceval.c:4481)
==5591==    by 0x4A6BD0: PyEval_EvalCodeEx (ceval.c:3000)
==5591==    by 0x4A4D58: PyEval_EvalFrameEx (ceval.c:3846)
==5591==    by 0x4A6BD0: PyEval_EvalCodeEx (ceval.c:3000)
==5591==    by 0x4A4D58: PyEval_EvalFrameEx (ceval.c:3846)
==5591==    by 0x4A5AAF: PyEval_EvalFrameEx (ceval.c:3836)
==5591==  Address 0x651f020 is 208 bytes inside a block of size 320 free'd
==5591==    at 0x4C29159: realloc (vg_replace_malloc.c:525)
==5591==    by 0x522C49: PyNode_AddChild (node.c:98)
==5591==    by 0x5233D6: PyParser_AddToken (parser.c:126)
==5591==    by 0x41830B: parsetok (parsetok.c:199)
==5591==    by 0x4C7B39: PyRun_StringFlags (pythonrun.c:1445)
==5591==    by 0x4A3C18: PyEval_EvalFrameEx (ceval.c:4481)
==5591==    by 0x4A6BD0: PyEval_EvalCodeEx (ceval.c:3000)
==5591==    by 0x4A4D58: PyEval_EvalFrameEx (ceval.c:3846)
==5591==    by 0x4A6BD0: PyEval_EvalCodeEx (ceval.c:3000)
==5591==    by 0x4A4D58: PyEval_EvalFrameEx (ceval.c:3846)
==5591==    by 0x4A5AAF: PyEval_EvalFrameEx (ceval.c:3836)
==5591==    by 0x4A5AAF: PyEval_EvalFrameEx (ceval.c:3836)
==5591== 
==5591== Invalid read of size 4
==5591==    at 0x4542F3: PyObject_Free (obmalloc.c:931)
==5591==    by 0x523055: freechildren (node.c:135)
==5591==    by 0x5230FD: PyNode_Free (node.c:123)
==5591==    by 0x4C7A76: PyRun_StringFlags (pythonrun.c:1455)
==5591==    by 0x4A3C18: PyEval_EvalFrameEx (ceval.c:4481)
==5591==    by 0x4A6BD0: PyEval_EvalCodeEx (ceval.c:3000)
==5591==    by 0x4A4D58: PyEval_EvalFrameEx (ceval.c:3846)
==5591==    by 0x4A6BD0: PyEval_EvalCodeEx (ceval.c:3000)
==5591==    by 0x4A4D58: PyEval_EvalFrameEx (ceval.c:3846)
==5591==    by 0x4A5AAF: PyEval_EvalFrameEx (ceval.c:3836)
==5591==    by 0x4A5AAF: PyEval_EvalFrameEx (ceval.c:3836)
==5591==    by 0x4A6BD0: PyEval_EvalCodeEx (ceval.c:3000)
==5591==  Address 0x651f020 is 208 bytes inside a block of size 320 free'd
==5591==    at 0x4C29159: realloc (vg_replace_malloc.c:525)
==5591==    by 0x522C49: PyNode_AddChild (node.c:98)
==5591==    by 0x5233D6: PyParser_AddToken (parser.c:126)
==5591==    by 0x41830B: parsetok (parsetok.c:199)
==5591==    by 0x4C7B39: PyRun_StringFlags (pythonrun.c:1445)
==5591==    by 0x4A3C18: PyEval_EvalFrameEx (ceval.c:4481)
==5591==    by 0x4A6BD0: PyEval_EvalCodeEx (ceval.c:3000)
==5591==    by 0x4A4D58: PyEval_EvalFrameEx (ceval.c:3846)
==5591==    by 0x4A6BD0: PyEval_EvalCodeEx (ceval.c:3000)
==5591==    by 0x4A4D58: PyEval_EvalFrameEx (ceval.c:3846)
==5591==    by 0x4A5AAF: PyEval_EvalFrameEx (ceval.c:3836)
==5591==    by 0x4A5AAF: PyEval_EvalFrameEx (ceval.c:3836)
==5591== 
==5591== Invalid read of size 4
==5591==    at 0x4548BE: PyObject_Realloc (obmalloc.c:1154)
==5591==    by 0x4A7758: compiler_next_instr (compile.c:639)
==5591==    by 0x4A9380: compiler_nameop (compile.c:1026)
==5591==    by 0x4AA966: compiler_visit_expr (compile.c:3085)
==5591==    by 0x4AB5B7: compiler_visit_expr (compile.c:2569)
==5591==    by 0x4ADDC6: compiler_visit_stmt (compile.c:2118)
==5591==    by 0x4B05CD: compiler_mod (compile.c:1187)
==5591==    by 0x4B0711: PyAST_Compile (compile.c:288)
==5591==    by 0x4C7A90: PyRun_StringFlags (pythonrun.c:1348)
==5591==    by 0x4A3C18: PyEval_EvalFrameEx (ceval.c:4481)
==5591==    by 0x4A6BD0: PyEval_EvalCodeEx (ceval.c:3000)
==5591==    by 0x4A4D58: PyEval_EvalFrameEx (ceval.c:3846)
==5591==  Address 0x651f020 is 208 bytes inside a block of size 320 free'd
==5591==    at 0x4C29159: realloc (vg_replace_malloc.c:525)
==5591==    by 0x522C49: PyNode_AddChild (node.c:98)
==5591==    by 0x5233D6: PyParser_AddToken (parser.c:126)
==5591==    by 0x41830B: parsetok (parsetok.c:199)
==5591==    by 0x4C7B39: PyRun_StringFlags (pythonrun.c:1445)
==5591==    by 0x4A3C18: PyEval_EvalFrameEx (ceval.c:4481)
==5591==    by 0x4A6BD0: PyEval_EvalCodeEx (ceval.c:3000)
==5591==    by 0x4A4D58: PyEval_EvalFrameEx (ceval.c:3846)
==5591==    by 0x4A6BD0: PyEval_EvalCodeEx (ceval.c:3000)
==5591==    by 0x4A4D58: PyEval_EvalFrameEx (ceval.c:3846)
==5591==    by 0x4A5AAF: PyEval_EvalFrameEx (ceval.c:3836)
==5591==    by 0x4A5AAF: PyEval_EvalFrameEx (ceval.c:3836)
==5591== 
==5591== Invalid read of size 4
==5591==    at 0x4542F3: PyObject_Free (obmalloc.c:931)
==5591==    by 0x522F74: freechildren (node.c:135)
==5591==    by 0x522F3B: freechildren (node.c:133)
==5591==    by 0x522F3B: freechildren (node.c:133)
==5591==    by 0x522F3B: freechildren (node.c:133)
==5591==    by 0x5230FD: PyNode_Free (node.c:123)
==5591==    by 0x4C7A76: PyRun_StringFlags (pythonrun.c:1455)
==5591==    by 0x4A3C18: PyEval_EvalFrameEx (ceval.c:4481)
==5591==    by 0x4A6BD0: PyEval_EvalCodeEx (ceval.c:3000)
==5591==    by 0x4A4D58: PyEval_EvalFrameEx (ceval.c:3846)
==5591==    by 0x4A6BD0: PyEval_EvalCodeEx (ceval.c:3000)
==5591==    by 0x4A4D58: PyEval_EvalFrameEx (ceval.c:3846)
==5591==  Address 0x6564020 is 35,168 bytes inside a block of size 36,032 free'd
==5591==    at 0x4C283A2: free (vg_replace_malloc.c:366)
==5591==    by 0x4184A7: parsetok (parsetok.c:219)
==5591==    by 0x4C7B39: PyRun_StringFlags (pythonrun.c:1445)
==5591==    by 0x4A3C18: PyEval_EvalFrameEx (ceval.c:4481)
==5591==    by 0x4A6BD0: PyEval_EvalCodeEx (ceval.c:3000)
==5591==    by 0x4A4D58: PyEval_EvalFrameEx (ceval.c:3846)
==5591==    by 0x4A6BD0: PyEval_EvalCodeEx (ceval.c:3000)
==5591==    by 0x4A4D58: PyEval_EvalFrameEx (ceval.c:3846)
==5591==    by 0x4A5AAF: PyEval_EvalFrameEx (ceval.c:3836)
==5591==    by 0x4A5AAF: PyEval_EvalFrameEx (ceval.c:3836)
==5591==    by 0x4A6BD0: PyEval_EvalCodeEx (ceval.c:3000)
==5591==    by 0x4A6CA1: PyEval_EvalCode (ceval.c:541)
==5591== 
==5591== Invalid read of size 4
==5591==    at 0x4542F3: PyObject_Free (obmalloc.c:931)
==5591==    by 0x522F9E: freechildren (node.c:135)
==5591==    by 0x5230FD: PyNode_Free (node.c:123)
==5591==    by 0x4C7A76: PyRun_StringFlags (pythonrun.c:1455)
==5591==    by 0x4A3C18: PyEval_EvalFrameEx (ceval.c:4481)
==5591==    by 0x4A6BD0: PyEval_EvalCodeEx (ceval.c:3000)
==5591==    by 0x4A4D58: PyEval_EvalFrameEx (ceval.c:3846)
==5591==    by 0x4A6BD0: PyEval_EvalCodeEx (ceval.c:3000)
==5591==    by 0x4A4D58: PyEval_EvalFrameEx (ceval.c:3846)
==5591==    by 0x4A5AAF: PyEval_EvalFrameEx (ceval.c:3836)
==5591==    by 0x4A5AAF: PyEval_EvalFrameEx (ceval.c:3836)
==5591==    by 0x4A6BD0: PyEval_EvalCodeEx (ceval.c:3000)
==5591==  Address 0x6564020 is 35,168 bytes inside a block of size 36,032 free'd
==5591==    at 0x4C283A2: free (vg_replace_malloc.c:366)
==5591==    by 0x4184A7: parsetok (parsetok.c:219)
==5591==    by 0x4C7B39: PyRun_StringFlags (pythonrun.c:1445)
==5591==    by 0x4A3C18: PyEval_EvalFrameEx (ceval.c:4481)
==5591==    by 0x4A6BD0: PyEval_EvalCodeEx (ceval.c:3000)
==5591==    by 0x4A4D58: PyEval_EvalFrameEx (ceval.c:3846)
==5591==    by 0x4A6BD0: PyEval_EvalCodeEx (ceval.c:3000)
==5591==    by 0x4A4D58: PyEval_EvalFrameEx (ceval.c:3846)
==5591==    by 0x4A5AAF: PyEval_EvalFrameEx (ceval.c:3836)
==5591==    by 0x4A5AAF: PyEval_EvalFrameEx (ceval.c:3836)
==5591==    by 0x4A6BD0: PyEval_EvalCodeEx (ceval.c:3000)
==5591==    by 0x4A6CA1: PyEval_EvalCode (ceval.c:541)
==5591== 
==5591== Invalid read of size 4
==5591==    at 0x4542F3: PyObject_Free (obmalloc.c:931)
==5591==    by 0x44D5D2: dict_dealloc (dictobject.c:911)
==5591==    by 0x5344A2: frame_dealloc (frameobject.c:431)
==5591==    by 0x4A6846: PyEval_EvalCodeEx (ceval.c:3011)
==5591==    by 0x4A4D58: PyEval_EvalFrameEx (ceval.c:3846)
==5591==    by 0x4A6BD0: PyEval_EvalCodeEx (ceval.c:3000)
==5591==    by 0x4A4D58: PyEval_EvalFrameEx (ceval.c:3846)
==5591==    by 0x4A5AAF: PyEval_EvalFrameEx (ceval.c:3836)
==5591==    by 0x4A5AAF: PyEval_EvalFrameEx (ceval.c:3836)
==5591==    by 0x4A6BD0: PyEval_EvalCodeEx (ceval.c:3000)
==5591==    by 0x4A6CA1: PyEval_EvalCode (ceval.c:541)
==5591==    by 0x4B952D: PyImport_ExecCodeModuleEx (import.c:677)
==5591==  Address 0x6558020 is 4,976 bytes inside a block of size 8,192 free'd
==5591==    at 0x4C283A2: free (vg_replace_malloc.c:366)
==5591==    by 0x43266B: readahead_get_line_skip (fileobject.c:2039)
==5591==    by 0x432792: file_iternext (fileobject.c:2143)
==5591==    by 0x4A0AFD: PyEval_EvalFrameEx (ceval.c:2272)
==5591==    by 0x4A6BD0: PyEval_EvalCodeEx (ceval.c:3000)
==5591==    by 0x4A4D58: PyEval_EvalFrameEx (ceval.c:3846)
==5591==    by 0x4A6BD0: PyEval_EvalCodeEx (ceval.c:3000)
==5591==    by 0x4A4D58: PyEval_EvalFrameEx (ceval.c:3846)
==5591==    by 0x4A5AAF: PyEval_EvalFrameEx (ceval.c:3836)
==5591==    by 0x4A5AAF: PyEval_EvalFrameEx (ceval.c:3836)
==5591==    by 0x4A6BD0: PyEval_EvalCodeEx (ceval.c:3000)
==5591==    by 0x4A6CA1: PyEval_EvalCode (ceval.c:541)
==5591== 
==5591== Invalid read of size 4
==5591==    at 0x4548BE: PyObject_Realloc (obmalloc.c:1154)
==5591==    by 0x522C49: PyNode_AddChild (node.c:98)
==5591==    by 0x5233D6: PyParser_AddToken (parser.c:126)
==5591==    by 0x41830B: parsetok (parsetok.c:199)
==5591==    by 0x4C7B39: PyRun_StringFlags (pythonrun.c:1445)
==5591==    by 0x4A3C18: PyEval_EvalFrameEx (ceval.c:4481)
==5591==    by 0x4A6BD0: PyEval_EvalCodeEx (ceval.c:3000)
==5591==    by 0x4A4D58: PyEval_EvalFrameEx (ceval.c:3846)
==5591==    by 0x4A6BD0: PyEval_EvalCodeEx (ceval.c:3000)
==5591==    by 0x4A4D58: PyEval_EvalFrameEx (ceval.c:3846)
==5591==    by 0x4A5AAF: PyEval_EvalFrameEx (ceval.c:3836)
==5591==    by 0x4A5AAF: PyEval_EvalFrameEx (ceval.c:3836)
==5591==  Address 0x658c020 is not stack'd, malloc'd or (recently) free'd
==5591== 
==5591== Invalid read of size 4
==5591==    at 0x4548BE: PyObject_Realloc (obmalloc.c:1154)
==5591==    by 0x4A7758: compiler_next_instr (compile.c:639)
==5591==    by 0x4A8C85: compiler_addop_name (compile.c:1026)
==5591==    by 0x4AB80D: compiler_visit_expr (compile.c:3044)
==5591==    by 0x4AB583: compiler_visit_expr (compile.c:2567)
==5591==    by 0x4AA8FD: compiler_visit_expr (compile.c:3032)
==5591==    by 0x4AB9C1: compiler_visit_expr (compile.c:3063)
==5591==    by 0x4AB5B7: compiler_visit_expr (compile.c:2569)
==5591==    by 0x4ADDC6: compiler_visit_stmt (compile.c:2118)
==5591==    by 0x4B05CD: compiler_mod (compile.c:1187)
==5591==    by 0x4B0711: PyAST_Compile (compile.c:288)
==5591==    by 0x4C7A90: PyRun_StringFlags (pythonrun.c:1348)
==5591==  Address 0x658d020 is 304 bytes inside a block of size 768 free'd
==5591==    at 0x4C283A2: free (vg_replace_malloc.c:366)
==5591==    by 0x44D5EE: dict_dealloc (dictobject.c:915)
==5591==    by 0x4C9A3D: analyze_block (symtable.c:719)
==5591==    by 0x4CBC4F: PySymtable_Build (symtable.c:787)
==5591==    by 0x4B06EF: PyAST_Compile (compile.c:278)
==5591==    by 0x4C7A90: PyRun_StringFlags (pythonrun.c:1348)
==5591==    by 0x4A3C18: PyEval_EvalFrameEx (ceval.c:4481)
==5591==    by 0x4A6BD0: PyEval_EvalCodeEx (ceval.c:3000)
==5591==    by 0x4A4D58: PyEval_EvalFrameEx (ceval.c:3846)
==5591==    by 0x4A6BD0: PyEval_EvalCodeEx (ceval.c:3000)
==5591==    by 0x4A4D58: PyEval_EvalFrameEx (ceval.c:3846)
==5591==    by 0x4A5AAF: PyEval_EvalFrameEx (ceval.c:3836)
==5591== 
==5591== Invalid read of size 4
==5591==    at 0x4542F3: PyObject_Free (obmalloc.c:931)
==5591==    by 0x44BEE6: insertdict (dictobject.c:459)
==5591==    by 0x44E3F6: PyDict_SetItem (dictobject.c:701)
==5591==    by 0x535528: PyFrame_FastToLocals (frameobject.c:769)
==5591==    by 0x49F84F: PyEval_GetLocals (ceval.c:3517)
==5591==    by 0x4A6130: PyEval_EvalFrameEx (ceval.c:4408)
==5591==    by 0x4A6BD0: PyEval_EvalCodeEx (ceval.c:3000)
==5591==    by 0x4A4D58: PyEval_EvalFrameEx (ceval.c:3846)
==5591==    by 0x4A6BD0: PyEval_EvalCodeEx (ceval.c:3000)
==5591==    by 0x4A4D58: PyEval_EvalFrameEx (ceval.c:3846)
==5591==    by 0x4A5AAF: PyEval_EvalFrameEx (ceval.c:3836)
==5591==    by 0x4A5AAF: PyEval_EvalFrameEx (ceval.c:3836)
==5591==  Address 0x6580020 is 6,256 bytes inside a block of size 8,192 free'd
==5591==    at 0x4C283A2: free (vg_replace_malloc.c:366)
==5591==    by 0x43266B: readahead_get_line_skip (fileobject.c:2039)
==5591==    by 0x432792: file_iternext (fileobject.c:2143)
==5591==    by 0x4A0AFD: PyEval_EvalFrameEx (ceval.c:2272)
==5591==    by 0x4A6BD0: PyEval_EvalCodeEx (ceval.c:3000)
==5591==    by 0x4A4D58: PyEval_EvalFrameEx (ceval.c:3846)
==5591==    by 0x4A6BD0: PyEval_EvalCodeEx (ceval.c:3000)
==5591==    by 0x4A4D58: PyEval_EvalFrameEx (ceval.c:3846)
==5591==    by 0x4A5AAF: PyEval_EvalFrameEx (ceval.c:3836)
==5591==    by 0x4A5AAF: PyEval_EvalFrameEx (ceval.c:3836)
==5591==    by 0x4A6BD0: PyEval_EvalCodeEx (ceval.c:3000)
==5591==    by 0x4A6CA1: PyEval_EvalCode (ceval.c:541)
==5591== 
==5591== Invalid read of size 4
==5591==    at 0x4542F3: PyObject_Free (obmalloc.c:931)
==5591==    by 0x4AA0DF: assemble (compile.c:3499)
==5591==    by 0x4B04D5: compiler_mod (compile.c:1231)
==5591==    by 0x4B0711: PyAST_Compile (compile.c:288)
==5591==    by 0x4C7A90: PyRun_StringFlags (pythonrun.c:1348)
==5591==    by 0x4A3C18: PyEval_EvalFrameEx (ceval.c:4481)
==5591==    by 0x4A6BD0: PyEval_EvalCodeEx (ceval.c:3000)
==5591==    by 0x4A4D58: PyEval_EvalFrameEx (ceval.c:3846)
==5591==    by 0x4A6BD0: PyEval_EvalCodeEx (ceval.c:3000)
==5591==    by 0x4A4D58: PyEval_EvalFrameEx (ceval.c:3846)
==5591==    by 0x4A5AAF: PyEval_EvalFrameEx (ceval.c:3836)
==5591==    by 0x4A5AAF: PyEval_EvalFrameEx (ceval.c:3836)
==5591==  Address 0x659f020 is 176 bytes inside a block of size 296 free'd
==5591==    at 0x4C29159: realloc (vg_replace_malloc.c:525)
==5591==    by 0x45E346: _PyString_Resize (stringobject.c:4277)
==5591==    by 0x4A9AE3: assemble (compile.c:3987)
==5591==    by 0x4B04D5: compiler_mod (compile.c:1231)
==5591==    by 0x4B0711: PyAST_Compile (compile.c:288)
==5591==    by 0x4C7A90: PyRun_StringFlags (pythonrun.c:1348)
==5591==    by 0x4A3C18: PyEval_EvalFrameEx (ceval.c:4481)
==5591==    by 0x4A6BD0: PyEval_EvalCodeEx (ceval.c:3000)
==5591==    by 0x4A4D58: PyEval_EvalFrameEx (ceval.c:3846)
==5591==    by 0x4A6BD0: PyEval_EvalCodeEx (ceval.c:3000)
==5591==    by 0x4A4D58: PyEval_EvalFrameEx (ceval.c:3846)
==5591==    by 0x4A5AAF: PyEval_EvalFrameEx (ceval.c:3836)
==5591== 
==5591== Invalid read of size 4
==5591==    at 0x4542F3: PyObject_Free (obmalloc.c:931)
==5591==    by 0x52D63A: code_dealloc (codeobject.c:260)
==5591==    by 0x4C7B05: PyRun_StringFlags (pythonrun.c:1352)
==5591==    by 0x4A3C18: PyEval_EvalFrameEx (ceval.c:4481)
==5591==    by 0x4A6BD0: PyEval_EvalCodeEx (ceval.c:3000)
==5591==    by 0x4A4D58: PyEval_EvalFrameEx (ceval.c:3846)
==5591==    by 0x4A6BD0: PyEval_EvalCodeEx (ceval.c:3000)
==5591==    by 0x4A4D58: PyEval_EvalFrameEx (ceval.c:3846)
==5591==    by 0x4A5AAF: PyEval_EvalFrameEx (ceval.c:3836)
==5591==    by 0x4A5AAF: PyEval_EvalFrameEx (ceval.c:3836)
==5591==    by 0x4A6BD0: PyEval_EvalCodeEx (ceval.c:3000)
==5591==    by 0x4A6CA1: PyEval_EvalCode (ceval.c:541)
==5591==  Address 0x659f020 is 176 bytes inside a block of size 296 free'd
==5591==    at 0x4C29159: realloc (vg_replace_malloc.c:525)
==5591==    by 0x45E346: _PyString_Resize (stringobject.c:4277)
==5591==    by 0x4A9AE3: assemble (compile.c:3987)
==5591==    by 0x4B04D5: compiler_mod (compile.c:1231)
==5591==    by 0x4B0711: PyAST_Compile (compile.c:288)
==5591==    by 0x4C7A90: PyRun_StringFlags (pythonrun.c:1348)
==5591==    by 0x4A3C18: PyEval_EvalFrameEx (ceval.c:4481)
==5591==    by 0x4A6BD0: PyEval_EvalCodeEx (ceval.c:3000)
==5591==    by 0x4A4D58: PyEval_EvalFrameEx (ceval.c:3846)
==5591==    by 0x4A6BD0: PyEval_EvalCodeEx (ceval.c:3000)
==5591==    by 0x4A4D58: PyEval_EvalFrameEx (ceval.c:3846)
==5591==    by 0x4A5AAF: PyEval_EvalFrameEx (ceval.c:3836)
==5591== 
==5591== Conditional jump or move depends on uninitialised value(s)
==5591==    at 0x4542FC: PyObject_Free (obmalloc.c:931)
==5591==    by 0x44BEE6: insertdict (dictobject.c:459)
==5591==    by 0x44E3F6: PyDict_SetItem (dictobject.c:701)
==5591==    by 0x535528: PyFrame_FastToLocals (frameobject.c:769)
==5591==    by 0x49F84F: PyEval_GetLocals (ceval.c:3517)
==5591==    by 0x4A6130: PyEval_EvalFrameEx (ceval.c:4408)
==5591==    by 0x4A6BD0: PyEval_EvalCodeEx (ceval.c:3000)
==5591==    by 0x4A4D58: PyEval_EvalFrameEx (ceval.c:3846)
==5591==    by 0x4A6BD0: PyEval_EvalCodeEx (ceval.c:3000)
==5591==    by 0x4A4D58: PyEval_EvalFrameEx (ceval.c:3846)
==5591==    by 0x4A5AAF: PyEval_EvalFrameEx (ceval.c:3836)
==5591==    by 0x4A5AAF: PyEval_EvalFrameEx (ceval.c:3836)
==5591== 
==5591== Use of uninitialised value of size 8
==5591==    at 0x454316: PyObject_Free (obmalloc.c:931)
==5591==    by 0x44BEE6: insertdict (dictobject.c:459)
==5591==    by 0x44E3F6: PyDict_SetItem (dictobject.c:701)
==5591==    by 0x535528: PyFrame_FastToLocals (frameobject.c:769)
==5591==    by 0x49F84F: PyEval_GetLocals (ceval.c:3517)
==5591==    by 0x4A6130: PyEval_EvalFrameEx (ceval.c:4408)
==5591==    by 0x4A6BD0: PyEval_EvalCodeEx (ceval.c:3000)
==5591==    by 0x4A4D58: PyEval_EvalFrameEx (ceval.c:3846)
==5591==    by 0x4A6BD0: PyEval_EvalCodeEx (ceval.c:3000)
==5591==    by 0x4A4D58: PyEval_EvalFrameEx (ceval.c:3846)
==5591==    by 0x4A5AAF: PyEval_EvalFrameEx (ceval.c:3836)
==5591==    by 0x4A5AAF: PyEval_EvalFrameEx (ceval.c:3836)
==5591== 
==5591== Invalid read of size 4
==5591==    at 0x4548BE: PyObject_Realloc (obmalloc.c:1154)
==5591==    by 0x45E346: _PyString_Resize (stringobject.c:4277)
==5591==    by 0x4A9AE3: assemble (compile.c:3987)
==5591==    by 0x4B04D5: compiler_mod (compile.c:1231)
==5591==    by 0x4B0711: PyAST_Compile (compile.c:288)
==5591==    by 0x4C7A90: PyRun_StringFlags (pythonrun.c:1348)
==5591==    by 0x4A3C18: PyEval_EvalFrameEx (ceval.c:4481)
==5591==    by 0x4A6BD0: PyEval_EvalCodeEx (ceval.c:3000)
==5591==    by 0x4A4D58: PyEval_EvalFrameEx (ceval.c:3846)
==5591==    by 0x4A6BD0: PyEval_EvalCodeEx (ceval.c:3000)
==5591==    by 0x4A4D58: PyEval_EvalFrameEx (ceval.c:3846)
==5591==    by 0x4A5AAF: PyEval_EvalFrameEx (ceval.c:3836)
==5591==  Address 0x65e7020 is 16 bytes before a block of size 296 alloc'd
==5591==    at 0x4C29061: malloc (vg_replace_malloc.c:236)
==5591==    by 0x454974: PyObject_Realloc (obmalloc.c:1173)
==5591==    by 0x45E346: _PyString_Resize (stringobject.c:4277)
==5591==    by 0x4A9A32: assemble (compile.c:3689)
==5591==    by 0x4B04D5: compiler_mod (compile.c:1231)
==5591==    by 0x4B0711: PyAST_Compile (compile.c:288)
==5591==    by 0x4C7A90: PyRun_StringFlags (pythonrun.c:1348)
==5591==    by 0x4A3C18: PyEval_EvalFrameEx (ceval.c:4481)
==5591==    by 0x4A6BD0: PyEval_EvalCodeEx (ceval.c:3000)
==5591==    by 0x4A4D58: PyEval_EvalFrameEx (ceval.c:3846)
==5591==    by 0x4A6BD0: PyEval_EvalCodeEx (ceval.c:3000)
==5591==    by 0x4A4D58: PyEval_EvalFrameEx (ceval.c:3846)
==5591== 
==5591== Invalid read of size 4
==5591==    at 0x4542F3: PyObject_Free (obmalloc.c:931)
==5591==    by 0x522F4E: freechildren (node.c:135)
==5591==    by 0x522F3B: freechildren (node.c:133)
==5591==    by 0x522F3B: freechildren (node.c:133)
==5591==    by 0x522F3B: freechildren (node.c:133)
==5591==    by 0x5230FD: PyNode_Free (node.c:123)
==5591==    by 0x4C7A76: PyRun_StringFlags (pythonrun.c:1455)
==5591==    by 0x4A3C18: PyEval_EvalFrameEx (ceval.c:4481)
==5591==    by 0x4A6BD0: PyEval_EvalCodeEx (ceval.c:3000)
==5591==    by 0x4A4D58: PyEval_EvalFrameEx (ceval.c:3846)
==5591==    by 0x4A6BD0: PyEval_EvalCodeEx (ceval.c:3000)
==5591==    by 0x4A4D58: PyEval_EvalFrameEx (ceval.c:3846)
==5591==  Address 0x661b020 is 34,464 bytes inside a block of size 36,032 free'd
==5591==    at 0x4C283A2: free (vg_replace_malloc.c:366)
==5591==    by 0x4184A7: parsetok (parsetok.c:219)
==5591==    by 0x4C7B39: PyRun_StringFlags (pythonrun.c:1445)
==5591==    by 0x4A3C18: PyEval_EvalFrameEx (ceval.c:4481)
==5591==    by 0x4A6BD0: PyEval_EvalCodeEx (ceval.c:3000)
==5591==    by 0x4A4D58: PyEval_EvalFrameEx (ceval.c:3846)
==5591==    by 0x4A6BD0: PyEval_EvalCodeEx (ceval.c:3000)
==5591==    by 0x4A4D58: PyEval_EvalFrameEx (ceval.c:3846)
==5591==    by 0x4A5AAF: PyEval_EvalFrameEx (ceval.c:3836)
==5591==    by 0x4A5AAF: PyEval_EvalFrameEx (ceval.c:3836)
==5591==    by 0x4A6BD0: PyEval_EvalCodeEx (ceval.c:3000)
==5591==    by 0x4A6CA1: PyEval_EvalCode (ceval.c:541)
==5591== 
==5591== Conditional jump or move depends on uninitialised value(s)
==5591==    at 0x4542FC: PyObject_Free (obmalloc.c:931)
==5591==    by 0x522F4E: freechildren (node.c:135)
==5591==    by 0x522F3B: freechildren (node.c:133)
==5591==    by 0x522F3B: freechildren (node.c:133)
==5591==    by 0x522F3B: freechildren (node.c:133)
==5591==    by 0x5230FD: PyNode_Free (node.c:123)
==5591==    by 0x4C7A76: PyRun_StringFlags (pythonrun.c:1455)
==5591==    by 0x4A3C18: PyEval_EvalFrameEx (ceval.c:4481)
==5591==    by 0x4A6BD0: PyEval_EvalCodeEx (ceval.c:3000)
==5591==    by 0x4A4D58: PyEval_EvalFrameEx (ceval.c:3846)
==5591==    by 0x4A6BD0: PyEval_EvalCodeEx (ceval.c:3000)
==5591==    by 0x4A4D58: PyEval_EvalFrameEx (ceval.c:3846)
==5591== 
==5591== Use of uninitialised value of size 8
==5591==    at 0x454316: PyObject_Free (obmalloc.c:931)
==5591==    by 0x522F4E: freechildren (node.c:135)
==5591==    by 0x522F3B: freechildren (node.c:133)
==5591==    by 0x522F3B: freechildren (node.c:133)
==5591==    by 0x522F3B: freechildren (node.c:133)
==5591==    by 0x5230FD: PyNode_Free (node.c:123)
==5591==    by 0x4C7A76: PyRun_StringFlags (pythonrun.c:1455)
==5591==    by 0x4A3C18: PyEval_EvalFrameEx (ceval.c:4481)
==5591==    by 0x4A6BD0: PyEval_EvalCodeEx (ceval.c:3000)
==5591==    by 0x4A4D58: PyEval_EvalFrameEx (ceval.c:3846)
==5591==    by 0x4A6BD0: PyEval_EvalCodeEx (ceval.c:3000)
==5591==    by 0x4A4D58: PyEval_EvalFrameEx (ceval.c:3846)
==5591== 
==5591== Conditional jump or move depends on uninitialised value(s)
==5591==    at 0x4542FC: PyObject_Free (obmalloc.c:931)
==5591==    by 0x522F74: freechildren (node.c:135)
==5591==    by 0x522F3B: freechildren (node.c:133)
==5591==    by 0x522F3B: freechildren (node.c:133)
==5591==    by 0x522F3B: freechildren (node.c:133)
==5591==    by 0x5230FD: PyNode_Free (node.c:123)
==5591==    by 0x4C7A76: PyRun_StringFlags (pythonrun.c:1455)
==5591==    by 0x4A3C18: PyEval_EvalFrameEx (ceval.c:4481)
==5591==    by 0x4A6BD0: PyEval_EvalCodeEx (ceval.c:3000)
==5591==    by 0x4A4D58: PyEval_EvalFrameEx (ceval.c:3846)
==5591==    by 0x4A6BD0: PyEval_EvalCodeEx (ceval.c:3000)
==5591==    by 0x4A4D58: PyEval_EvalFrameEx (ceval.c:3846)
==5591== 
==5591== Use of uninitialised value of size 8
==5591==    at 0x454316: PyObject_Free (obmalloc.c:931)
==5591==    by 0x522F74: freechildren (node.c:135)
==5591==    by 0x522F3B: freechildren (node.c:133)
==5591==    by 0x522F3B: freechildren (node.c:133)
==5591==    by 0x522F3B: freechildren (node.c:133)
==5591==    by 0x5230FD: PyNode_Free (node.c:123)
==5591==    by 0x4C7A76: PyRun_StringFlags (pythonrun.c:1455)
==5591==    by 0x4A3C18: PyEval_EvalFrameEx (ceval.c:4481)
==5591==    by 0x4A6BD0: PyEval_EvalCodeEx (ceval.c:3000)
==5591==    by 0x4A4D58: PyEval_EvalFrameEx (ceval.c:3846)
==5591==    by 0x4A6BD0: PyEval_EvalCodeEx (ceval.c:3000)
==5591==    by 0x4A4D58: PyEval_EvalFrameEx (ceval.c:3846)
==5591== 
==5591== Conditional jump or move depends on uninitialised value(s)
==5591==    at 0x4542FC: PyObject_Free (obmalloc.c:931)
==5591==    by 0x522F9E: freechildren (node.c:135)
==5591==    by 0x5230FD: PyNode_Free (node.c:123)
==5591==    by 0x4C7A76: PyRun_StringFlags (pythonrun.c:1455)
==5591==    by 0x4A3C18: PyEval_EvalFrameEx (ceval.c:4481)
==5591==    by 0x4A6BD0: PyEval_EvalCodeEx (ceval.c:3000)
==5591==    by 0x4A4D58: PyEval_EvalFrameEx (ceval.c:3846)
==5591==    by 0x4A6BD0: PyEval_EvalCodeEx (ceval.c:3000)
==5591==    by 0x4A4D58: PyEval_EvalFrameEx (ceval.c:3846)
==5591==    by 0x4A5AAF: PyEval_EvalFrameEx (ceval.c:3836)
==5591==    by 0x4A5AAF: PyEval_EvalFrameEx (ceval.c:3836)
==5591==    by 0x4A6BD0: PyEval_EvalCodeEx (ceval.c:3000)
==5591== 
==5591== Use of uninitialised value of size 8
==5591==    at 0x454316: PyObject_Free (obmalloc.c:931)
==5591==    by 0x522F9E: freechildren (node.c:135)
==5591==    by 0x5230FD: PyNode_Free (node.c:123)
==5591==    by 0x4C7A76: PyRun_StringFlags (pythonrun.c:1455)
==5591==    by 0x4A3C18: PyEval_EvalFrameEx (ceval.c:4481)
==5591==    by 0x4A6BD0: PyEval_EvalCodeEx (ceval.c:3000)
==5591==    by 0x4A4D58: PyEval_EvalFrameEx (ceval.c:3846)
==5591==    by 0x4A6BD0: PyEval_EvalCodeEx (ceval.c:3000)
==5591==    by 0x4A4D58: PyEval_EvalFrameEx (ceval.c:3846)
==5591==    by 0x4A5AAF: PyEval_EvalFrameEx (ceval.c:3836)
==5591==    by 0x4A5AAF: PyEval_EvalFrameEx (ceval.c:3836)
==5591==    by 0x4A6BD0: PyEval_EvalCodeEx (ceval.c:3000)
==5591== 
==5591== Conditional jump or move depends on uninitialised value(s)
==5591==    at 0x4542FC: PyObject_Free (obmalloc.c:931)
==5591==    by 0x523055: freechildren (node.c:135)
==5591==    by 0x522F3B: freechildren (node.c:133)
==5591==    by 0x522F3B: freechildren (node.c:133)
==5591==    by 0x5230FD: PyNode_Free (node.c:123)
==5591==    by 0x4C7A76: PyRun_StringFlags (pythonrun.c:1455)
==5591==    by 0x4A3C18: PyEval_EvalFrameEx (ceval.c:4481)
==5591==    by 0x4A6BD0: PyEval_EvalCodeEx (ceval.c:3000)
==5591==    by 0x4A4D58: PyEval_EvalFrameEx (ceval.c:3846)
==5591==    by 0x4A6BD0: PyEval_EvalCodeEx (ceval.c:3000)
==5591==    by 0x4A4D58: PyEval_EvalFrameEx (ceval.c:3846)
==5591==    by 0x4A5AAF: PyEval_EvalFrameEx (ceval.c:3836)
==5591== 
==5591== Use of uninitialised value of size 8
==5591==    at 0x454316: PyObject_Free (obmalloc.c:931)
==5591==    by 0x523055: freechildren (node.c:135)
==5591==    by 0x522F3B: freechildren (node.c:133)
==5591==    by 0x522F3B: freechildren (node.c:133)
==5591==    by 0x5230FD: PyNode_Free (node.c:123)
==5591==    by 0x4C7A76: PyRun_StringFlags (pythonrun.c:1455)
==5591==    by 0x4A3C18: PyEval_EvalFrameEx (ceval.c:4481)
==5591==    by 0x4A6BD0: PyEval_EvalCodeEx (ceval.c:3000)
==5591==    by 0x4A4D58: PyEval_EvalFrameEx (ceval.c:3846)
==5591==    by 0x4A6BD0: PyEval_EvalCodeEx (ceval.c:3000)
==5591==    by 0x4A4D58: PyEval_EvalFrameEx (ceval.c:3846)
==5591==    by 0x4A5AAF: PyEval_EvalFrameEx (ceval.c:3836)
==5591== 
==5591== Invalid read of size 4
==5591==    at 0x4542F3: PyObject_Free (obmalloc.c:931)
==5591==    by 0x522FF7: freechildren (node.c:135)
==5591==    by 0x5230FD: PyNode_Free (node.c:123)
==5591==    by 0x4C6FF1: PyRun_FileExFlags (pythonrun.c:1483)
==5591==    by 0x4C7243: PyRun_SimpleFileExFlags (pythonrun.c:941)
==5591==    by 0x4180C0: Py_Main (main.c:577)
==5591==    by 0x5EEAD8D: (below main) (libc-start.c:226)
==5591==  Address 0x677b020 is 35,840 bytes inside a block of size 36,032 free'd
==5591==    at 0x4C283A2: free (vg_replace_malloc.c:366)
==5591==    by 0x4184A7: parsetok (parsetok.c:219)
==5591==    by 0x4C6FBE: PyRun_FileExFlags (pythonrun.c:1474)
==5591==    by 0x4C7243: PyRun_SimpleFileExFlags (pythonrun.c:941)
==5591==    by 0x4180C0: Py_Main (main.c:577)
==5591==    by 0x5EEAD8D: (below main) (libc-start.c:226)
==5591== 
==5591== Invalid read of size 4
==5591==    at 0x4542F3: PyObject_Free (obmalloc.c:931)
==5591==    by 0x523026: freechildren (node.c:135)
==5591==    by 0x5230FD: PyNode_Free (node.c:123)
==5591==    by 0x4C6FF1: PyRun_FileExFlags (pythonrun.c:1483)
==5591==    by 0x4C7243: PyRun_SimpleFileExFlags (pythonrun.c:941)
==5591==    by 0x4180C0: Py_Main (main.c:577)
==5591==    by 0x5EEAD8D: (below main) (libc-start.c:226)
==5591==  Address 0x677b020 is 35,840 bytes inside a block of size 36,032 free'd
==5591==    at 0x4C283A2: free (vg_replace_malloc.c:366)
==5591==    by 0x4184A7: parsetok (parsetok.c:219)
==5591==    by 0x4C6FBE: PyRun_FileExFlags (pythonrun.c:1474)
==5591==    by 0x4C7243: PyRun_SimpleFileExFlags (pythonrun.c:941)
==5591==    by 0x4180C0: Py_Main (main.c:577)
==5591==    by 0x5EEAD8D: (below main) (libc-start.c:226)
==5591== 
==5591== Invalid read of size 4
==5591==    at 0x4542F3: PyObject_Free (obmalloc.c:931)
==5591==    by 0x5230B3: freechildren (node.c:135)
==5591==    by 0x5230FD: PyNode_Free (node.c:123)
==5591==    by 0x4C6FF1: PyRun_FileExFlags (pythonrun.c:1483)
==5591==    by 0x4C7243: PyRun_SimpleFileExFlags (pythonrun.c:941)
==5591==    by 0x4180C0: Py_Main (main.c:577)
==5591==    by 0x5EEAD8D: (below main) (libc-start.c:226)
==5591==  Address 0x677b020 is 35,840 bytes inside a block of size 36,032 free'd
==5591==    at 0x4C283A2: free (vg_replace_malloc.c:366)
==5591==    by 0x4184A7: parsetok (parsetok.c:219)
==5591==    by 0x4C6FBE: PyRun_FileExFlags (pythonrun.c:1474)
==5591==    by 0x4C7243: PyRun_SimpleFileExFlags (pythonrun.c:941)
==5591==    by 0x4180C0: Py_Main (main.c:577)
==5591==    by 0x5EEAD8D: (below main) (libc-start.c:226)
==5591== 
==5591== Conditional jump or move depends on uninitialised value(s)
==5591==    at 0x4548C7: PyObject_Realloc (obmalloc.c:1154)
==5591==    by 0x4A7758: compiler_next_instr (compile.c:639)
==5591==    by 0x4A9380: compiler_nameop (compile.c:1026)
==5591==    by 0x4AE59C: compiler_visit_stmt (compile.c:2039)
==5591==    by 0x4B05CD: compiler_mod (compile.c:1187)
==5591==    by 0x4B0711: PyAST_Compile (compile.c:288)
==5591==    by 0x4C7013: PyRun_FileExFlags (pythonrun.c:1348)
==5591==    by 0x4C7243: PyRun_SimpleFileExFlags (pythonrun.c:941)
==5591==    by 0x4180C0: Py_Main (main.c:577)
==5591==    by 0x5EEAD8D: (below main) (libc-start.c:226)
==5591== 
==5591== Use of uninitialised value of size 8
==5591==    at 0x4548DA: PyObject_Realloc (obmalloc.c:1154)
==5591==    by 0x4A7758: compiler_next_instr (compile.c:639)
==5591==    by 0x4A9380: compiler_nameop (compile.c:1026)
==5591==    by 0x4AE59C: compiler_visit_stmt (compile.c:2039)
==5591==    by 0x4B05CD: compiler_mod (compile.c:1187)
==5591==    by 0x4B0711: PyAST_Compile (compile.c:288)
==5591==    by 0x4C7013: PyRun_FileExFlags (pythonrun.c:1348)
==5591==    by 0x4C7243: PyRun_SimpleFileExFlags (pythonrun.c:941)
==5591==    by 0x4180C0: Py_Main (main.c:577)
==5591==    by 0x5EEAD8D: (below main) (libc-start.c:226)
==5591== 
==5591== Conditional jump or move depends on uninitialised value(s)
==5591==    at 0x4548C7: PyObject_Realloc (obmalloc.c:1154)
==5591==    by 0x4A7758: compiler_next_instr (compile.c:639)
==5591==    by 0x4AE2CF: compiler_visit_stmt (compile.c:1026)
==5591==    by 0x4B05CD: compiler_mod (compile.c:1187)
==5591==    by 0x4B0711: PyAST_Compile (compile.c:288)
==5591==    by 0x4C7013: PyRun_FileExFlags (pythonrun.c:1348)
==5591==    by 0x4C7243: PyRun_SimpleFileExFlags (pythonrun.c:941)
==5591==    by 0x4180C0: Py_Main (main.c:577)
==5591==    by 0x5EEAD8D: (below main) (libc-start.c:226)
==5591== 
==5591== Use of uninitialised value of size 8
==5591==    at 0x4548DA: PyObject_Realloc (obmalloc.c:1154)
==5591==    by 0x4A7758: compiler_next_instr (compile.c:639)
==5591==    by 0x4AE2CF: compiler_visit_stmt (compile.c:1026)
==5591==    by 0x4B05CD: compiler_mod (compile.c:1187)
==5591==    by 0x4B0711: PyAST_Compile (compile.c:288)
==5591==    by 0x4C7013: PyRun_FileExFlags (pythonrun.c:1348)
==5591==    by 0x4C7243: PyRun_SimpleFileExFlags (pythonrun.c:941)
==5591==    by 0x4180C0: Py_Main (main.c:577)
==5591==    by 0x5EEAD8D: (below main) (libc-start.c:226)
==5591== 
==5591== Conditional jump or move depends on uninitialised value(s)
==5591==    at 0x4542FC: PyObject_Free (obmalloc.c:931)
==5591==    by 0x4B04E5: compiler_mod (compile.c:521)
==5591==    by 0x4B0711: PyAST_Compile (compile.c:288)
==5591==    by 0x4C7013: PyRun_FileExFlags (pythonrun.c:1348)
==5591==    by 0x4C7243: PyRun_SimpleFileExFlags (pythonrun.c:941)
==5591==    by 0x4180C0: Py_Main (main.c:577)
==5591==    by 0x5EEAD8D: (below main) (libc-start.c:226)
==5591== 
==5591== Use of uninitialised value of size 8
==5591==    at 0x454316: PyObject_Free (obmalloc.c:931)
==5591==    by 0x4B04E5: compiler_mod (compile.c:521)
==5591==    by 0x4B0711: PyAST_Compile (compile.c:288)
==5591==    by 0x4C7013: PyRun_FileExFlags (pythonrun.c:1348)
==5591==    by 0x4C7243: PyRun_SimpleFileExFlags (pythonrun.c:941)
==5591==    by 0x4180C0: Py_Main (main.c:577)
==5591==    by 0x5EEAD8D: (below main) (libc-start.c:226)
==5591== 
==5591== Invalid read of size 4
==5591==    at 0x4542F3: PyObject_Free (obmalloc.c:931)
==5591==    by 0x467E2C: tupledealloc (tupleobject.c:170)
==5591==    by 0x52D6F2: code_dealloc (codeobject.c:261)
==5591==    by 0x4BB4BD: load_source_module (import.c:1018)
==5591==    by 0x4BC2D2: import_submodule (import.c:2592)
==5591==    by 0x4BC54E: load_next (import.c:2412)
==5591==    by 0x4BCC10: import_module_level (import.c:2133)
==5591==    by 0x4BD133: PyImport_ImportModuleLevel (import.c:2185)
==5591==    by 0x49EBFA: builtin___import__ (bltinmodule.c:48)
==5591==    by 0x41C9D6: PyObject_Call (abstract.c:2492)
==5591==    by 0x49F7BE: T.420 (ceval.c:3619)
==5591==    by 0x4A2B4B: PyEval_EvalFrameEx (ceval.c:2159)
==5591==  Address 0x6918020 is 128 bytes inside a block of size 704 free'd
==5591==    at 0x4C29159: realloc (vg_replace_malloc.c:525)
==5591==    by 0x440C20: PyList_Append (listobject.c:62)
==5591==    by 0x4BEFA5: r_object (marshal.c:731)
==5591==    by 0x4BE6DC: r_object (marshal.c:794)
==5591==    by 0x4BE985: r_object (marshal.c:927)
==5591==    by 0x4C0D44: PyMarshal_ReadLastObjectFromFile (marshal.c:1107)
==5591==    by 0x4BB5B6: load_source_module (import.c:769)
==5591==    by 0x4BC2D2: import_submodule (import.c:2592)
==5591==    by 0x4BC54E: load_next (import.c:2412)
==5591==    by 0x4BCC10: import_module_level (import.c:2133)
==5591==    by 0x4BD133: PyImport_ImportModuleLevel (import.c:2185)
==5591==    by 0x49EBFA: builtin___import__ (bltinmodule.c:48)
==5591== 
==5591== Conditional jump or move depends on uninitialised value(s)
==5591==    at 0x4542FC: PyObject_Free (obmalloc.c:931)
==5591==    by 0x52D625: code_dealloc (codeobject.c:270)
==5591==    by 0x467E2C: tupledealloc (tupleobject.c:170)
==5591==    by 0x52D6F2: code_dealloc (codeobject.c:261)
==5591==    by 0x4BB4BD: load_source_module (import.c:1018)
==5591==    by 0x4BC2D2: import_submodule (import.c:2592)
==5591==    by 0x4BC54E: load_next (import.c:2412)
==5591==    by 0x4BCC10: import_module_level (import.c:2133)
==5591==    by 0x4BD133: PyImport_ImportModuleLevel (import.c:2185)
==5591==    by 0x49EBFA: builtin___import__ (bltinmodule.c:48)
==5591==    by 0x41C9D6: PyObject_Call (abstract.c:2492)
==5591==    by 0x49F7BE: T.420 (ceval.c:3619)
==5591== 
==5591== Use of uninitialised value of size 8
==5591==    at 0x454316: PyObject_Free (obmalloc.c:931)
==5591==    by 0x52D625: code_dealloc (codeobject.c:270)
==5591==    by 0x467E2C: tupledealloc (tupleobject.c:170)
==5591==    by 0x52D6F2: code_dealloc (codeobject.c:261)
==5591==    by 0x4BB4BD: load_source_module (import.c:1018)
==5591==    by 0x4BC2D2: import_submodule (import.c:2592)
==5591==    by 0x4BC54E: load_next (import.c:2412)
==5591==    by 0x4BCC10: import_module_level (import.c:2133)
==5591==    by 0x4BD133: PyImport_ImportModuleLevel (import.c:2185)
==5591==    by 0x49EBFA: builtin___import__ (bltinmodule.c:48)
==5591==    by 0x41C9D6: PyObject_Call (abstract.c:2492)
==5591==    by 0x49F7BE: T.420 (ceval.c:3619)
==5591== 
==5591== Invalid read of size 4
==5591==    at 0x4542F3: PyObject_Free (obmalloc.c:931)
==5591==    by 0x52D625: code_dealloc (codeobject.c:270)
==5591==    by 0x4BB4BD: load_source_module (import.c:1018)
==5591==    by 0x4BBCE2: load_package (import.c:1073)
==5591==    by 0x4BC2D2: import_submodule (import.c:2592)
==5591==    by 0x4BC54E: load_next (import.c:2412)
==5591==    by 0x4BCC57: import_module_level (import.c:2141)
==5591==    by 0x4BD133: PyImport_ImportModuleLevel (import.c:2185)
==5591==    by 0x49EBFA: builtin___import__ (bltinmodule.c:48)
==5591==    by 0x41C9D6: PyObject_Call (abstract.c:2492)
==5591==    by 0x49F7BE: T.420 (ceval.c:3619)
==5591==    by 0x4A2B4B: PyEval_EvalFrameEx (ceval.c:2159)
==5591==  Address 0x6936020 is not stack'd, malloc'd or (recently) free'd
==5591== 
==5591== Invalid read of size 4
==5591==    at 0x4548BE: PyObject_Realloc (obmalloc.c:1154)
==5591==    by 0x4D39C8: _PyObject_GC_Resize (gcmodule.c:1396)
==5591==    by 0x534D84: PyFrame_New (frameobject.c:655)
==5591==    by 0x4A5733: PyEval_EvalFrameEx (ceval.c:3825)
==5591==    by 0x4A5AAF: PyEval_EvalFrameEx (ceval.c:3836)
==5591==    by 0x4A5AAF: PyEval_EvalFrameEx (ceval.c:3836)
==5591==    by 0x4A6BD0: PyEval_EvalCodeEx (ceval.c:3000)
==5591==    by 0x4A4D58: PyEval_EvalFrameEx (ceval.c:3846)
==5591==    by 0x4A6BD0: PyEval_EvalCodeEx (ceval.c:3000)
==5591==    by 0x4A4D58: PyEval_EvalFrameEx (ceval.c:3846)
==5591==    by 0x4A6BD0: PyEval_EvalCodeEx (ceval.c:3000)
==5591==    by 0x4A4D58: PyEval_EvalFrameEx (ceval.c:3846)
==5591==  Address 0x693e020 is 16 bytes before a block of size 280 alloc'd
==5591==    at 0x4C29061: malloc (vg_replace_malloc.c:236)
==5591==    by 0x4D4F38: _PyObject_GC_New (gcmodule.c:1351)
==5591==    by 0x44D2D1: PyDict_New (dictobject.c:256)
==5591==    by 0x42542C: PyInstance_NewRaw (classobject.c:512)
==5591==    by 0x4279D2: PyInstance_New (classobject.c:548)
==5591==    by 0x41C9D6: PyObject_Call (abstract.c:2492)
==5591==    by 0x4A48C7: PyEval_EvalFrameEx (ceval.c:3968)
==5591==    by 0x4A6BD0: PyEval_EvalCodeEx (ceval.c:3000)
==5591==    by 0x4A4D58: PyEval_EvalFrameEx (ceval.c:3846)
==5591==    by 0x4A6BD0: PyEval_EvalCodeEx (ceval.c:3000)
==5591==    by 0x4A4D58: PyEval_EvalFrameEx (ceval.c:3846)
==5591==    by 0x4A6BD0: PyEval_EvalCodeEx (ceval.c:3000)
==5591== 
==5591== Invalid read of size 4
==5591==    at 0x4542F3: PyObject_Free (obmalloc.c:931)
==5591==    by 0x523026: freechildren (node.c:135)
==5591==    by 0x5230FD: PyNode_Free (node.c:123)
==5591==    by 0x4C7A76: PyRun_StringFlags (pythonrun.c:1455)
==5591==    by 0x504717: initcPickle (cPickle.c:5778)
==5591==    by 0x4BAB69: init_builtin (import.c:1914)
==5591==    by 0x4BBABB: load_module (import.c:1843)
==5591==    by 0x4BC2D2: import_submodule (import.c:2592)
==5591==    by 0x4BC593: load_next (import.c:2416)
==5591==    by 0x4BCC10: import_module_level (import.c:2133)
==5591==    by 0x4BD133: PyImport_ImportModuleLevel (import.c:2185)
==5591==    by 0x49EBFA: builtin___import__ (bltinmodule.c:48)
==5591==  Address 0x92da020 is 33,184 bytes inside a block of size 36,032 free'd
==5591==    at 0x4C283A2: free (vg_replace_malloc.c:366)
==5591==    by 0x4184A7: parsetok (parsetok.c:219)
==5591==    by 0x4C7B39: PyRun_StringFlags (pythonrun.c:1445)
==5591==    by 0x504717: initcPickle (cPickle.c:5778)
==5591==    by 0x4BAB69: init_builtin (import.c:1914)
==5591==    by 0x4BBABB: load_module (import.c:1843)
==5591==    by 0x4BC2D2: import_submodule (import.c:2592)
==5591==    by 0x4BC593: load_next (import.c:2416)
==5591==    by 0x4BCC10: import_module_level (import.c:2133)
==5591==    by 0x4BD133: PyImport_ImportModuleLevel (import.c:2185)
==5591==    by 0x49EBFA: builtin___import__ (bltinmodule.c:48)
==5591==    by 0x41C9D6: PyObject_Call (abstract.c:2492)
==5591== 
==5591== Invalid read of size 4
==5591==    at 0x4542F3: PyObject_Free (obmalloc.c:931)
==5591==    by 0x52D625: code_dealloc (codeobject.c:270)
==5591==    by 0x4C7B05: PyRun_StringFlags (pythonrun.c:1352)
==5591==    by 0x504717: initcPickle (cPickle.c:5778)
==5591==    by 0x4BAB69: init_builtin (import.c:1914)
==5591==    by 0x4BBABB: load_module (import.c:1843)
==5591==    by 0x4BC2D2: import_submodule (import.c:2592)
==5591==    by 0x4BC593: load_next (import.c:2416)
==5591==    by 0x4BCC10: import_module_level (import.c:2133)
==5591==    by 0x4BD133: PyImport_ImportModuleLevel (import.c:2185)
==5591==    by 0x49EBFA: builtin___import__ (bltinmodule.c:48)
==5591==    by 0x41C9D6: PyObject_Call (abstract.c:2492)
==5591==  Address 0x92dd020 is 368 bytes inside a block of size 384 free'd
==5591==    at 0x4C283A2: free (vg_replace_malloc.c:366)
==5591==    by 0x4A70BD: compiler_unit_free (compile.c:423)
==5591==    by 0x4A7D65: compiler_exit_scope (compile.c:521)
==5591==    by 0x4AF3FF: compiler_function (compile.c:1401)
==5591==    by 0x4B05CD: compiler_mod (compile.c:1187)
==5591==    by 0x4B0711: PyAST_Compile (compile.c:288)
==5591==    by 0x4C7A90: PyRun_StringFlags (pythonrun.c:1348)
==5591==    by 0x504717: initcPickle (cPickle.c:5778)
==5591==    by 0x4BAB69: init_builtin (import.c:1914)
==5591==    by 0x4BBABB: load_module (import.c:1843)
==5591==    by 0x4BC2D2: import_submodule (import.c:2592)
==5591==    by 0x4BC593: load_next (import.c:2416)
==5591== 
==5591== Invalid read of size 4
==5591==    at 0x4542F3: PyObject_Free (obmalloc.c:931)
==5591==    by 0x522FF7: freechildren (node.c:135)
==5591==    by 0x5230FD: PyNode_Free (node.c:123)
==5591==    by 0x4C7A76: PyRun_StringFlags (pythonrun.c:1455)
==5591==    by 0x5047BD: initcPickle (cPickle.c:5797)
==5591==    by 0x4BAB69: init_builtin (import.c:1914)
==5591==    by 0x4BBABB: load_module (import.c:1843)
==5591==    by 0x4BC2D2: import_submodule (import.c:2592)
==5591==    by 0x4BC593: load_next (import.c:2416)
==5591==    by 0x4BCC10: import_module_level (import.c:2133)
==5591==    by 0x4BD133: PyImport_ImportModuleLevel (import.c:2185)
==5591==    by 0x49EBFA: builtin___import__ (bltinmodule.c:48)
==5591==  Address 0x92e9020 is 33,040 bytes inside a block of size 36,032 free'd
==5591==    at 0x4C283A2: free (vg_replace_malloc.c:366)
==5591==    by 0x4184A7: parsetok (parsetok.c:219)
==5591==    by 0x4C7B39: PyRun_StringFlags (pythonrun.c:1445)
==5591==    by 0x5047BD: initcPickle (cPickle.c:5797)
==5591==    by 0x4BAB69: init_builtin (import.c:1914)
==5591==    by 0x4BBABB: load_module (import.c:1843)
==5591==    by 0x4BC2D2: import_submodule (import.c:2592)
==5591==    by 0x4BC593: load_next (import.c:2416)
==5591==    by 0x4BCC10: import_module_level (import.c:2133)
==5591==    by 0x4BD133: PyImport_ImportModuleLevel (import.c:2185)
==5591==    by 0x49EBFA: builtin___import__ (bltinmodule.c:48)
==5591==    by 0x41C9D6: PyObject_Call (abstract.c:2492)
==5591== 
==5591== Invalid read of size 4
==5591==    at 0x4542F3: PyObject_Free (obmalloc.c:931)
==5591==    by 0x52D625: code_dealloc (codeobject.c:270)
==5591==    by 0x4C7B05: PyRun_StringFlags (pythonrun.c:1352)
==5591==    by 0x5047BD: initcPickle (cPickle.c:5797)
==5591==    by 0x4BAB69: init_builtin (import.c:1914)
==5591==    by 0x4BBABB: load_module (import.c:1843)
==5591==    by 0x4BC2D2: import_submodule (import.c:2592)
==5591==    by 0x4BC593: load_next (import.c:2416)
==5591==    by 0x4BCC10: import_module_level (import.c:2133)
==5591==    by 0x4BD133: PyImport_ImportModuleLevel (import.c:2185)
==5591==    by 0x49EBFA: builtin___import__ (bltinmodule.c:48)
==5591==    by 0x41C9D6: PyObject_Call (abstract.c:2492)
==5591==  Address 0x92eb020 is 144 bytes inside a block of size 188 free'd
==5591==    at 0x4C283A2: free (vg_replace_malloc.c:366)
==5591==    by 0x4C3129: PyCode_Optimize (peephole.c:616)
==5591==    by 0x4AA2D5: assemble (compile.c:3874)
==5591==    by 0x4AF3F4: compiler_function (compile.c:1400)
==5591==    by 0x4B05CD: compiler_mod (compile.c:1187)
==5591==    by 0x4B0711: PyAST_Compile (compile.c:288)
==5591==    by 0x4C7A90: PyRun_StringFlags (pythonrun.c:1348)
==5591==    by 0x5047BD: initcPickle (cPickle.c:5797)
==5591==    by 0x4BAB69: init_builtin (import.c:1914)
==5591==    by 0x4BBABB: load_module (import.c:1843)
==5591==    by 0x4BC2D2: import_submodule (import.c:2592)
==5591==    by 0x4BC593: load_next (import.c:2416)
==5591== 
==5591== Invalid read of size 4
==5591==    at 0x4542F3: PyObject_Free (obmalloc.c:931)
==5591==    by 0x467E2C: tupledealloc (tupleobject.c:170)
==5591==    by 0x52D6F2: code_dealloc (codeobject.c:261)
==5591==    by 0x467E2C: tupledealloc (tupleobject.c:170)
==5591==    by 0x52D6F2: code_dealloc (codeobject.c:261)
==5591==    by 0x4BB4BD: load_source_module (import.c:1018)
==5591==    by 0x4BC2D2: import_submodule (import.c:2592)
==5591==    by 0x4BC54E: load_next (import.c:2412)
==5591==    by 0x4BCC10: import_module_level (import.c:2133)
==5591==    by 0x4BD133: PyImport_ImportModuleLevel (import.c:2185)
==5591==    by 0x49EBFA: builtin___import__ (bltinmodule.c:48)
==5591==    by 0x41C9D6: PyObject_Call (abstract.c:2492)
==5591==  Address 0x932e020 is not stack'd, malloc'd or (recently) free'd
==5591== 
==5591== Invalid read of size 4
==5591==    at 0x4548BE: PyObject_Realloc (obmalloc.c:1154)
==5591==    by 0x45E346: _PyString_Resize (stringobject.c:4277)
==5591==    by 0x4651D5: PyString_Format (stringobject.c:5064)
==5591==    by 0x41D06E: binary_op1 (abstract.c:917)
==5591==    by 0x41FE2D: PyNumber_Remainder (abstract.c:969)
==5591==    by 0x4A2663: PyEval_EvalFrameEx (ceval.c:1180)
==5591==    by 0x4A5AAF: PyEval_EvalFrameEx (ceval.c:3836)
==5591==    by 0x4A6BD0: PyEval_EvalCodeEx (ceval.c:3000)
==5591==    by 0x4A4D58: PyEval_EvalFrameEx (ceval.c:3846)
==5591==    by 0x4A6BD0: PyEval_EvalCodeEx (ceval.c:3000)
==5591==    by 0x4A6CA1: PyEval_EvalCode (ceval.c:541)
==5591==    by 0x4B952D: PyImport_ExecCodeModuleEx (import.c:677)
==5591==  Address 0xa073020 is 736 bytes inside a block of size 1,039 free'd
==5591==    at 0x4C29159: realloc (vg_replace_malloc.c:525)
==5591==    by 0x45E346: _PyString_Resize (stringobject.c:4277)
==5591==    by 0x4651D5: PyString_Format (stringobject.c:5064)
==5591==    by 0x41D06E: binary_op1 (abstract.c:917)
==5591==    by 0x41FE2D: PyNumber_Remainder (abstract.c:969)
==5591==    by 0x4A2663: PyEval_EvalFrameEx (ceval.c:1180)
==5591==    by 0x4A5AAF: PyEval_EvalFrameEx (ceval.c:3836)
==5591==    by 0x4A6BD0: PyEval_EvalCodeEx (ceval.c:3000)
==5591==    by 0x4A4D58: PyEval_EvalFrameEx (ceval.c:3846)
==5591==    by 0x4A6BD0: PyEval_EvalCodeEx (ceval.c:3000)
==5591==    by 0x4A6CA1: PyEval_EvalCode (ceval.c:541)
==5591==    by 0x4B952D: PyImport_ExecCodeModuleEx (import.c:677)
==5591== 
==5591== Invalid read of size 4
==5591==    at 0x4542F3: PyObject_Free (obmalloc.c:931)
==5591==    by 0x534242: frame_dealloc (frameobject.c:420)
==5591==    by 0x4A5AD5: PyEval_EvalFrameEx (ceval.c:3838)
==5591==    by 0x4A6BD0: PyEval_EvalCodeEx (ceval.c:3000)
==5591==    by 0x4A4D58: PyEval_EvalFrameEx (ceval.c:3846)
==5591==    by 0x4A6BD0: PyEval_EvalCodeEx (ceval.c:3000)
==5591==    by 0x4A6CA1: PyEval_EvalCode (ceval.c:541)
==5591==    by 0x4B952D: PyImport_ExecCodeModuleEx (import.c:677)
==5591==    by 0x4BB49D: load_source_module (import.c:1017)
==5591==    by 0x4BC2D2: import_submodule (import.c:2592)
==5591==    by 0x4BC54E: load_next (import.c:2412)
==5591==    by 0x4BCC10: import_module_level (import.c:2133)
==5591==  Address 0xa073020 is 736 bytes inside a block of size 1,039 free'd
==5591==    at 0x4C29159: realloc (vg_replace_malloc.c:525)
==5591==    by 0x45E346: _PyString_Resize (stringobject.c:4277)
==5591==    by 0x4651D5: PyString_Format (stringobject.c:5064)
==5591==    by 0x41D06E: binary_op1 (abstract.c:917)
==5591==    by 0x41FE2D: PyNumber_Remainder (abstract.c:969)
==5591==    by 0x4A2663: PyEval_EvalFrameEx (ceval.c:1180)
==5591==    by 0x4A5AAF: PyEval_EvalFrameEx (ceval.c:3836)
==5591==    by 0x4A6BD0: PyEval_EvalCodeEx (ceval.c:3000)
==5591==    by 0x4A4D58: PyEval_EvalFrameEx (ceval.c:3846)
==5591==    by 0x4A6BD0: PyEval_EvalCodeEx (ceval.c:3000)
==5591==    by 0x4A6CA1: PyEval_EvalCode (ceval.c:541)
==5591==    by 0x4B952D: PyImport_ExecCodeModuleEx (import.c:677)
==5591== 
==5591== Conditional jump or move depends on uninitialised value(s)
==5591==    at 0x4542FC: PyObject_Free (obmalloc.c:931)
==5591==    by 0x52D625: code_dealloc (codeobject.c:270)
==5591==    by 0x4BB4BD: load_source_module (import.c:1018)
==5591==    by 0x4BC2D2: import_submodule (import.c:2592)
==5591==    by 0x4BC54E: load_next (import.c:2412)
==5591==    by 0x4BCC10: import_module_level (import.c:2133)
==5591==    by 0x4BD133: PyImport_ImportModuleLevel (import.c:2185)
==5591==    by 0x49EBFA: builtin___import__ (bltinmodule.c:48)
==5591==    by 0x41C9D6: PyObject_Call (abstract.c:2492)
==5591==    by 0x49F7BE: T.420 (ceval.c:3619)
==5591==    by 0x4A2B4B: PyEval_EvalFrameEx (ceval.c:2159)
==5591==    by 0x4A6BD0: PyEval_EvalCodeEx (ceval.c:3000)
==5591== 
==5591== Use of uninitialised value of size 8
==5591==    at 0x454316: PyObject_Free (obmalloc.c:931)
==5591==    by 0x52D625: code_dealloc (codeobject.c:270)
==5591==    by 0x4BB4BD: load_source_module (import.c:1018)
==5591==    by 0x4BC2D2: import_submodule (import.c:2592)
==5591==    by 0x4BC54E: load_next (import.c:2412)
==5591==    by 0x4BCC10: import_module_level (import.c:2133)
==5591==    by 0x4BD133: PyImport_ImportModuleLevel (import.c:2185)
==5591==    by 0x49EBFA: builtin___import__ (bltinmodule.c:48)
==5591==    by 0x41C9D6: PyObject_Call (abstract.c:2492)
==5591==    by 0x49F7BE: T.420 (ceval.c:3619)
==5591==    by 0x4A2B4B: PyEval_EvalFrameEx (ceval.c:2159)
==5591==    by 0x4A6BD0: PyEval_EvalCodeEx (ceval.c:3000)
==5591== 
==5591== Invalid read of size 4
==5591==    at 0x4542F3: PyObject_Free (obmalloc.c:931)
==5591==    by 0x52D63A: code_dealloc (codeobject.c:260)
==5591==    by 0x4BB4BD: load_source_module (import.c:1018)
==5591==    by 0x4BBCE2: load_package (import.c:1073)
==5591==    by 0x4BC2D2: import_submodule (import.c:2592)
==5591==    by 0x4BC54E: load_next (import.c:2412)
==5591==    by 0x4BCC57: import_module_level (import.c:2141)
==5591==    by 0x4BD133: PyImport_ImportModuleLevel (import.c:2185)
==5591==    by 0x49EBFA: builtin___import__ (bltinmodule.c:48)
==5591==    by 0x41C9D6: PyObject_Call (abstract.c:2492)
==5591==    by 0x49F7BE: T.420 (ceval.c:3619)
==5591==    by 0x4A2B4B: PyEval_EvalFrameEx (ceval.c:2159)
==5591==  Address 0x689f020 is 0 bytes after a block of size 112 alloc'd
==5591==    at 0x4C29061: malloc (vg_replace_malloc.c:236)
==5591==    by 0x43F3EA: PyList_New (listobject.c:152)
==5591==    by 0x4A1B8D: PyEval_EvalFrameEx (ceval.c:2050)
==5591==    by 0x4A6BD0: PyEval_EvalCodeEx (ceval.c:3000)
==5591==    by 0x4A6CA1: PyEval_EvalCode (ceval.c:541)
==5591==    by 0x4B952D: PyImport_ExecCodeModuleEx (import.c:677)
==5591==    by 0x4BB49D: load_source_module (import.c:1017)
==5591==    by 0x4BC2D2: import_submodule (import.c:2592)
==5591==    by 0x4BC54E: load_next (import.c:2412)
==5591==    by 0x4BCC10: import_module_level (import.c:2133)
==5591==    by 0x4BD133: PyImport_ImportModuleLevel (import.c:2185)
==5591==    by 0x49EBFA: builtin___import__ (bltinmodule.c:48)
==5591== 
==5591== Invalid read of size 4
==5591==    at 0x4542F3: PyObject_Free (obmalloc.c:931)
==5591==    by 0x467E45: tupledealloc (tupleobject.c:183)
==5591==    by 0x52D6F2: code_dealloc (codeobject.c:261)
==5591==    by 0x467E2C: tupledealloc (tupleobject.c:170)
==5591==    by 0x52D6F2: code_dealloc (codeobject.c:261)
==5591==    by 0x4BB4BD: load_source_module (import.c:1018)
==5591==    by 0x4BC2D2: import_submodule (import.c:2592)
==5591==    by 0x4BC54E: load_next (import.c:2412)
==5591==    by 0x4BCC10: import_module_level (import.c:2133)
==5591==    by 0x4BD133: PyImport_ImportModuleLevel (import.c:2185)
==5591==    by 0x49EBFA: builtin___import__ (bltinmodule.c:48)
==5591==    by 0x41C9D6: PyObject_Call (abstract.c:2492)
==5591==  Address 0xa155020 is 32 bytes inside a block of size 336 free'd
==5591==    at 0x4C283A2: free (vg_replace_malloc.c:366)
==5591==    by 0x52D63A: code_dealloc (codeobject.c:260)
==5591==    by 0x467E2C: tupledealloc (tupleobject.c:170)
==5591==    by 0x52D6F2: code_dealloc (codeobject.c:261)
==5591==    by 0x4BB4BD: load_source_module (import.c:1018)
==5591==    by 0x4BC2D2: import_submodule (import.c:2592)
==5591==    by 0x4BC54E: load_next (import.c:2412)
==5591==    by 0x4BCC10: import_module_level (import.c:2133)
==5591==    by 0x4BD133: PyImport_ImportModuleLevel (import.c:2185)
==5591==    by 0x49EBFA: builtin___import__ (bltinmodule.c:48)
==5591==    by 0x41C9D6: PyObject_Call (abstract.c:2492)
==5591==    by 0x49F7BE: T.420 (ceval.c:3619)
==5591== 
==5591== Conditional jump or move depends on uninitialised value(s)
==5591==    at 0x4542FC: PyObject_Free (obmalloc.c:931)
==5591==    by 0x4A70BD: compiler_unit_free (compile.c:423)
==5591==    by 0x4B04E5: compiler_mod (compile.c:521)
==5591==    by 0x4B0711: PyAST_Compile (compile.c:288)
==5591==    by 0x4C7A90: PyRun_StringFlags (pythonrun.c:1348)
==5591==    by 0x4A3C18: PyEval_EvalFrameEx (ceval.c:4481)
==5591==    by 0x4A6BD0: PyEval_EvalCodeEx (ceval.c:3000)
==5591==    by 0x4A4D58: PyEval_EvalFrameEx (ceval.c:3846)
==5591==    by 0x4A6BD0: PyEval_EvalCodeEx (ceval.c:3000)
==5591==    by 0x4A6CA1: PyEval_EvalCode (ceval.c:541)
==5591==    by 0x4B952D: PyImport_ExecCodeModuleEx (import.c:677)
==5591==    by 0x4BB49D: load_source_module (import.c:1017)
==5591== 
==5591== Use of uninitialised value of size 8
==5591==    at 0x454316: PyObject_Free (obmalloc.c:931)
==5591==    by 0x4A70BD: compiler_unit_free (compile.c:423)
==5591==    by 0x4B04E5: compiler_mod (compile.c:521)
==5591==    by 0x4B0711: PyAST_Compile (compile.c:288)
==5591==    by 0x4C7A90: PyRun_StringFlags (pythonrun.c:1348)
==5591==    by 0x4A3C18: PyEval_EvalFrameEx (ceval.c:4481)
==5591==    by 0x4A6BD0: PyEval_EvalCodeEx (ceval.c:3000)
==5591==    by 0x4A4D58: PyEval_EvalFrameEx (ceval.c:3846)
==5591==    by 0x4A6BD0: PyEval_EvalCodeEx (ceval.c:3000)
==5591==    by 0x4A6CA1: PyEval_EvalCode (ceval.c:541)
==5591==    by 0x4B952D: PyImport_ExecCodeModuleEx (import.c:677)
==5591==    by 0x4BB49D: load_source_module (import.c:1017)
==5591== 
==5591== Conditional jump or move depends on uninitialised value(s)
==5591==    at 0x4542FC: PyObject_Free (obmalloc.c:931)
==5591==    by 0x4B04E5: compiler_mod (compile.c:521)
==5591==    by 0x4B0711: PyAST_Compile (compile.c:288)
==5591==    by 0x4C7A90: PyRun_StringFlags (pythonrun.c:1348)
==5591==    by 0x4A3C18: PyEval_EvalFrameEx (ceval.c:4481)
==5591==    by 0x4A6BD0: PyEval_EvalCodeEx (ceval.c:3000)
==5591==    by 0x4A4D58: PyEval_EvalFrameEx (ceval.c:3846)
==5591==    by 0x4A6BD0: PyEval_EvalCodeEx (ceval.c:3000)
==5591==    by 0x4A6CA1: PyEval_EvalCode (ceval.c:541)
==5591==    by 0x4B952D: PyImport_ExecCodeModuleEx (import.c:677)
==5591==    by 0x4BB49D: load_source_module (import.c:1017)
==5591==    by 0x4BC2D2: import_submodule (import.c:2592)
==5591== 
==5591== Use of uninitialised value of size 8
==5591==    at 0x454316: PyObject_Free (obmalloc.c:931)
==5591==    by 0x4B04E5: compiler_mod (compile.c:521)
==5591==    by 0x4B0711: PyAST_Compile (compile.c:288)
==5591==    by 0x4C7A90: PyRun_StringFlags (pythonrun.c:1348)
==5591==    by 0x4A3C18: PyEval_EvalFrameEx (ceval.c:4481)
==5591==    by 0x4A6BD0: PyEval_EvalCodeEx (ceval.c:3000)
==5591==    by 0x4A4D58: PyEval_EvalFrameEx (ceval.c:3846)
==5591==    by 0x4A6BD0: PyEval_EvalCodeEx (ceval.c:3000)
==5591==    by 0x4A6CA1: PyEval_EvalCode (ceval.c:541)
==5591==    by 0x4B952D: PyImport_ExecCodeModuleEx (import.c:677)
==5591==    by 0x4BB49D: load_source_module (import.c:1017)
==5591==    by 0x4BC2D2: import_submodule (import.c:2592)
==5591== 
==5591== Conditional jump or move depends on uninitialised value(s)
==5591==    at 0x4542FC: PyObject_Free (obmalloc.c:931)
==5591==    by 0x52D625: code_dealloc (codeobject.c:270)
==5591==    by 0x4C7B05: PyRun_StringFlags (pythonrun.c:1352)
==5591==    by 0x4A3C18: PyEval_EvalFrameEx (ceval.c:4481)
==5591==    by 0x4A6BD0: PyEval_EvalCodeEx (ceval.c:3000)
==5591==    by 0x4A4D58: PyEval_EvalFrameEx (ceval.c:3846)
==5591==    by 0x4A6BD0: PyEval_EvalCodeEx (ceval.c:3000)
==5591==    by 0x4A6CA1: PyEval_EvalCode (ceval.c:541)
==5591==    by 0x4B952D: PyImport_ExecCodeModuleEx (import.c:677)
==5591==    by 0x4BB49D: load_source_module (import.c:1017)
==5591==    by 0x4BC2D2: import_submodule (import.c:2592)
==5591==    by 0x4BC54E: load_next (import.c:2412)
==5591== 
==5591== Use of uninitialised value of size 8
==5591==    at 0x454316: PyObject_Free (obmalloc.c:931)
==5591==    by 0x52D625: code_dealloc (codeobject.c:270)
==5591==    by 0x4C7B05: PyRun_StringFlags (pythonrun.c:1352)
==5591==    by 0x4A3C18: PyEval_EvalFrameEx (ceval.c:4481)
==5591==    by 0x4A6BD0: PyEval_EvalCodeEx (ceval.c:3000)
==5591==    by 0x4A4D58: PyEval_EvalFrameEx (ceval.c:3846)
==5591==    by 0x4A6BD0: PyEval_EvalCodeEx (ceval.c:3000)
==5591==    by 0x4A6CA1: PyEval_EvalCode (ceval.c:541)
==5591==    by 0x4B952D: PyImport_ExecCodeModuleEx (import.c:677)
==5591==    by 0x4BB49D: load_source_module (import.c:1017)
==5591==    by 0x4BC2D2: import_submodule (import.c:2592)
==5591==    by 0x4BC54E: load_next (import.c:2412)
==5591== 
==5591== Invalid read of size 4
==5591==    at 0x4542F3: PyObject_Free (obmalloc.c:931)
==5591==    by 0x467E2C: tupledealloc (tupleobject.c:170)
==5591==    by 0x4A5445: PyEval_EvalFrameEx (ceval.c:3752)
==5591==    by 0x4A6BD0: PyEval_EvalCodeEx (ceval.c:3000)
==5591==    by 0x4A4D58: PyEval_EvalFrameEx (ceval.c:3846)
==5591==    by 0x4A6BD0: PyEval_EvalCodeEx (ceval.c:3000)
==5591==    by 0x4A6CA1: PyEval_EvalCode (ceval.c:541)
==5591==    by 0x4B952D: PyImport_ExecCodeModuleEx (import.c:677)
==5591==    by 0x4BB49D: load_source_module (import.c:1017)
==5591==    by 0x4BC2D2: import_submodule (import.c:2592)
==5591==    by 0x4BC54E: load_next (import.c:2412)
==5591==    by 0x4BCC10: import_module_level (import.c:2133)
==5591==  Address 0x632f020 is 320 bytes inside a block of size 432 free'd
==5591==    at 0x4C283A2: free (vg_replace_malloc.c:366)
==5591==    by 0x4B04E5: compiler_mod (compile.c:521)
==5591==    by 0x4B0711: PyAST_Compile (compile.c:288)
==5591==    by 0x4C7A90: PyRun_StringFlags (pythonrun.c:1348)
==5591==    by 0x4A3C18: PyEval_EvalFrameEx (ceval.c:4481)
==5591==    by 0x4A6BD0: PyEval_EvalCodeEx (ceval.c:3000)
==5591==    by 0x4A4D58: PyEval_EvalFrameEx (ceval.c:3846)
==5591==    by 0x4A6BD0: PyEval_EvalCodeEx (ceval.c:3000)
==5591==    by 0x4A6CA1: PyEval_EvalCode (ceval.c:541)
==5591==    by 0x4B952D: PyImport_ExecCodeModuleEx (import.c:677)
==5591==    by 0x4BB49D: load_source_module (import.c:1017)
==5591==    by 0x4BC2D2: import_submodule (import.c:2592)
==5591== 
==5591== Conditional jump or move depends on uninitialised value(s)
==5591==    at 0x4542FC: PyObject_Free (obmalloc.c:931)
==5591==    by 0x467E2C: tupledealloc (tupleobject.c:170)
==5591==    by 0x467E2C: tupledealloc (tupleobject.c:170)
==5591==    by 0x52D6F2: code_dealloc (codeobject.c:261)
==5591==    by 0x4BB4BD: load_source_module (import.c:1018)
==5591==    by 0x4BC2D2: import_submodule (import.c:2592)
==5591==    by 0x4BC54E: load_next (import.c:2412)
==5591==    by 0x4BCC10: import_module_level (import.c:2133)
==5591==    by 0x4BD133: PyImport_ImportModuleLevel (import.c:2185)
==5591==    by 0x49EBFA: builtin___import__ (bltinmodule.c:48)
==5591==    by 0x41C9D6: PyObject_Call (abstract.c:2492)
==5591==    by 0x49F7BE: T.420 (ceval.c:3619)
==5591== 
==5591== Use of uninitialised value of size 8
==5591==    at 0x454316: PyObject_Free (obmalloc.c:931)
==5591==    by 0x467E2C: tupledealloc (tupleobject.c:170)
==5591==    by 0x467E2C: tupledealloc (tupleobject.c:170)
==5591==    by 0x52D6F2: code_dealloc (codeobject.c:261)
==5591==    by 0x4BB4BD: load_source_module (import.c:1018)
==5591==    by 0x4BC2D2: import_submodule (import.c:2592)
==5591==    by 0x4BC54E: load_next (import.c:2412)
==5591==    by 0x4BCC10: import_module_level (import.c:2133)
==5591==    by 0x4BD133: PyImport_ImportModuleLevel (import.c:2185)
==5591==    by 0x49EBFA: builtin___import__ (bltinmodule.c:48)
==5591==    by 0x41C9D6: PyObject_Call (abstract.c:2492)
==5591==    by 0x49F7BE: T.420 (ceval.c:3619)
==5591== 
==5591== Invalid read of size 4
==5591==    at 0x4542F3: PyObject_Free (obmalloc.c:931)
==5591==    by 0x467E2C: tupledealloc (tupleobject.c:170)
==5591==    by 0x467E2C: tupledealloc (tupleobject.c:170)
==5591==    by 0x52D6F2: code_dealloc (codeobject.c:261)
==5591==    by 0x4BB4BD: load_source_module (import.c:1018)
==5591==    by 0x4BC2D2: import_submodule (import.c:2592)
==5591==    by 0x4BC54E: load_next (import.c:2412)
==5591==    by 0x4BCC10: import_module_level (import.c:2133)
==5591==    by 0x4BD133: PyImport_ImportModuleLevel (import.c:2185)
==5591==    by 0x49EBFA: builtin___import__ (bltinmodule.c:48)
==5591==    by 0x41C9D6: PyObject_Call (abstract.c:2492)
==5591==    by 0x49F7BE: T.420 (ceval.c:3619)
==5591==  Address 0x683d020 is not stack'd, malloc'd or (recently) free'd
==5591== 
==5591== Conditional jump or move depends on uninitialised value(s)
==5591==    at 0x4542FC: PyObject_Free (obmalloc.c:931)
==5591==    by 0x467E2C: tupledealloc (tupleobject.c:170)
==5591==    by 0x52D6F2: code_dealloc (codeobject.c:261)
==5591==    by 0x4BB4BD: load_source_module (import.c:1018)
==5591==    by 0x4BC2D2: import_submodule (import.c:2592)
==5591==    by 0x4BC54E: load_next (import.c:2412)
==5591==    by 0x4BCC10: import_module_level (import.c:2133)
==5591==    by 0x4BD133: PyImport_ImportModuleLevel (import.c:2185)
==5591==    by 0x49EBFA: builtin___import__ (bltinmodule.c:48)
==5591==    by 0x41C9D6: PyObject_Call (abstract.c:2492)
==5591==    by 0x49F7BE: T.420 (ceval.c:3619)
==5591==    by 0x4A2B4B: PyEval_EvalFrameEx (ceval.c:2159)
==5591== 
==5591== Use of uninitialised value of size 8
==5591==    at 0x454316: PyObject_Free (obmalloc.c:931)
==5591==    by 0x467E2C: tupledealloc (tupleobject.c:170)
==5591==    by 0x52D6F2: code_dealloc (codeobject.c:261)
==5591==    by 0x4BB4BD: load_source_module (import.c:1018)
==5591==    by 0x4BC2D2: import_submodule (import.c:2592)
==5591==    by 0x4BC54E: load_next (import.c:2412)
==5591==    by 0x4BCC10: import_module_level (import.c:2133)
==5591==    by 0x4BD133: PyImport_ImportModuleLevel (import.c:2185)
==5591==    by 0x49EBFA: builtin___import__ (bltinmodule.c:48)
==5591==    by 0x41C9D6: PyObject_Call (abstract.c:2492)
==5591==    by 0x49F7BE: T.420 (ceval.c:3619)
==5591==    by 0x4A2B4B: PyEval_EvalFrameEx (ceval.c:2159)
==5591== 
==5591== Invalid read of size 4
==5591==    at 0x4542F3: PyObject_Free (obmalloc.c:931)
==5591==    by 0x43F2C1: list_dealloc (listobject.c:306)
==5591==    by 0x467E2C: tupledealloc (tupleobject.c:170)
==5591==    by 0x51DF31: pattern_subx (_sre.c:2024)
==5591==    by 0x51E0CC: pattern_sub (_sre.c:2474)
==5591==    by 0x4A51AD: PyEval_EvalFrameEx (ceval.c:3750)
==5591==    by 0x4A6BD0: PyEval_EvalCodeEx (ceval.c:3000)
==5591==    by 0x4A4D58: PyEval_EvalFrameEx (ceval.c:3846)
==5591==    by 0x4A6BD0: PyEval_EvalCodeEx (ceval.c:3000)
==5591==    by 0x4A6CA1: PyEval_EvalCode (ceval.c:541)
==5591==    by 0x4B952D: PyImport_ExecCodeModuleEx (import.c:677)
==5591==    by 0x4BB49D: load_source_module (import.c:1017)
==5591==  Address 0x63af020 is 992 bytes inside a block of size 1,104 free'd
==5591==    at 0x4C283A2: free (vg_replace_malloc.c:366)
==5591==    by 0x51D8C9: pattern_subx (_sre.c:260)
==5591==    by 0x51E0CC: pattern_sub (_sre.c:2474)
==5591==    by 0x4A51AD: PyEval_EvalFrameEx (ceval.c:3750)
==5591==    by 0x4A6BD0: PyEval_EvalCodeEx (ceval.c:3000)
==5591==    by 0x4A4D58: PyEval_EvalFrameEx (ceval.c:3846)
==5591==    by 0x4A6BD0: PyEval_EvalCodeEx (ceval.c:3000)
==5591==    by 0x4A6CA1: PyEval_EvalCode (ceval.c:541)
==5591==    by 0x4B952D: PyImport_ExecCodeModuleEx (import.c:677)
==5591==    by 0x4BB49D: load_source_module (import.c:1017)
==5591==    by 0x4BC2D2: import_submodule (import.c:2592)
==5591==    by 0x4BC54E: load_next (import.c:2412)
==5591== 
==5591== Invalid read of size 4
==5591==    at 0x4548BE: PyObject_Realloc (obmalloc.c:1154)
==5591==    by 0x522C49: PyNode_AddChild (node.c:98)
==5591==    by 0x523602: PyParser_AddToken (parser.c:112)
==5591==    by 0x41830B: parsetok (parsetok.c:199)
==5591==    by 0x4C7A41: PyRun_StringFlags (pythonrun.c:1445)
==5591==    by 0x4A3C18: PyEval_EvalFrameEx (ceval.c:4481)
==5591==    by 0x4A6BD0: PyEval_EvalCodeEx (ceval.c:3000)
==5591==    by 0x4A6CA1: PyEval_EvalCode (ceval.c:541)
==5591==    by 0x4B952D: PyImport_ExecCodeModuleEx (import.c:677)
==5591==    by 0x4BB49D: load_source_module (import.c:1017)
==5591==    by 0x4BC2D2: import_submodule (import.c:2592)
==5591==    by 0x4BC54E: load_next (import.c:2412)
==5591==  Address 0x68d6020 is 992 bytes inside a block of size 1,104 free'd
==5591==    at 0x4C283A2: free (vg_replace_malloc.c:366)
==5591==    by 0x51DC65: pattern_subx (_sre.c:260)
==5591==    by 0x51E0CC: pattern_sub (_sre.c:2474)
==5591==    by 0x4A51AD: PyEval_EvalFrameEx (ceval.c:3750)
==5591==    by 0x4A6BD0: PyEval_EvalCodeEx (ceval.c:3000)
==5591==    by 0x4A4D58: PyEval_EvalFrameEx (ceval.c:3846)
==5591==    by 0x4A6BD0: PyEval_EvalCodeEx (ceval.c:3000)
==5591==    by 0x4A6CA1: PyEval_EvalCode (ceval.c:541)
==5591==    by 0x4B952D: PyImport_ExecCodeModuleEx (import.c:677)
==5591==    by 0x4BB49D: load_source_module (import.c:1017)
==5591==    by 0x4BC2D2: import_submodule (import.c:2592)
==5591==    by 0x4BC54E: load_next (import.c:2412)
==5591== 
==5591== Conditional jump or move depends on uninitialised value(s)
==5591==    at 0x4542FC: PyObject_Free (obmalloc.c:931)
==5591==    by 0x523026: freechildren (node.c:135)
==5591==    by 0x522F3B: freechildren (node.c:133)
==5591==    by 0x522F3B: freechildren (node.c:133)
==5591==    by 0x522F3B: freechildren (node.c:133)
==5591==    by 0x5230FD: PyNode_Free (node.c:123)
==5591==    by 0x4C7A76: PyRun_StringFlags (pythonrun.c:1455)
==5591==    by 0x4A3C18: PyEval_EvalFrameEx (ceval.c:4481)
==5591==    by 0x4A6BD0: PyEval_EvalCodeEx (ceval.c:3000)
==5591==    by 0x4A6CA1: PyEval_EvalCode (ceval.c:541)
==5591==    by 0x4B952D: PyImport_ExecCodeModuleEx (import.c:677)
==5591==    by 0x4BB49D: load_source_module (import.c:1017)
==5591== 
==5591== Use of uninitialised value of size 8
==5591==    at 0x454316: PyObject_Free (obmalloc.c:931)
==5591==    by 0x523026: freechildren (node.c:135)
==5591==    by 0x522F3B: freechildren (node.c:133)
==5591==    by 0x522F3B: freechildren (node.c:133)
==5591==    by 0x522F3B: freechildren (node.c:133)
==5591==    by 0x5230FD: PyNode_Free (node.c:123)
==5591==    by 0x4C7A76: PyRun_StringFlags (pythonrun.c:1455)
==5591==    by 0x4A3C18: PyEval_EvalFrameEx (ceval.c:4481)
==5591==    by 0x4A6BD0: PyEval_EvalCodeEx (ceval.c:3000)
==5591==    by 0x4A6CA1: PyEval_EvalCode (ceval.c:541)
==5591==    by 0x4B952D: PyImport_ExecCodeModuleEx (import.c:677)
==5591==    by 0x4BB49D: load_source_module (import.c:1017)
==5591== 
==5591== Conditional jump or move depends on uninitialised value(s)
==5591==    at 0x4542FC: PyObject_Free (obmalloc.c:931)
==5591==    by 0x52306C: freechildren (node.c:137)
==5591==    by 0x522F3B: freechildren (node.c:133)
==5591==    by 0x522F3B: freechildren (node.c:133)
==5591==    by 0x522F3B: freechildren (node.c:133)
==5591==    by 0x5230FD: PyNode_Free (node.c:123)
==5591==    by 0x4C7A76: PyRun_StringFlags (pythonrun.c:1455)
==5591==    by 0x4A3C18: PyEval_EvalFrameEx (ceval.c:4481)
==5591==    by 0x4A6BD0: PyEval_EvalCodeEx (ceval.c:3000)
==5591==    by 0x4A6CA1: PyEval_EvalCode (ceval.c:541)
==5591==    by 0x4B952D: PyImport_ExecCodeModuleEx (import.c:677)
==5591==    by 0x4BB49D: load_source_module (import.c:1017)
==5591== 
==5591== Use of uninitialised value of size 8
==5591==    at 0x454316: PyObject_Free (obmalloc.c:931)
==5591==    by 0x52306C: freechildren (node.c:137)
==5591==    by 0x522F3B: freechildren (node.c:133)
==5591==    by 0x522F3B: freechildren (node.c:133)
==5591==    by 0x522F3B: freechildren (node.c:133)
==5591==    by 0x5230FD: PyNode_Free (node.c:123)
==5591==    by 0x4C7A76: PyRun_StringFlags (pythonrun.c:1455)
==5591==    by 0x4A3C18: PyEval_EvalFrameEx (ceval.c:4481)
==5591==    by 0x4A6BD0: PyEval_EvalCodeEx (ceval.c:3000)
==5591==    by 0x4A6CA1: PyEval_EvalCode (ceval.c:541)
==5591==    by 0x4B952D: PyImport_ExecCodeModuleEx (import.c:677)
==5591==    by 0x4BB49D: load_source_module (import.c:1017)
==5591== 
==5591== Conditional jump or move depends on uninitialised value(s)
==5591==    at 0x4542FC: PyObject_Free (obmalloc.c:931)
==5591==    by 0x523084: freechildren (node.c:135)
==5591==    by 0x522F3B: freechildren (node.c:133)
==5591==    by 0x5230FD: PyNode_Free (node.c:123)
==5591==    by 0x4C7A76: PyRun_StringFlags (pythonrun.c:1455)
==5591==    by 0x4A3C18: PyEval_EvalFrameEx (ceval.c:4481)
==5591==    by 0x4A6BD0: PyEval_EvalCodeEx (ceval.c:3000)
==5591==    by 0x4A6CA1: PyEval_EvalCode (ceval.c:541)
==5591==    by 0x4B952D: PyImport_ExecCodeModuleEx (import.c:677)
==5591==    by 0x4BB49D: load_source_module (import.c:1017)
==5591==    by 0x4BC2D2: import_submodule (import.c:2592)
==5591==    by 0x4BC54E: load_next (import.c:2412)
==5591== 
==5591== Use of uninitialised value of size 8
==5591==    at 0x454316: PyObject_Free (obmalloc.c:931)
==5591==    by 0x523084: freechildren (node.c:135)
==5591==    by 0x522F3B: freechildren (node.c:133)
==5591==    by 0x5230FD: PyNode_Free (node.c:123)
==5591==    by 0x4C7A76: PyRun_StringFlags (pythonrun.c:1455)
==5591==    by 0x4A3C18: PyEval_EvalFrameEx (ceval.c:4481)
==5591==    by 0x4A6BD0: PyEval_EvalCodeEx (ceval.c:3000)
==5591==    by 0x4A6CA1: PyEval_EvalCode (ceval.c:541)
==5591==    by 0x4B952D: PyImport_ExecCodeModuleEx (import.c:677)
==5591==    by 0x4BB49D: load_source_module (import.c:1017)
==5591==    by 0x4BC2D2: import_submodule (import.c:2592)
==5591==    by 0x4BC54E: load_next (import.c:2412)
==5591== 
==5591== Conditional jump or move depends on uninitialised value(s)
==5591==    at 0x4542FC: PyObject_Free (obmalloc.c:931)
==5591==    by 0x522F4E: freechildren (node.c:135)
==5591==    by 0x5230FD: PyNode_Free (node.c:123)
==5591==    by 0x4C7A76: PyRun_StringFlags (pythonrun.c:1455)
==5591==    by 0x4A3C18: PyEval_EvalFrameEx (ceval.c:4481)
==5591==    by 0x4A6BD0: PyEval_EvalCodeEx (ceval.c:3000)
==5591==    by 0x4A6CA1: PyEval_EvalCode (ceval.c:541)
==5591==    by 0x4B952D: PyImport_ExecCodeModuleEx (import.c:677)
==5591==    by 0x4BB49D: load_source_module (import.c:1017)
==5591==    by 0x4BC2D2: import_submodule (import.c:2592)
==5591==    by 0x4BC54E: load_next (import.c:2412)
==5591==    by 0x4BCC10: import_module_level (import.c:2133)
==5591== 
==5591== Use of uninitialised value of size 8
==5591==    at 0x454316: PyObject_Free (obmalloc.c:931)
==5591==    by 0x522F4E: freechildren (node.c:135)
==5591==    by 0x5230FD: PyNode_Free (node.c:123)
==5591==    by 0x4C7A76: PyRun_StringFlags (pythonrun.c:1455)
==5591==    by 0x4A3C18: PyEval_EvalFrameEx (ceval.c:4481)
==5591==    by 0x4A6BD0: PyEval_EvalCodeEx (ceval.c:3000)
==5591==    by 0x4A6CA1: PyEval_EvalCode (ceval.c:541)
==5591==    by 0x4B952D: PyImport_ExecCodeModuleEx (import.c:677)
==5591==    by 0x4BB49D: load_source_module (import.c:1017)
==5591==    by 0x4BC2D2: import_submodule (import.c:2592)
==5591==    by 0x4BC54E: load_next (import.c:2412)
==5591==    by 0x4BCC10: import_module_level (import.c:2133)
==5591== 
==5591== Invalid read of size 4
==5591==    at 0x4542F3: PyObject_Free (obmalloc.c:931)
==5591==    by 0x522FF7: freechildren (node.c:135)
==5591==    by 0x522F3B: freechildren (node.c:133)
==5591==    by 0x5230FD: PyNode_Free (node.c:123)
==5591==    by 0x4C7A76: PyRun_StringFlags (pythonrun.c:1455)
==5591==    by 0x4A3C18: PyEval_EvalFrameEx (ceval.c:4481)
==5591==    by 0x4A6BD0: PyEval_EvalCodeEx (ceval.c:3000)
==5591==    by 0x4A6CA1: PyEval_EvalCode (ceval.c:541)
==5591==    by 0x4B952D: PyImport_ExecCodeModuleEx (import.c:677)
==5591==    by 0x4BB49D: load_source_module (import.c:1017)
==5591==    by 0x4BC2D2: import_submodule (import.c:2592)
==5591==    by 0x4BC54E: load_next (import.c:2412)
==5591==  Address 0xc844020 is 480 bytes inside a block of size 640 free'd
==5591==    at 0x4C29159: realloc (vg_replace_malloc.c:525)
==5591==    by 0x522C49: PyNode_AddChild (node.c:98)
==5591==    by 0x5233D6: PyParser_AddToken (parser.c:126)
==5591==    by 0x41830B: parsetok (parsetok.c:199)
==5591==    by 0x4C7A41: PyRun_StringFlags (pythonrun.c:1445)
==5591==    by 0x4A3C18: PyEval_EvalFrameEx (ceval.c:4481)
==5591==    by 0x4A6BD0: PyEval_EvalCodeEx (ceval.c:3000)
==5591==    by 0x4A6CA1: PyEval_EvalCode (ceval.c:541)
==5591==    by 0x4B952D: PyImport_ExecCodeModuleEx (import.c:677)
==5591==    by 0x4BB49D: load_source_module (import.c:1017)
==5591==    by 0x4BC2D2: import_submodule (import.c:2592)
==5591==    by 0x4BC54E: load_next (import.c:2412)
==5591== 
==5591== Invalid read of size 4
==5591==    at 0x4542F3: PyObject_Free (obmalloc.c:931)
==5591==    by 0x523026: freechildren (node.c:135)
==5591==    by 0x522F3B: freechildren (node.c:133)
==5591==    by 0x5230FD: PyNode_Free (node.c:123)
==5591==    by 0x4C7A76: PyRun_StringFlags (pythonrun.c:1455)
==5591==    by 0x4A3C18: PyEval_EvalFrameEx (ceval.c:4481)
==5591==    by 0x4A6BD0: PyEval_EvalCodeEx (ceval.c:3000)
==5591==    by 0x4A6CA1: PyEval_EvalCode (ceval.c:541)
==5591==    by 0x4B952D: PyImport_ExecCodeModuleEx (import.c:677)
==5591==    by 0x4BB49D: load_source_module (import.c:1017)
==5591==    by 0x4BC2D2: import_submodule (import.c:2592)
==5591==    by 0x4BC54E: load_next (import.c:2412)
==5591==  Address 0xc844020 is 480 bytes inside a block of size 640 free'd
==5591==    at 0x4C29159: realloc (vg_replace_malloc.c:525)
==5591==    by 0x522C49: PyNode_AddChild (node.c:98)
==5591==    by 0x5233D6: PyParser_AddToken (parser.c:126)
==5591==    by 0x41830B: parsetok (parsetok.c:199)
==5591==    by 0x4C7A41: PyRun_StringFlags (pythonrun.c:1445)
==5591==    by 0x4A3C18: PyEval_EvalFrameEx (ceval.c:4481)
==5591==    by 0x4A6BD0: PyEval_EvalCodeEx (ceval.c:3000)
==5591==    by 0x4A6CA1: PyEval_EvalCode (ceval.c:541)
==5591==    by 0x4B952D: PyImport_ExecCodeModuleEx (import.c:677)
==5591==    by 0x4BB49D: load_source_module (import.c:1017)
==5591==    by 0x4BC2D2: import_submodule (import.c:2592)
==5591==    by 0x4BC54E: load_next (import.c:2412)
==5591== 
==5591== Invalid read of size 4
==5591==    at 0x4542F3: PyObject_Free (obmalloc.c:931)
==5591==    by 0x52306C: freechildren (node.c:137)
==5591==    by 0x522F3B: freechildren (node.c:133)
==5591==    by 0x522F3B: freechildren (node.c:133)
==5591==    by 0x522F3B: freechildren (node.c:133)
==5591==    by 0x5230FD: PyNode_Free (node.c:123)
==5591==    by 0x4C7A76: PyRun_StringFlags (pythonrun.c:1455)
==5591==    by 0x4A3C18: PyEval_EvalFrameEx (ceval.c:4481)
==5591==    by 0x4A6BD0: PyEval_EvalCodeEx (ceval.c:3000)
==5591==    by 0x4A6CA1: PyEval_EvalCode (ceval.c:541)
==5591==    by 0x4B952D: PyImport_ExecCodeModuleEx (import.c:677)
==5591==    by 0x4BB49D: load_source_module (import.c:1017)
==5591==  Address 0x8c27020 is 352 bytes inside a block of size 480 free'd
==5591==    at 0x4C29159: realloc (vg_replace_malloc.c:525)
==5591==    by 0x522C49: PyNode_AddChild (node.c:98)
==5591==    by 0x5233D6: PyParser_AddToken (parser.c:126)
==5591==    by 0x41830B: parsetok (parsetok.c:199)
==5591==    by 0x4C7A41: PyRun_StringFlags (pythonrun.c:1445)
==5591==    by 0x4A3C18: PyEval_EvalFrameEx (ceval.c:4481)
==5591==    by 0x4A6BD0: PyEval_EvalCodeEx (ceval.c:3000)
==5591==    by 0x4A6CA1: PyEval_EvalCode (ceval.c:541)
==5591==    by 0x4B952D: PyImport_ExecCodeModuleEx (import.c:677)
==5591==    by 0x4BB49D: load_source_module (import.c:1017)
==5591==    by 0x4BC2D2: import_submodule (import.c:2592)
==5591==    by 0x4BC54E: load_next (import.c:2412)
==5591== 
==5591== Invalid read of size 4
==5591==    at 0x4542F3: PyObject_Free (obmalloc.c:931)
==5591==    by 0x5230B3: freechildren (node.c:135)
==5591==    by 0x522F3B: freechildren (node.c:133)
==5591==    by 0x5230FD: PyNode_Free (node.c:123)
==5591==    by 0x4C7A76: PyRun_StringFlags (pythonrun.c:1455)
==5591==    by 0x4A3C18: PyEval_EvalFrameEx (ceval.c:4481)
==5591==    by 0x4A6BD0: PyEval_EvalCodeEx (ceval.c:3000)
==5591==    by 0x4A6CA1: PyEval_EvalCode (ceval.c:541)
==5591==    by 0x4B952D: PyImport_ExecCodeModuleEx (import.c:677)
==5591==    by 0x4BB49D: load_source_module (import.c:1017)
==5591==    by 0x4BC2D2: import_submodule (import.c:2592)
==5591==    by 0x4BC54E: load_next (import.c:2412)
==5591==  Address 0xc844020 is 480 bytes inside a block of size 640 free'd
==5591==    at 0x4C29159: realloc (vg_replace_malloc.c:525)
==5591==    by 0x522C49: PyNode_AddChild (node.c:98)
==5591==    by 0x5233D6: PyParser_AddToken (parser.c:126)
==5591==    by 0x41830B: parsetok (parsetok.c:199)
==5591==    by 0x4C7A41: PyRun_StringFlags (pythonrun.c:1445)
==5591==    by 0x4A3C18: PyEval_EvalFrameEx (ceval.c:4481)
==5591==    by 0x4A6BD0: PyEval_EvalCodeEx (ceval.c:3000)
==5591==    by 0x4A6CA1: PyEval_EvalCode (ceval.c:541)
==5591==    by 0x4B952D: PyImport_ExecCodeModuleEx (import.c:677)
==5591==    by 0x4BB49D: load_source_module (import.c:1017)
==5591==    by 0x4BC2D2: import_submodule (import.c:2592)
==5591==    by 0x4BC54E: load_next (import.c:2412)
==5591== 
==5591== Invalid read of size 4
==5591==    at 0x4542F3: PyObject_Free (obmalloc.c:931)
==5591==    by 0x523084: freechildren (node.c:135)
==5591==    by 0x522F3B: freechildren (node.c:133)
==5591==    by 0x5230FD: PyNode_Free (node.c:123)
==5591==    by 0x4C7A76: PyRun_StringFlags (pythonrun.c:1455)
==5591==    by 0x4A3C18: PyEval_EvalFrameEx (ceval.c:4481)
==5591==    by 0x4A6BD0: PyEval_EvalCodeEx (ceval.c:3000)
==5591==    by 0x4A6CA1: PyEval_EvalCode (ceval.c:541)
==5591==    by 0x4B952D: PyImport_ExecCodeModuleEx (import.c:677)
==5591==    by 0x4BB49D: load_source_module (import.c:1017)
==5591==    by 0x4BC2D2: import_submodule (import.c:2592)
==5591==    by 0x4BC54E: load_next (import.c:2412)
==5591==  Address 0x8c27020 is 352 bytes inside a block of size 480 free'd
==5591==    at 0x4C29159: realloc (vg_replace_malloc.c:525)
==5591==    by 0x522C49: PyNode_AddChild (node.c:98)
==5591==    by 0x5233D6: PyParser_AddToken (parser.c:126)
==5591==    by 0x41830B: parsetok (parsetok.c:199)
==5591==    by 0x4C7A41: PyRun_StringFlags (pythonrun.c:1445)
==5591==    by 0x4A3C18: PyEval_EvalFrameEx (ceval.c:4481)
==5591==    by 0x4A6BD0: PyEval_EvalCodeEx (ceval.c:3000)
==5591==    by 0x4A6CA1: PyEval_EvalCode (ceval.c:541)
==5591==    by 0x4B952D: PyImport_ExecCodeModuleEx (import.c:677)
==5591==    by 0x4BB49D: load_source_module (import.c:1017)
==5591==    by 0x4BC2D2: import_submodule (import.c:2592)
==5591==    by 0x4BC54E: load_next (import.c:2412)
==5591== 
==5591== Invalid read of size 4
==5591==    at 0x4542F3: PyObject_Free (obmalloc.c:931)
==5591==    by 0x522F4E: freechildren (node.c:135)
==5591==    by 0x5230FD: PyNode_Free (node.c:123)
==5591==    by 0x4C7A76: PyRun_StringFlags (pythonrun.c:1455)
==5591==    by 0x4A3C18: PyEval_EvalFrameEx (ceval.c:4481)
==5591==    by 0x4A6BD0: PyEval_EvalCodeEx (ceval.c:3000)
==5591==    by 0x4A6CA1: PyEval_EvalCode (ceval.c:541)
==5591==    by 0x4B952D: PyImport_ExecCodeModuleEx (import.c:677)
==5591==    by 0x4BB49D: load_source_module (import.c:1017)
==5591==    by 0x4BC2D2: import_submodule (import.c:2592)
==5591==    by 0x4BC54E: load_next (import.c:2412)
==5591==    by 0x4BCC10: import_module_level (import.c:2133)
==5591==  Address 0xc844020 is 480 bytes inside a block of size 640 free'd
==5591==    at 0x4C29159: realloc (vg_replace_malloc.c:525)
==5591==    by 0x522C49: PyNode_AddChild (node.c:98)
==5591==    by 0x5233D6: PyParser_AddToken (parser.c:126)
==5591==    by 0x41830B: parsetok (parsetok.c:199)
==5591==    by 0x4C7A41: PyRun_StringFlags (pythonrun.c:1445)
==5591==    by 0x4A3C18: PyEval_EvalFrameEx (ceval.c:4481)
==5591==    by 0x4A6BD0: PyEval_EvalCodeEx (ceval.c:3000)
==5591==    by 0x4A6CA1: PyEval_EvalCode (ceval.c:541)
==5591==    by 0x4B952D: PyImport_ExecCodeModuleEx (import.c:677)
==5591==    by 0x4BB49D: load_source_module (import.c:1017)
==5591==    by 0x4BC2D2: import_submodule (import.c:2592)
==5591==    by 0x4BC54E: load_next (import.c:2412)
==5591== 
==5591== Invalid read of size 4
==5591==    at 0x4542F3: PyObject_Free (obmalloc.c:931)
==5591==    by 0x523084: freechildren (node.c:135)
==5591==    by 0x522F3B: freechildren (node.c:133)
==5591==    by 0x522F3B: freechildren (node.c:133)
==5591==    by 0x522F3B: freechildren (node.c:133)
==5591==    by 0x5230FD: PyNode_Free (node.c:123)
==5591==    by 0x4C7A76: PyRun_StringFlags (pythonrun.c:1455)
==5591==    by 0x4A3C18: PyEval_EvalFrameEx (ceval.c:4481)
==5591==    by 0x4A6BD0: PyEval_EvalCodeEx (ceval.c:3000)
==5591==    by 0x4A6CA1: PyEval_EvalCode (ceval.c:541)
==5591==    by 0x4B952D: PyImport_ExecCodeModuleEx (import.c:677)
==5591==    by 0x4BB49D: load_source_module (import.c:1017)
==5591==  Address 0x8c26020 is 1,984 bytes inside a block of size 2,080 free'd
==5591==    at 0x4C29159: realloc (vg_replace_malloc.c:525)
==5591==    by 0x522C49: PyNode_AddChild (node.c:98)
==5591==    by 0x5233D6: PyParser_AddToken (parser.c:126)
==5591==    by 0x41830B: parsetok (parsetok.c:199)
==5591==    by 0x4C7A41: PyRun_StringFlags (pythonrun.c:1445)
==5591==    by 0x4A3C18: PyEval_EvalFrameEx (ceval.c:4481)
==5591==    by 0x4A6BD0: PyEval_EvalCodeEx (ceval.c:3000)
==5591==    by 0x4A6CA1: PyEval_EvalCode (ceval.c:541)
==5591==    by 0x4B952D: PyImport_ExecCodeModuleEx (import.c:677)
==5591==    by 0x4BB49D: load_source_module (import.c:1017)
==5591==    by 0x4BC2D2: import_submodule (import.c:2592)
==5591==    by 0x4BC54E: load_next (import.c:2412)
==5591== 
==5591== Invalid read of size 4
==5591==    at 0x4542F3: PyObject_Free (obmalloc.c:931)
==5591==    by 0x52309B: freechildren (node.c:137)
==5591==    by 0x522F3B: freechildren (node.c:133)
==5591==    by 0x522F3B: freechildren (node.c:133)
==5591==    by 0x522F3B: freechildren (node.c:133)
==5591==    by 0x5230FD: PyNode_Free (node.c:123)
==5591==    by 0x4C7A76: PyRun_StringFlags (pythonrun.c:1455)
==5591==    by 0x4A3C18: PyEval_EvalFrameEx (ceval.c:4481)
==5591==    by 0x4A6BD0: PyEval_EvalCodeEx (ceval.c:3000)
==5591==    by 0x4A6CA1: PyEval_EvalCode (ceval.c:541)
==5591==    by 0x4B952D: PyImport_ExecCodeModuleEx (import.c:677)
==5591==    by 0x4BB49D: load_source_module (import.c:1017)
==5591==  Address 0x8c26020 is 1,984 bytes inside a block of size 2,080 free'd
==5591==    at 0x4C29159: realloc (vg_replace_malloc.c:525)
==5591==    by 0x522C49: PyNode_AddChild (node.c:98)
==5591==    by 0x5233D6: PyParser_AddToken (parser.c:126)
==5591==    by 0x41830B: parsetok (parsetok.c:199)
==5591==    by 0x4C7A41: PyRun_StringFlags (pythonrun.c:1445)
==5591==    by 0x4A3C18: PyEval_EvalFrameEx (ceval.c:4481)
==5591==    by 0x4A6BD0: PyEval_EvalCodeEx (ceval.c:3000)
==5591==    by 0x4A6CA1: PyEval_EvalCode (ceval.c:541)
==5591==    by 0x4B952D: PyImport_ExecCodeModuleEx (import.c:677)
==5591==    by 0x4BB49D: load_source_module (import.c:1017)
==5591==    by 0x4BC2D2: import_submodule (import.c:2592)
==5591==    by 0x4BC54E: load_next (import.c:2412)
==5591== 
==5591== Conditional jump or move depends on uninitialised value(s)
==5591==    at 0x4542FC: PyObject_Free (obmalloc.c:931)
==5591==    by 0x5230B3: freechildren (node.c:135)
==5591==    by 0x522F3B: freechildren (node.c:133)
==5591==    by 0x5230FD: PyNode_Free (node.c:123)
==5591==    by 0x4C7A76: PyRun_StringFlags (pythonrun.c:1455)
==5591==    by 0x4A3C18: PyEval_EvalFrameEx (ceval.c:4481)
==5591==    by 0x4A6BD0: PyEval_EvalCodeEx (ceval.c:3000)
==5591==    by 0x4A6CA1: PyEval_EvalCode (ceval.c:541)
==5591==    by 0x4B952D: PyImport_ExecCodeModuleEx (import.c:677)
==5591==    by 0x4BB49D: load_source_module (import.c:1017)
==5591==    by 0x4BC2D2: import_submodule (import.c:2592)
==5591==    by 0x4BC54E: load_next (import.c:2412)
==5591== 
==5591== Use of uninitialised value of size 8
==5591==    at 0x454316: PyObject_Free (obmalloc.c:931)
==5591==    by 0x5230B3: freechildren (node.c:135)
==5591==    by 0x522F3B: freechildren (node.c:133)
==5591==    by 0x5230FD: PyNode_Free (node.c:123)
==5591==    by 0x4C7A76: PyRun_StringFlags (pythonrun.c:1455)
==5591==    by 0x4A3C18: PyEval_EvalFrameEx (ceval.c:4481)
==5591==    by 0x4A6BD0: PyEval_EvalCodeEx (ceval.c:3000)
==5591==    by 0x4A6CA1: PyEval_EvalCode (ceval.c:541)
==5591==    by 0x4B952D: PyImport_ExecCodeModuleEx (import.c:677)
==5591==    by 0x4BB49D: load_source_module (import.c:1017)
==5591==    by 0x4BC2D2: import_submodule (import.c:2592)
==5591==    by 0x4BC54E: load_next (import.c:2412)
==5591== 
==5591== 
==5591== More than 100 errors detected.  Subsequent errors
==5591== will still be recorded, but in less detail than before.
==5591== Conditional jump or move depends on uninitialised value(s)
==5591==    at 0x4542FC: PyObject_Free (obmalloc.c:931)
==5591==    by 0x52309B: freechildren (node.c:137)
==5591==    by 0x522F3B: freechildren (node.c:133)
==5591==    by 0x522F3B: freechildren (node.c:133)
==5591==    by 0x522F3B: freechildren (node.c:133)
==5591==    by 0x5230FD: PyNode_Free (node.c:123)
==5591==    by 0x4C7A76: PyRun_StringFlags (pythonrun.c:1455)
==5591==    by 0x4A3C18: PyEval_EvalFrameEx (ceval.c:4481)
==5591==    by 0x4A6BD0: PyEval_EvalCodeEx (ceval.c:3000)
==5591==    by 0x4A6CA1: PyEval_EvalCode (ceval.c:541)
==5591==    by 0x4B952D: PyImport_ExecCodeModuleEx (import.c:677)
==5591==    by 0x4BB49D: load_source_module (import.c:1017)
==5591== 
==5591== Use of uninitialised value of size 8
==5591==    at 0x454316: PyObject_Free (obmalloc.c:931)
==5591==    by 0x52309B: freechildren (node.c:137)
==5591==    by 0x522F3B: freechildren (node.c:133)
==5591==    by 0x522F3B: freechildren (node.c:133)
==5591==    by 0x522F3B: freechildren (node.c:133)
==5591==    by 0x5230FD: PyNode_Free (node.c:123)
==5591==    by 0x4C7A76: PyRun_StringFlags (pythonrun.c:1455)
==5591==    by 0x4A3C18: PyEval_EvalFrameEx (ceval.c:4481)
==5591==    by 0x4A6BD0: PyEval_EvalCodeEx (ceval.c:3000)
==5591==    by 0x4A6CA1: PyEval_EvalCode (ceval.c:541)
==5591==    by 0x4B952D: PyImport_ExecCodeModuleEx (import.c:677)
==5591==    by 0x4BB49D: load_source_module (import.c:1017)
==5591== 
==5591== Invalid read of size 4
==5591==    at 0x4542F3: PyObject_Free (obmalloc.c:931)
==5591==    by 0x522FB0: freechildren (node.c:137)
==5591==    by 0x522F3B: freechildren (node.c:133)
==5591==    by 0x522F3B: freechildren (node.c:133)
==5591==    by 0x5230FD: PyNode_Free (node.c:123)
==5591==    by 0x4C7A76: PyRun_StringFlags (pythonrun.c:1455)
==5591==    by 0x4A3C18: PyEval_EvalFrameEx (ceval.c:4481)
==5591==    by 0x4A6BD0: PyEval_EvalCodeEx (ceval.c:3000)
==5591==    by 0x4A6CA1: PyEval_EvalCode (ceval.c:541)
==5591==    by 0x4B952D: PyImport_ExecCodeModuleEx (import.c:677)
==5591==    by 0x4BB49D: load_source_module (import.c:1017)
==5591==    by 0x4BC2D2: import_submodule (import.c:2592)
==5591==  Address 0x6326020 is 16 bytes before a block of size 280 alloc'd
==5591==    at 0x4C29061: malloc (vg_replace_malloc.c:236)
==5591==    by 0x4D4F38: _PyObject_GC_New (gcmodule.c:1351)
==5591==    by 0x44D2D1: PyDict_New (dictobject.c:256)
==5591==    by 0x4504FE: PyModule_New (moduleobject.c:26)
==5591==    by 0x4B9347: PyImport_AddModule (import.c:608)
==5591==    by 0x4C5B27: initmain (pythonrun.c:692)
==5591==    by 0x4C8546: Py_InitializeEx (pythonrun.c:247)
==5591==    by 0x417A46: Py_Main (main.c:485)
==5591==    by 0x5EEAD8D: (below main) (libc-start.c:226)
==5591== 
==5591== Invalid read of size 4
==5591==    at 0x4542F3: PyObject_Free (obmalloc.c:931)
==5591==    by 0x4A7D65: compiler_exit_scope (compile.c:521)
==5591==    by 0x4AF3FF: compiler_function (compile.c:1401)
==5591==    by 0x4AF00D: compiler_class (compile.c:1187)
==5591==    by 0x4B05CD: compiler_mod (compile.c:1187)
==5591==    by 0x4B0711: PyAST_Compile (compile.c:288)
==5591==    by 0x4C7A90: PyRun_StringFlags (pythonrun.c:1348)
==5591==    by 0x4A3C18: PyEval_EvalFrameEx (ceval.c:4481)
==5591==    by 0x4A6BD0: PyEval_EvalCodeEx (ceval.c:3000)
==5591==    by 0x4A6CA1: PyEval_EvalCode (ceval.c:541)
==5591==    by 0x4B952D: PyImport_ExecCodeModuleEx (import.c:677)
==5591==    by 0x4BB49D: load_source_module (import.c:1017)
==5591==  Address 0xc867020 is 368 bytes inside a block of size 384 free'd
==5591==    at 0x4C283A2: free (vg_replace_malloc.c:366)
==5591==    by 0x4A70BD: compiler_unit_free (compile.c:423)
==5591==    by 0x4A7D65: compiler_exit_scope (compile.c:521)
==5591==    by 0x4AF3FF: compiler_function (compile.c:1401)
==5591==    by 0x4AF00D: compiler_class (compile.c:1187)
==5591==    by 0x4B05CD: compiler_mod (compile.c:1187)
==5591==    by 0x4B0711: PyAST_Compile (compile.c:288)
==5591==    by 0x4C7A90: PyRun_StringFlags (pythonrun.c:1348)
==5591==    by 0x4A3C18: PyEval_EvalFrameEx (ceval.c:4481)
==5591==    by 0x4A6BD0: PyEval_EvalCodeEx (ceval.c:3000)
==5591==    by 0x4A6CA1: PyEval_EvalCode (ceval.c:541)
==5591==    by 0x4B952D: PyImport_ExecCodeModuleEx (import.c:677)
==5591== 
==5591== Invalid read of size 4
==5591==    at 0x4542F3: PyObject_Free (obmalloc.c:931)
==5591==    by 0x441391: list_ass_slice (listobject.c:591)
==5591==    by 0x4C39E9: PyArena_Free (pyarena.c:181)
==5591==    by 0x4C7AC5: PyRun_StringFlags (pythonrun.c:1315)
==5591==    by 0x4A3C18: PyEval_EvalFrameEx (ceval.c:4481)
==5591==    by 0x4A6BD0: PyEval_EvalCodeEx (ceval.c:3000)
==5591==    by 0x4A6CA1: PyEval_EvalCode (ceval.c:541)
==5591==    by 0x4B952D: PyImport_ExecCodeModuleEx (import.c:677)
==5591==    by 0x4BB49D: load_source_module (import.c:1017)
==5591==    by 0x4BC2D2: import_submodule (import.c:2592)
==5591==    by 0x4BC54E: load_next (import.c:2412)
==5591==    by 0x4BCC10: import_module_level (import.c:2133)
==5591==  Address 0xc8bb020 is not stack'd, malloc'd or (recently) free'd
==5591== 
==5591== Invalid read of size 4
==5591==    at 0x4542F3: PyObject_Free (obmalloc.c:931)
==5591==    by 0x4A46B3: PyEval_EvalFrameEx (ceval.c:1742)
==5591==    by 0x4A6BD0: PyEval_EvalCodeEx (ceval.c:3000)
==5591==    by 0x4A6CA1: PyEval_EvalCode (ceval.c:541)
==5591==    by 0x4B952D: PyImport_ExecCodeModuleEx (import.c:677)
==5591==    by 0x4BB49D: load_source_module (import.c:1017)
==5591==    by 0x4BC2D2: import_submodule (import.c:2592)
==5591==    by 0x4BC54E: load_next (import.c:2412)
==5591==    by 0x4BCC10: import_module_level (import.c:2133)
==5591==    by 0x4BD133: PyImport_ImportModuleLevel (import.c:2185)
==5591==    by 0x49EBFA: builtin___import__ (bltinmodule.c:48)
==5591==    by 0x41C9D6: PyObject_Call (abstract.c:2492)
==5591==  Address 0xc8a4020 is 944 bytes inside a block of size 1,608 free'd
==5591==    at 0x4C283A2: free (vg_replace_malloc.c:366)
==5591==    by 0x43F2CC: list_dealloc (listobject.c:308)
==5591==    by 0x467E2C: tupledealloc (tupleobject.c:170)
==5591==    by 0x51DF31: pattern_subx (_sre.c:2024)
==5591==    by 0x51E0CC: pattern_sub (_sre.c:2474)
==5591==    by 0x4A51AD: PyEval_EvalFrameEx (ceval.c:3750)
==5591==    by 0x4A6BD0: PyEval_EvalCodeEx (ceval.c:3000)
==5591==    by 0x4A4D58: PyEval_EvalFrameEx (ceval.c:3846)
==5591==    by 0x4A6BD0: PyEval_EvalCodeEx (ceval.c:3000)
==5591==    by 0x4A6CA1: PyEval_EvalCode (ceval.c:541)
==5591==    by 0x4B952D: PyImport_ExecCodeModuleEx (import.c:677)
==5591==    by 0x4BB49D: load_source_module (import.c:1017)
==5591== 
==5591== Conditional jump or move depends on uninitialised value(s)
==5591==    at 0x4542FC: PyObject_Free (obmalloc.c:931)
==5591==    by 0x43F2C1: list_dealloc (listobject.c:306)
==5591==    by 0x467E2C: tupledealloc (tupleobject.c:170)
==5591==    by 0x51DF31: pattern_subx (_sre.c:2024)
==5591==    by 0x51E0CC: pattern_sub (_sre.c:2474)
==5591==    by 0x4A51AD: PyEval_EvalFrameEx (ceval.c:3750)
==5591==    by 0x4A6BD0: PyEval_EvalCodeEx (ceval.c:3000)
==5591==    by 0x4A4D58: PyEval_EvalFrameEx (ceval.c:3846)
==5591==    by 0x4A6BD0: PyEval_EvalCodeEx (ceval.c:3000)
==5591==    by 0x4A6CA1: PyEval_EvalCode (ceval.c:541)
==5591==    by 0x4B952D: PyImport_ExecCodeModuleEx (import.c:677)
==5591==    by 0x4BB49D: load_source_module (import.c:1017)
==5591== 
==5591== Use of uninitialised value of size 8
==5591==    at 0x454316: PyObject_Free (obmalloc.c:931)
==5591==    by 0x43F2C1: list_dealloc (listobject.c:306)
==5591==    by 0x467E2C: tupledealloc (tupleobject.c:170)
==5591==    by 0x51DF31: pattern_subx (_sre.c:2024)
==5591==    by 0x51E0CC: pattern_sub (_sre.c:2474)
==5591==    by 0x4A51AD: PyEval_EvalFrameEx (ceval.c:3750)
==5591==    by 0x4A6BD0: PyEval_EvalCodeEx (ceval.c:3000)
==5591==    by 0x4A4D58: PyEval_EvalFrameEx (ceval.c:3846)
==5591==    by 0x4A6BD0: PyEval_EvalCodeEx (ceval.c:3000)
==5591==    by 0x4A6CA1: PyEval_EvalCode (ceval.c:541)
==5591==    by 0x4B952D: PyImport_ExecCodeModuleEx (import.c:677)
==5591==    by 0x4BB49D: load_source_module (import.c:1017)
==5591== 
==5591== Conditional jump or move depends on uninitialised value(s)
==5591==    at 0x4542FC: PyObject_Free (obmalloc.c:931)
==5591==    by 0x522FC8: freechildren (node.c:135)
==5591==    by 0x522F3B: freechildren (node.c:133)
==5591==    by 0x522F3B: freechildren (node.c:133)
==5591==    by 0x522F3B: freechildren (node.c:133)
==5591==    by 0x5230FD: PyNode_Free (node.c:123)
==5591==    by 0x4C7A76: PyRun_StringFlags (pythonrun.c:1455)
==5591==    by 0x4A3C18: PyEval_EvalFrameEx (ceval.c:4481)
==5591==    by 0x4A6BD0: PyEval_EvalCodeEx (ceval.c:3000)
==5591==    by 0x4A6CA1: PyEval_EvalCode (ceval.c:541)
==5591==    by 0x4B952D: PyImport_ExecCodeModuleEx (import.c:677)
==5591==    by 0x4BB49D: load_source_module (import.c:1017)
==5591== 
==5591== Use of uninitialised value of size 8
==5591==    at 0x454316: PyObject_Free (obmalloc.c:931)
==5591==    by 0x522FC8: freechildren (node.c:135)
==5591==    by 0x522F3B: freechildren (node.c:133)
==5591==    by 0x522F3B: freechildren (node.c:133)
==5591==    by 0x522F3B: freechildren (node.c:133)
==5591==    by 0x5230FD: PyNode_Free (node.c:123)
==5591==    by 0x4C7A76: PyRun_StringFlags (pythonrun.c:1455)
==5591==    by 0x4A3C18: PyEval_EvalFrameEx (ceval.c:4481)
==5591==    by 0x4A6BD0: PyEval_EvalCodeEx (ceval.c:3000)
==5591==    by 0x4A6CA1: PyEval_EvalCode (ceval.c:541)
==5591==    by 0x4B952D: PyImport_ExecCodeModuleEx (import.c:677)
==5591==    by 0x4BB49D: load_source_module (import.c:1017)
==5591== 
==5591== Invalid read of size 4
==5591==    at 0x4542F3: PyObject_Free (obmalloc.c:931)
==5591==    by 0x522FC8: freechildren (node.c:135)
==5591==    by 0x522F3B: freechildren (node.c:133)
==5591==    by 0x522F3B: freechildren (node.c:133)
==5591==    by 0x522F3B: freechildren (node.c:133)
==5591==    by 0x5230FD: PyNode_Free (node.c:123)
==5591==    by 0x4C7A76: PyRun_StringFlags (pythonrun.c:1455)
==5591==    by 0x4A3C18: PyEval_EvalFrameEx (ceval.c:4481)
==5591==    by 0x4A6BD0: PyEval_EvalCodeEx (ceval.c:3000)
==5591==    by 0x4A6CA1: PyEval_EvalCode (ceval.c:541)
==5591==    by 0x4B952D: PyImport_ExecCodeModuleEx (import.c:677)
==5591==    by 0x4BB49D: load_source_module (import.c:1017)
==5591==  Address 0x8f0c020 is 1,072 bytes inside a block of size 1,280 free'd
==5591==    at 0x4C29159: realloc (vg_replace_malloc.c:525)
==5591==    by 0x522C49: PyNode_AddChild (node.c:98)
==5591==    by 0x5233D6: PyParser_AddToken (parser.c:126)
==5591==    by 0x41830B: parsetok (parsetok.c:199)
==5591==    by 0x4C7A41: PyRun_StringFlags (pythonrun.c:1445)
==5591==    by 0x4A3C18: PyEval_EvalFrameEx (ceval.c:4481)
==5591==    by 0x4A6BD0: PyEval_EvalCodeEx (ceval.c:3000)
==5591==    by 0x4A6CA1: PyEval_EvalCode (ceval.c:541)
==5591==    by 0x4B952D: PyImport_ExecCodeModuleEx (import.c:677)
==5591==    by 0x4BB49D: load_source_module (import.c:1017)
==5591==    by 0x4BC2D2: import_submodule (import.c:2592)
==5591==    by 0x4BC54E: load_next (import.c:2412)
==5591== 
==5591== Conditional jump or move depends on uninitialised value(s)
==5591==    at 0x4542FC: PyObject_Free (obmalloc.c:931)
==5591==    by 0x4A7D65: compiler_exit_scope (compile.c:521)
==5591==    by 0x4AF3FF: compiler_function (compile.c:1401)
==5591==    by 0x4AF00D: compiler_class (compile.c:1187)
==5591==    by 0x4B05CD: compiler_mod (compile.c:1187)
==5591==    by 0x4B0711: PyAST_Compile (compile.c:288)
==5591==    by 0x4C7A90: PyRun_StringFlags (pythonrun.c:1348)
==5591==    by 0x4A3C18: PyEval_EvalFrameEx (ceval.c:4481)
==5591==    by 0x4A6BD0: PyEval_EvalCodeEx (ceval.c:3000)
==5591==    by 0x4A6CA1: PyEval_EvalCode (ceval.c:541)
==5591==    by 0x4B952D: PyImport_ExecCodeModuleEx (import.c:677)
==5591==    by 0x4BB49D: load_source_module (import.c:1017)
==5591== 
==5591== Use of uninitialised value of size 8
==5591==    at 0x454316: PyObject_Free (obmalloc.c:931)
==5591==    by 0x4A7D65: compiler_exit_scope (compile.c:521)
==5591==    by 0x4AF3FF: compiler_function (compile.c:1401)
==5591==    by 0x4AF00D: compiler_class (compile.c:1187)
==5591==    by 0x4B05CD: compiler_mod (compile.c:1187)
==5591==    by 0x4B0711: PyAST_Compile (compile.c:288)
==5591==    by 0x4C7A90: PyRun_StringFlags (pythonrun.c:1348)
==5591==    by 0x4A3C18: PyEval_EvalFrameEx (ceval.c:4481)
==5591==    by 0x4A6BD0: PyEval_EvalCodeEx (ceval.c:3000)
==5591==    by 0x4A6CA1: PyEval_EvalCode (ceval.c:541)
==5591==    by 0x4B952D: PyImport_ExecCodeModuleEx (import.c:677)
==5591==    by 0x4BB49D: load_source_module (import.c:1017)
==5591== 
==5591== Invalid read of size 4
==5591==    at 0x4548BE: PyObject_Realloc (obmalloc.c:1154)
==5591==    by 0x463B04: PyString_FromFormatV (stringobject.c:4277)
==5591==    by 0x46414A: PyString_FromFormat (stringobject.c:357)
==5591==    by 0x89230DB: ufunc_get_doc (ufunc_object.c:4910)
==5591==    by 0x4516CA: PyObject_GenericGetAttr (object.c:1424)
==5591==    by 0x49D05E: builtin_getattr (bltinmodule.c:828)
==5591==    by 0x4A51AD: PyEval_EvalFrameEx (ceval.c:3750)
==5591==    by 0x4A6BD0: PyEval_EvalCodeEx (ceval.c:3000)
==5591==    by 0x535B4F: function_call (funcobject.c:524)
==5591==    by 0x41C9D6: PyObject_Call (abstract.c:2492)
==5591==    by 0x42570E: instancemethod_call (classobject.c:2579)
==5591==    by 0x41C9D6: PyObject_Call (abstract.c:2492)
==5591==  Address 0x689e020 is 16 bytes before a block of size 348 alloc'd
==5591==    at 0x4C29061: malloc (vg_replace_malloc.c:236)
==5591==    by 0x45BF41: PyString_FromStringAndSize (stringobject.c:83)
==5591==    by 0x4A51AD: PyEval_EvalFrameEx (ceval.c:3750)
==5591==    by 0x4A6BD0: PyEval_EvalCodeEx (ceval.c:3000)
==5591==    by 0x4A4D58: PyEval_EvalFrameEx (ceval.c:3846)
==5591==    by 0x4A6BD0: PyEval_EvalCodeEx (ceval.c:3000)
==5591==    by 0x4A6CA1: PyEval_EvalCode (ceval.c:541)
==5591==    by 0x4B952D: PyImport_ExecCodeModuleEx (import.c:677)
==5591==    by 0x4BB49D: load_source_module (import.c:1017)
==5591==    by 0x4BC2D2: import_submodule (import.c:2592)
==5591==    by 0x4BC54E: load_next (import.c:2412)
==5591==    by 0x4BCC10: import_module_level (import.c:2133)
==5591== 
==5591== Conditional jump or move depends on uninitialised value(s)
==5591==    at 0x4548C7: PyObject_Realloc (obmalloc.c:1154)
==5591==    by 0x45E346: _PyString_Resize (stringobject.c:4277)
==5591==    by 0x4651D5: PyString_Format (stringobject.c:5064)
==5591==    by 0x41D06E: binary_op1 (abstract.c:917)
==5591==    by 0x41FE2D: PyNumber_Remainder (abstract.c:969)
==5591==    by 0x4A2663: PyEval_EvalFrameEx (ceval.c:1180)
==5591==    by 0x4A5AAF: PyEval_EvalFrameEx (ceval.c:3836)
==5591==    by 0x4A6BD0: PyEval_EvalCodeEx (ceval.c:3000)
==5591==    by 0x535B4F: function_call (funcobject.c:524)
==5591==    by 0x41C9D6: PyObject_Call (abstract.c:2492)
==5591==    by 0x42570E: instancemethod_call (classobject.c:2579)
==5591==    by 0x41C9D6: PyObject_Call (abstract.c:2492)
==5591== 
==5591== Use of uninitialised value of size 8
==5591==    at 0x4548DA: PyObject_Realloc (obmalloc.c:1154)
==5591==    by 0x45E346: _PyString_Resize (stringobject.c:4277)
==5591==    by 0x46528F: PyString_Format (stringobject.c:4667)
==5591==    by 0x41D06E: binary_op1 (abstract.c:917)
==5591==    by 0x41FE2D: PyNumber_Remainder (abstract.c:969)
==5591==    by 0x4A2663: PyEval_EvalFrameEx (ceval.c:1180)
==5591==    by 0x4A6BD0: PyEval_EvalCodeEx (ceval.c:3000)
==5591==    by 0x4A4D58: PyEval_EvalFrameEx (ceval.c:3846)
==5591==    by 0x4A6BD0: PyEval_EvalCodeEx (ceval.c:3000)
==5591==    by 0x4A6CA1: PyEval_EvalCode (ceval.c:541)
==5591==    by 0x4B952D: PyImport_ExecCodeModuleEx (import.c:677)
==5591==    by 0x4BB49D: load_source_module (import.c:1017)
==5591== 
==5591== Conditional jump or move depends on uninitialised value(s)
==5591==    at 0x4548C7: PyObject_Realloc (obmalloc.c:1154)
==5591==    by 0x4D39C8: _PyObject_GC_Resize (gcmodule.c:1396)
==5591==    by 0x534D84: PyFrame_New (frameobject.c:655)
==5591==    by 0x4A6333: PyEval_EvalCodeEx (ceval.c:2778)
==5591==    by 0x535C4C: function_call (funcobject.c:524)
==5591==    by 0x41C9D6: PyObject_Call (abstract.c:2492)
==5591==    by 0x42570E: instancemethod_call (classobject.c:2579)
==5591==    by 0x41C9D6: PyObject_Call (abstract.c:2492)
==5591==    by 0x47552E: slot_tp_init (typeobject.c:5632)
==5591==    by 0x46CD9E: type_call (typeobject.c:742)
==5591==    by 0x41C9D6: PyObject_Call (abstract.c:2492)
==5591==    by 0x4A48C7: PyEval_EvalFrameEx (ceval.c:3968)
==5591== 
==5591== Use of uninitialised value of size 8
==5591==    at 0x4548DA: PyObject_Realloc (obmalloc.c:1154)
==5591==    by 0x4D39C8: _PyObject_GC_Resize (gcmodule.c:1396)
==5591==    by 0x534D84: PyFrame_New (frameobject.c:655)
==5591==    by 0x4A6333: PyEval_EvalCodeEx (ceval.c:2778)
==5591==    by 0x535C4C: function_call (funcobject.c:524)
==5591==    by 0x41C9D6: PyObject_Call (abstract.c:2492)
==5591==    by 0x42570E: instancemethod_call (classobject.c:2579)
==5591==    by 0x41C9D6: PyObject_Call (abstract.c:2492)
==5591==    by 0x47552E: slot_tp_init (typeobject.c:5632)
==5591==    by 0x46CD9E: type_call (typeobject.c:742)
==5591==    by 0x41C9D6: PyObject_Call (abstract.c:2492)
==5591==    by 0x4A48C7: PyEval_EvalFrameEx (ceval.c:3968)
==5591== 
==5591== Invalid read of size 4
==5591==    at 0x4548BE: PyObject_Realloc (obmalloc.c:1154)
==5591==    by 0x48440E: _PyUnicode_Resize (unicodeobject.c:262)
==5591==    by 0x491FF8: PyUnicodeUCS4_Join (unicodeobject.c:5547)
==5591==    by 0x45E05F: string_join (stringobject.c:1863)
==5591==    by 0x4A5A0D: PyEval_EvalFrameEx (ceval.c:3738)
==5591==    by 0x4A5AAF: PyEval_EvalFrameEx (ceval.c:3836)
==5591==    by 0x4A5AAF: PyEval_EvalFrameEx (ceval.c:3836)
==5591==    by 0x4A6BD0: PyEval_EvalCodeEx (ceval.c:3000)
==5591==    by 0x4A6CA1: PyEval_EvalCode (ceval.c:541)
==5591==    by 0x4B952D: PyImport_ExecCodeModuleEx (import.c:677)
==5591==    by 0x4BB49D: load_source_module (import.c:1017)
==5591==    by 0x4BC2D2: import_submodule (import.c:2592)
==5591==  Address 0xd236020 is 32 bytes inside a block of size 64 free'd
==5591==    at 0x4C283A2: free (vg_replace_malloc.c:366)
==5591==    by 0x43F2CC: list_dealloc (listobject.c:308)
==5591==    by 0x43CF46: listiter_next (listobject.c:2897)
==5591==    by 0x4A0AFD: PyEval_EvalFrameEx (ceval.c:2272)
==5591==    by 0x4A5AAF: PyEval_EvalFrameEx (ceval.c:3836)
==5591==    by 0x4A5AAF: PyEval_EvalFrameEx (ceval.c:3836)
==5591==    by 0x4A6BD0: PyEval_EvalCodeEx (ceval.c:3000)
==5591==    by 0x4A6CA1: PyEval_EvalCode (ceval.c:541)
==5591==    by 0x4B952D: PyImport_ExecCodeModuleEx (import.c:677)
==5591==    by 0x4BB49D: load_source_module (import.c:1017)
==5591==    by 0x4BC2D2: import_submodule (import.c:2592)
==5591==    by 0x4BC54E: load_next (import.c:2412)
==5591== 
==5591== Invalid read of size 4
==5591==    at 0x4542F3: PyObject_Free (obmalloc.c:931)
==5591==    by 0x44D60F: dict_dealloc (dictobject.c:919)
==5591==    by 0x46AB19: subtype_dealloc (typeobject.c:861)
==5591==    by 0x4A0B80: PyEval_EvalFrameEx (ceval.c:1010)
==5591==    by 0x4A6BD0: PyEval_EvalCodeEx (ceval.c:3000)
==5591==    by 0x4A4D58: PyEval_EvalFrameEx (ceval.c:3846)
==5591==    by 0x4A6BD0: PyEval_EvalCodeEx (ceval.c:3000)
==5591==    by 0x4A4D58: PyEval_EvalFrameEx (ceval.c:3846)
==5591==    by 0x4A6BD0: PyEval_EvalCodeEx (ceval.c:3000)
==5591==    by 0x4A4D58: PyEval_EvalFrameEx (ceval.c:3846)
==5591==    by 0x4A6BD0: PyEval_EvalCodeEx (ceval.c:3000)
==5591==    by 0x4A4D58: PyEval_EvalFrameEx (ceval.c:3846)
==5591==  Address 0xd230020 is 16 bytes before a block of size 1,104 free'd
==5591==    at 0x4C283A2: free (vg_replace_malloc.c:366)
==5591==    by 0x51600C: state_fini (_sre.c:260)
==5591==    by 0x51E4EA: pattern_match (_sre.c:1899)
==5591==    by 0x4A51AD: PyEval_EvalFrameEx (ceval.c:3750)
==5591==    by 0x4A6BD0: PyEval_EvalCodeEx (ceval.c:3000)
==5591==    by 0x4A4D58: PyEval_EvalFrameEx (ceval.c:3846)
==5591==    by 0x4A6BD0: PyEval_EvalCodeEx (ceval.c:3000)
==5591==    by 0x4A4D58: PyEval_EvalFrameEx (ceval.c:3846)
==5591==    by 0x4A6BD0: PyEval_EvalCodeEx (ceval.c:3000)
==5591==    by 0x4A4D58: PyEval_EvalFrameEx (ceval.c:3846)
==5591==    by 0x4A6BD0: PyEval_EvalCodeEx (ceval.c:3000)
==5591==    by 0x4A4D58: PyEval_EvalFrameEx (ceval.c:3846)
==5591== 
==5591== Conditional jump or move depends on uninitialised value(s)
==5591==    at 0x4542FC: PyObject_Free (obmalloc.c:931)
==5591==    by 0x44D60F: dict_dealloc (dictobject.c:919)
==5591==    by 0x46AB19: subtype_dealloc (typeobject.c:861)
==5591==    by 0x4A0B80: PyEval_EvalFrameEx (ceval.c:1010)
==5591==    by 0x4A6BD0: PyEval_EvalCodeEx (ceval.c:3000)
==5591==    by 0x4A4D58: PyEval_EvalFrameEx (ceval.c:3846)
==5591==    by 0x4A6BD0: PyEval_EvalCodeEx (ceval.c:3000)
==5591==    by 0x4A4D58: PyEval_EvalFrameEx (ceval.c:3846)
==5591==    by 0x4A6BD0: PyEval_EvalCodeEx (ceval.c:3000)
==5591==    by 0x4A4D58: PyEval_EvalFrameEx (ceval.c:3846)
==5591==    by 0x4A6BD0: PyEval_EvalCodeEx (ceval.c:3000)
==5591==    by 0x4A4D58: PyEval_EvalFrameEx (ceval.c:3846)
==5591== 
==5591== Use of uninitialised value of size 8
==5591==    at 0x454316: PyObject_Free (obmalloc.c:931)
==5591==    by 0x44D60F: dict_dealloc (dictobject.c:919)
==5591==    by 0x46AB19: subtype_dealloc (typeobject.c:861)
==5591==    by 0x4A0B80: PyEval_EvalFrameEx (ceval.c:1010)
==5591==    by 0x4A6BD0: PyEval_EvalCodeEx (ceval.c:3000)
==5591==    by 0x4A4D58: PyEval_EvalFrameEx (ceval.c:3846)
==5591==    by 0x4A6BD0: PyEval_EvalCodeEx (ceval.c:3000)
==5591==    by 0x4A4D58: PyEval_EvalFrameEx (ceval.c:3846)
==5591==    by 0x4A6BD0: PyEval_EvalCodeEx (ceval.c:3000)
==5591==    by 0x4A4D58: PyEval_EvalFrameEx (ceval.c:3846)
==5591==    by 0x4A6BD0: PyEval_EvalCodeEx (ceval.c:3000)
==5591==    by 0x4A4D58: PyEval_EvalFrameEx (ceval.c:3846)
==5591== 
==5591== Invalid read of size 4
==5591==    at 0x4542F3: PyObject_Free (obmalloc.c:931)
==5591==    by 0x4A34CD: PyEval_EvalFrameEx (ceval.c:1345)
==5591==    by 0x4A6BD0: PyEval_EvalCodeEx (ceval.c:3000)
==5591==    by 0x535B4F: function_call (funcobject.c:524)
==5591==    by 0x41C9D6: PyObject_Call (abstract.c:2492)
==5591==    by 0x42570E: instancemethod_call (classobject.c:2579)
==5591==    by 0x41C9D6: PyObject_Call (abstract.c:2492)
==5591==    by 0x49F052: PyEval_CallObjectWithKeywords (ceval.c:3619)
==5591==    by 0x47529C: slot_tp_str (typeobject.c:5312)
==5591==    by 0x452400: _PyObject_Str (object.c:425)
==5591==    by 0x4524B9: PyObject_Str (object.c:446)
==5591==    by 0x45F662: string_new (stringobject.c:4075)
==5591==  Address 0xd261020 is 112 bytes inside a block of size 464 free'd
==5591==    at 0x4C29159: realloc (vg_replace_malloc.c:525)
==5591==    by 0x440F9A: listappend (listobject.c:62)
==5591==    by 0x4A5A0D: PyEval_EvalFrameEx (ceval.c:3738)
==5591==    by 0x4A5AAF: PyEval_EvalFrameEx (ceval.c:3836)
==5591==    by 0x4A5AAF: PyEval_EvalFrameEx (ceval.c:3836)
==5591==    by 0x4A5AAF: PyEval_EvalFrameEx (ceval.c:3836)
==5591==    by 0x4A5AAF: PyEval_EvalFrameEx (ceval.c:3836)
==5591==    by 0x4A5AAF: PyEval_EvalFrameEx (ceval.c:3836)
==5591==    by 0x4A5AAF: PyEval_EvalFrameEx (ceval.c:3836)
==5591==    by 0x4A5AAF: PyEval_EvalFrameEx (ceval.c:3836)
==5591==    by 0x4A6BD0: PyEval_EvalCodeEx (ceval.c:3000)
==5591==    by 0x4A4D58: PyEval_EvalFrameEx (ceval.c:3846)
==5591== 
==5591== Invalid read of size 4
==5591==    at 0x4542F3: PyObject_Free (obmalloc.c:931)
==5591==    by 0x464788: PyString_Format (stringobject.c:5053)
==5591==    by 0x41D06E: binary_op1 (abstract.c:917)
==5591==    by 0x41FE2D: PyNumber_Remainder (abstract.c:969)
==5591==    by 0x4A2663: PyEval_EvalFrameEx (ceval.c:1180)
==5591==    by 0x4A6BD0: PyEval_EvalCodeEx (ceval.c:3000)
==5591==    by 0x4A6CA1: PyEval_EvalCode (ceval.c:541)
==5591==    by 0x4B952D: PyImport_ExecCodeModuleEx (import.c:677)
==5591==    by 0x4BB49D: load_source_module (import.c:1017)
==5591==    by 0x4BBCE2: load_package (import.c:1073)
==5591==    by 0x4BC2D2: import_submodule (import.c:2592)
==5591==    by 0x4BC593: load_next (import.c:2416)
==5591==  Address 0xc561020 is 2,768 bytes inside a block of size 2,832 free'd
==5591==    at 0x4C283A2: free (vg_replace_malloc.c:366)
==5591==    by 0x43F2CC: list_dealloc (listobject.c:308)
==5591==    by 0x440B2E: list_repr (listobject.c:423)
==5591==    by 0x452400: _PyObject_Str (object.c:425)
==5591==    by 0x4649CB: PyString_Format (stringobject.c:4850)
==5591==    by 0x41D06E: binary_op1 (abstract.c:917)
==5591==    by 0x41FE2D: PyNumber_Remainder (abstract.c:969)
==5591==    by 0x4A2663: PyEval_EvalFrameEx (ceval.c:1180)
==5591==    by 0x4A6BD0: PyEval_EvalCodeEx (ceval.c:3000)
==5591==    by 0x4A6CA1: PyEval_EvalCode (ceval.c:541)
==5591==    by 0x4B952D: PyImport_ExecCodeModuleEx (import.c:677)
==5591==    by 0x4BB49D: load_source_module (import.c:1017)
==5591== 
==5591== Invalid read of size 4
==5591==    at 0x4542F3: PyObject_Free (obmalloc.c:931)
==5591==    by 0x4A2EF8: PyEval_EvalFrameEx (ceval.c:3784)
==5591==    by 0x4A6BD0: PyEval_EvalCodeEx (ceval.c:3000)
==5591==    by 0x4A6CA1: PyEval_EvalCode (ceval.c:541)
==5591==    by 0x4B952D: PyImport_ExecCodeModuleEx (import.c:677)
==5591==    by 0x4BB49D: load_source_module (import.c:1017)
==5591==    by 0x4BBCE2: load_package (import.c:1073)
==5591==    by 0x4BC2D2: import_submodule (import.c:2592)
==5591==    by 0x4BC593: load_next (import.c:2416)
==5591==    by 0x4BCC10: import_module_level (import.c:2133)
==5591==    by 0x4BD133: PyImport_ImportModuleLevel (import.c:2185)
==5591==    by 0x49EBFA: builtin___import__ (bltinmodule.c:48)
==5591==  Address 0xc563020 is 2,336 bytes inside a block of size 5,777 free'd
==5591==    at 0x4C29159: realloc (vg_replace_malloc.c:525)
==5591==    by 0x45E346: _PyString_Resize (stringobject.c:4277)
==5591==    by 0x4651D5: PyString_Format (stringobject.c:5064)
==5591==    by 0x41D06E: binary_op1 (abstract.c:917)
==5591==    by 0x41FE2D: PyNumber_Remainder (abstract.c:969)
==5591==    by 0x4A2663: PyEval_EvalFrameEx (ceval.c:1180)
==5591==    by 0x4A6BD0: PyEval_EvalCodeEx (ceval.c:3000)
==5591==    by 0x4A6CA1: PyEval_EvalCode (ceval.c:541)
==5591==    by 0x4B952D: PyImport_ExecCodeModuleEx (import.c:677)
==5591==    by 0x4BB49D: load_source_module (import.c:1017)
==5591==    by 0x4BBCE2: load_package (import.c:1073)
==5591==    by 0x4BC2D2: import_submodule (import.c:2592)
==5591== 
==5591== Invalid read of size 4
==5591==    at 0x4542F3: PyObject_Free (obmalloc.c:931)
==5591==    by 0x4A0B80: PyEval_EvalFrameEx (ceval.c:1010)
==5591==    by 0x4A6BD0: PyEval_EvalCodeEx (ceval.c:3000)
==5591==    by 0x4A4D58: PyEval_EvalFrameEx (ceval.c:3846)
==5591==    by 0x4A6BD0: PyEval_EvalCodeEx (ceval.c:3000)
==5591==    by 0x4A4D58: PyEval_EvalFrameEx (ceval.c:3846)
==5591==    by 0x4A6BD0: PyEval_EvalCodeEx (ceval.c:3000)
==5591==    by 0x4A6CA1: PyEval_EvalCode (ceval.c:541)
==5591==    by 0x4B952D: PyImport_ExecCodeModuleEx (import.c:677)
==5591==    by 0x4BB49D: load_source_module (import.c:1017)
==5591==    by 0x4BC2D2: import_submodule (import.c:2592)
==5591==    by 0x4BC54E: load_next (import.c:2412)
==5591==  Address 0x6375020 is 336 bytes inside a block of size 984 free'd
==5591==    at 0x4C283A2: free (vg_replace_malloc.c:366)
==5591==    by 0x4184F5: parsetok (parsetok.c:260)
==5591==    by 0x4C7B39: PyRun_StringFlags (pythonrun.c:1445)
==5591==    by 0x4A3C18: PyEval_EvalFrameEx (ceval.c:4481)
==5591==    by 0x4A6BD0: PyEval_EvalCodeEx (ceval.c:3000)
==5591==    by 0x4A4D58: PyEval_EvalFrameEx (ceval.c:3846)
==5591==    by 0x4A6BD0: PyEval_EvalCodeEx (ceval.c:3000)
==5591==    by 0x4A6CA1: PyEval_EvalCode (ceval.c:541)
==5591==    by 0x4B952D: PyImport_ExecCodeModuleEx (import.c:677)
==5591==    by 0x4BB49D: load_source_module (import.c:1017)
==5591==    by 0x4BC2D2: import_submodule (import.c:2592)
==5591==    by 0x4BC54E: load_next (import.c:2412)
==5591== 
==5591== Invalid read of size 4
==5591==    at 0x4542F3: PyObject_Free (obmalloc.c:931)
==5591==    by 0x536CB2: meth_dealloc (methodobject.c:134)
==5591==    by 0x4A2EF8: PyEval_EvalFrameEx (ceval.c:3784)
==5591==    by 0x4A5AAF: PyEval_EvalFrameEx (ceval.c:3836)
==5591==    by 0x4A5AAF: PyEval_EvalFrameEx (ceval.c:3836)
==5591==    by 0x4A5AAF: PyEval_EvalFrameEx (ceval.c:3836)
==5591==    by 0x4A6BD0: PyEval_EvalCodeEx (ceval.c:3000)
==5591==    by 0x4A6CA1: PyEval_EvalCode (ceval.c:541)
==5591==    by 0x4B952D: PyImport_ExecCodeModuleEx (import.c:677)
==5591==    by 0x4BB49D: load_source_module (import.c:1017)
==5591==    by 0x4BC2D2: import_submodule (import.c:2592)
==5591==    by 0x4BC54E: load_next (import.c:2412)
==5591==  Address 0x6582020 is 2,400 bytes inside a block of size 4,144 free'd
==5591==    at 0x4C29159: realloc (vg_replace_malloc.c:525)
==5591==    by 0x516914: data_stack_grow (_sre.c:276)
==5591==    by 0x51BD60: sre_match (_sre.c:1294)
==5591==    by 0x51CBD5: sre_search (_sre.c:1584)
==5591==    by 0x51D5EB: pattern_search (_sre.c:1925)
==5591==    by 0x4A51AD: PyEval_EvalFrameEx (ceval.c:3750)
==5591==    by 0x4A6BD0: PyEval_EvalCodeEx (ceval.c:3000)
==5591==    by 0x4A4D58: PyEval_EvalFrameEx (ceval.c:3846)
==5591==    by 0x4A6BD0: PyEval_EvalCodeEx (ceval.c:3000)
==5591==    by 0x4A4D58: PyEval_EvalFrameEx (ceval.c:3846)
==5591==    by 0x4A6BD0: PyEval_EvalCodeEx (ceval.c:3000)
==5591==    by 0x4A4D58: PyEval_EvalFrameEx (ceval.c:3846)
==5591== 
==5591== Invalid read of size 4
==5591==    at 0x4548BE: PyObject_Realloc (obmalloc.c:1154)
==5591==    by 0x46450A: PyString_Format (stringobject.c:4277)
==5591==    by 0x41D06E: binary_op1 (abstract.c:917)
==5591==    by 0x41FE2D: PyNumber_Remainder (abstract.c:969)
==5591==    by 0x4A2663: PyEval_EvalFrameEx (ceval.c:1180)
==5591==    by 0x4A6BD0: PyEval_EvalCodeEx (ceval.c:3000)
==5591==    by 0x535B4F: function_call (funcobject.c:524)
==5591==    by 0x41C9D6: PyObject_Call (abstract.c:2492)
==5591==    by 0x42570E: instancemethod_call (classobject.c:2579)
==5591==    by 0x41C9D6: PyObject_Call (abstract.c:2492)
==5591==    by 0x47603E: slot_tp_call (typeobject.c:5372)
==5591==    by 0x41C9D6: PyObject_Call (abstract.c:2492)
==5591==  Address 0x65af020 is 1,584 bytes inside a block of size 2,464 free'd
==5591==    at 0x4C29159: realloc (vg_replace_malloc.c:525)
==5591==    by 0x516914: data_stack_grow (_sre.c:276)
==5591==    by 0x51BD60: sre_match (_sre.c:1294)
==5591==    by 0x51CBD5: sre_search (_sre.c:1584)
==5591==    by 0x51D5EB: pattern_search (_sre.c:1925)
==5591==    by 0x4A51AD: PyEval_EvalFrameEx (ceval.c:3750)
==5591==    by 0x4A6BD0: PyEval_EvalCodeEx (ceval.c:3000)
==5591==    by 0x4A4D58: PyEval_EvalFrameEx (ceval.c:3846)
==5591==    by 0x4A6BD0: PyEval_EvalCodeEx (ceval.c:3000)
==5591==    by 0x4A4D58: PyEval_EvalFrameEx (ceval.c:3846)
==5591==    by 0x4A6BD0: PyEval_EvalCodeEx (ceval.c:3000)
==5591==    by 0x4A4D58: PyEval_EvalFrameEx (ceval.c:3846)
==5591== 
==5591== Invalid read of size 4
==5591==    at 0x4542F3: PyObject_Free (obmalloc.c:931)
==5591==    by 0x54B9EF: PyMember_SetOne (structmember.c:320)
==5591==    by 0x452B7E: PyObject_GenericSetAttr (object.c:1485)
==5591==    by 0x4518BE: PyObject_SetAttr (object.c:1252)
==5591==    by 0x4A1DCB: PyEval_EvalFrameEx (ceval.c:1864)
==5591==    by 0x4A6BD0: PyEval_EvalCodeEx (ceval.c:3000)
==5591==    by 0x535B4F: function_call (funcobject.c:524)
==5591==    by 0x41C9D6: PyObject_Call (abstract.c:2492)
==5591==    by 0x42570E: instancemethod_call (classobject.c:2579)
==5591==    by 0x41C9D6: PyObject_Call (abstract.c:2492)
==5591==    by 0x47603E: slot_tp_call (typeobject.c:5372)
==5591==    by 0x41C9D6: PyObject_Call (abstract.c:2492)
==5591==  Address 0x6523020 is 6,080 bytes inside a block of size 6,224 free'd
==5591==    at 0x4C29159: realloc (vg_replace_malloc.c:525)
==5591==    by 0x516914: data_stack_grow (_sre.c:276)
==5591==    by 0x51C17D: sre_match (_sre.c:1003)
==5591==    by 0x51CBD5: sre_search (_sre.c:1584)
==5591==    by 0x51D5EB: pattern_search (_sre.c:1925)
==5591==    by 0x4A51AD: PyEval_EvalFrameEx (ceval.c:3750)
==5591==    by 0x4A6BD0: PyEval_EvalCodeEx (ceval.c:3000)
==5591==    by 0x4A4D58: PyEval_EvalFrameEx (ceval.c:3846)
==5591==    by 0x4A6BD0: PyEval_EvalCodeEx (ceval.c:3000)
==5591==    by 0x4A4D58: PyEval_EvalFrameEx (ceval.c:3846)
==5591==    by 0x4A6BD0: PyEval_EvalCodeEx (ceval.c:3000)
==5591==    by 0x4A4D58: PyEval_EvalFrameEx (ceval.c:3846)
==5591== 
==5591== Invalid read of size 4
==5591==    at 0x4542F3: PyObject_Free (obmalloc.c:931)
==5591==    by 0x4A181D: PyEval_EvalFrameEx (ceval.c:1344)
==5591==    by 0x4A6BD0: PyEval_EvalCodeEx (ceval.c:3000)
==5591==    by 0x4A4D58: PyEval_EvalFrameEx (ceval.c:3846)
==5591==    by 0x4A6BD0: PyEval_EvalCodeEx (ceval.c:3000)
==5591==    by 0x4A6CA1: PyEval_EvalCode (ceval.c:541)
==5591==    by 0x4B952D: PyImport_ExecCodeModuleEx (import.c:677)
==5591==    by 0x4BB49D: load_source_module (import.c:1017)
==5591==    by 0x4BC2D2: import_submodule (import.c:2592)
==5591==    by 0x4BC54E: load_next (import.c:2412)
==5591==    by 0x4BCC57: import_module_level (import.c:2141)
==5591==    by 0x4BD133: PyImport_ImportModuleLevel (import.c:2185)
==5591==  Address 0x6781020 is 928 bytes inside a block of size 1,104 free'd
==5591==    at 0x4C283A2: free (vg_replace_malloc.c:366)
==5591==    by 0x51D8C9: pattern_subx (_sre.c:260)
==5591==    by 0x51E0CC: pattern_sub (_sre.c:2474)
==5591==    by 0x4A51AD: PyEval_EvalFrameEx (ceval.c:3750)
==5591==    by 0x4A5AAF: PyEval_EvalFrameEx (ceval.c:3836)
==5591==    by 0x4A6BD0: PyEval_EvalCodeEx (ceval.c:3000)
==5591==    by 0x4A4D58: PyEval_EvalFrameEx (ceval.c:3846)
==5591==    by 0x4A6BD0: PyEval_EvalCodeEx (ceval.c:3000)
==5591==    by 0x4A6CA1: PyEval_EvalCode (ceval.c:541)
==5591==    by 0x4B952D: PyImport_ExecCodeModuleEx (import.c:677)
==5591==    by 0x4BB49D: load_source_module (import.c:1017)
==5591==    by 0x4BC2D2: import_submodule (import.c:2592)
==5591== 
==5591== Conditional jump or move depends on uninitialised value(s)
==5591==    at 0x4548C7: PyObject_Realloc (obmalloc.c:1154)
==5591==    by 0x46450A: PyString_Format (stringobject.c:4277)
==5591==    by 0x41D06E: binary_op1 (abstract.c:917)
==5591==    by 0x41FE2D: PyNumber_Remainder (abstract.c:969)
==5591==    by 0x4A2663: PyEval_EvalFrameEx (ceval.c:1180)
==5591==    by 0x4A6BD0: PyEval_EvalCodeEx (ceval.c:3000)
==5591==    by 0x535B4F: function_call (funcobject.c:524)
==5591==    by 0x41C9D6: PyObject_Call (abstract.c:2492)
==5591==    by 0x42570E: instancemethod_call (classobject.c:2579)
==5591==    by 0x41C9D6: PyObject_Call (abstract.c:2492)
==5591==    by 0x47603E: slot_tp_call (typeobject.c:5372)
==5591==    by 0x41C9D6: PyObject_Call (abstract.c:2492)
==5591== 
==5591== Use of uninitialised value of size 8
==5591==    at 0x4548DA: PyObject_Realloc (obmalloc.c:1154)
==5591==    by 0x46450A: PyString_Format (stringobject.c:4277)
==5591==    by 0x41D06E: binary_op1 (abstract.c:917)
==5591==    by 0x41FE2D: PyNumber_Remainder (abstract.c:969)
==5591==    by 0x4A2663: PyEval_EvalFrameEx (ceval.c:1180)
==5591==    by 0x4A6BD0: PyEval_EvalCodeEx (ceval.c:3000)
==5591==    by 0x535B4F: function_call (funcobject.c:524)
==5591==    by 0x41C9D6: PyObject_Call (abstract.c:2492)
==5591==    by 0x42570E: instancemethod_call (classobject.c:2579)
==5591==    by 0x41C9D6: PyObject_Call (abstract.c:2492)
==5591==    by 0x47603E: slot_tp_call (typeobject.c:5372)
==5591==    by 0x41C9D6: PyObject_Call (abstract.c:2492)
==5591== 
==5591== Invalid read of size 4
==5591==    at 0x4542F3: PyObject_Free (obmalloc.c:931)
==5591==    by 0x535E2A: func_dealloc (funcobject.c:459)
==5591==    by 0x44BEE6: insertdict (dictobject.c:459)
==5591==    by 0x44E3F6: PyDict_SetItem (dictobject.c:701)
==5591==    by 0x4502FD: _PyModule_Clear (moduleobject.c:138)
==5591==    by 0x4B8C23: PyImport_Cleanup (import.c:476)
==5591==    by 0x4C684E: Py_Finalize (pythonrun.c:444)
==5591==    by 0x417B45: Py_Main (main.c:602)
==5591==    by 0x5EEAD8D: (below main) (libc-start.c:226)
==5591==  Address 0x6330020 is 624 bytes inside a block of size 704 free'd
==5591==    at 0x4C29159: realloc (vg_replace_malloc.c:525)
==5591==    by 0x440C20: PyList_Append (listobject.c:62)
==5591==    by 0x4A0C9F: PyEval_EvalFrameEx (ceval.c:1321)
==5591==    by 0x4A6BD0: PyEval_EvalCodeEx (ceval.c:3000)
==5591==    by 0x4A4D58: PyEval_EvalFrameEx (ceval.c:3846)
==5591==    by 0x4A6BD0: PyEval_EvalCodeEx (ceval.c:3000)
==5591==    by 0x535B4F: function_call (funcobject.c:524)
==5591==    by 0x41C9D6: PyObject_Call (abstract.c:2492)
==5591==    by 0x42570E: instancemethod_call (classobject.c:2579)
==5591==    by 0x41C9D6: PyObject_Call (abstract.c:2492)
==5591==    by 0x47603E: slot_tp_call (typeobject.c:5372)
==5591==    by 0x41C9D6: PyObject_Call (abstract.c:2492)
==5591== 
==5591== Conditional jump or move depends on uninitialised value(s)
==5591==    at 0x4542FC: PyObject_Free (obmalloc.c:931)
==5591==    by 0x535E2A: func_dealloc (funcobject.c:459)
==5591==    by 0x44BEE6: insertdict (dictobject.c:459)
==5591==    by 0x44E3F6: PyDict_SetItem (dictobject.c:701)
==5591==    by 0x4502FD: _PyModule_Clear (moduleobject.c:138)
==5591==    by 0x4B8DF6: PyImport_Cleanup (import.c:494)
==5591==    by 0x4C684E: Py_Finalize (pythonrun.c:444)
==5591==    by 0x417B45: Py_Main (main.c:602)
==5591==    by 0x5EEAD8D: (below main) (libc-start.c:226)
==5591== 
==5591== Use of uninitialised value of size 8
==5591==    at 0x454316: PyObject_Free (obmalloc.c:931)
==5591==    by 0x535E2A: func_dealloc (funcobject.c:459)
==5591==    by 0x44BEE6: insertdict (dictobject.c:459)
==5591==    by 0x44E3F6: PyDict_SetItem (dictobject.c:701)
==5591==    by 0x4502FD: _PyModule_Clear (moduleobject.c:138)
==5591==    by 0x4B8DF6: PyImport_Cleanup (import.c:494)
==5591==    by 0x4C684E: Py_Finalize (pythonrun.c:444)
==5591==    by 0x417B45: Py_Main (main.c:602)
==5591==    by 0x5EEAD8D: (below main) (libc-start.c:226)
==5591== 
==5591== Invalid read of size 4
==5591==    at 0x4542F3: PyObject_Free (obmalloc.c:931)
==5591==    by 0x47E924: unicode_dealloc (unicodeobject.c:368)
==5591==    by 0x44BEE6: insertdict (dictobject.c:459)
==5591==    by 0x44E3F6: PyDict_SetItem (dictobject.c:701)
==5591==    by 0x4502FD: _PyModule_Clear (moduleobject.c:138)
==5591==    by 0x4B8DF6: PyImport_Cleanup (import.c:494)
==5591==    by 0x4C684E: Py_Finalize (pythonrun.c:444)
==5591==    by 0x417B45: Py_Main (main.c:602)
==5591==    by 0x5EEAD8D: (below main) (libc-start.c:226)
==5591==  Address 0xc38b020 is not stack'd, malloc'd or (recently) free'd
==5591== 
==5591== Invalid read of size 4
==5591==    at 0x4542F3: PyObject_Free (obmalloc.c:931)
==5591==    by 0x46ABF7: subtype_dealloc (typeobject.c:1019)
==5591==    by 0x44D5D2: dict_dealloc (dictobject.c:911)
==5591==    by 0x427DD3: instance_dealloc (classobject.c:668)
==5591==    by 0x44BEE6: insertdict (dictobject.c:459)
==5591==    by 0x44E3F6: PyDict_SetItem (dictobject.c:701)
==5591==    by 0x450263: _PyModule_Clear (moduleobject.c:125)
==5591==    by 0x4B8DF6: PyImport_Cleanup (import.c:494)
==5591==    by 0x4C684E: Py_Finalize (pythonrun.c:444)
==5591==    by 0x417B45: Py_Main (main.c:602)
==5591==    by 0x5EEAD8D: (below main) (libc-start.c:226)
==5591==  Address 0xd3a0020 is 336 bytes inside a block of size 456 free'd
==5591==    at 0x4C283A2: free (vg_replace_malloc.c:366)
==5591==    by 0x52D625: code_dealloc (codeobject.c:270)
==5591==    by 0x535E12: func_dealloc (funcobject.c:454)
==5591==    by 0x44BEE6: insertdict (dictobject.c:459)
==5591==    by 0x44E3F6: PyDict_SetItem (dictobject.c:701)
==5591==    by 0x450263: _PyModule_Clear (moduleobject.c:125)
==5591==    by 0x4B8DF6: PyImport_Cleanup (import.c:494)
==5591==    by 0x4C684E: Py_Finalize (pythonrun.c:444)
==5591==    by 0x417B45: Py_Main (main.c:602)
==5591==    by 0x5EEAD8D: (below main) (libc-start.c:226)
==5591== 
==5591== Invalid read of size 4
==5591==    at 0x4542F3: PyObject_Free (obmalloc.c:931)
==5591==    by 0x516A0A: pattern_dealloc (_sre.c:1859)
==5591==    by 0x44D5D2: dict_dealloc (dictobject.c:911)
==5591==    by 0x44BEE6: insertdict (dictobject.c:459)
==5591==    by 0x44E3F6: PyDict_SetItem (dictobject.c:701)
==5591==    by 0x450263: _PyModule_Clear (moduleobject.c:125)
==5591==    by 0x4B8DF6: PyImport_Cleanup (import.c:494)
==5591==    by 0x4C684E: Py_Finalize (pythonrun.c:444)
==5591==    by 0x417B45: Py_Main (main.c:602)
==5591==    by 0x5EEAD8D: (below main) (libc-start.c:226)
==5591==  Address 0xc38f020 is 1,728 bytes inside a block of size 3,072 free'd
==5591==    at 0x4C283A2: free (vg_replace_malloc.c:366)
==5591==    by 0x44D5EE: dict_dealloc (dictobject.c:915)
==5591==    by 0x44D5D2: dict_dealloc (dictobject.c:911)
==5591==    by 0x46AC46: subtype_dealloc (typeobject.c:1004)
==5591==    by 0x43F2C1: list_dealloc (listobject.c:306)
==5591==    by 0x44D5D2: dict_dealloc (dictobject.c:911)
==5591==    by 0x46AC46: subtype_dealloc (typeobject.c:1004)
==5591==    by 0x43F2C1: list_dealloc (listobject.c:306)
==5591==    by 0x44D5D2: dict_dealloc (dictobject.c:911)
==5591==    by 0x46AC46: subtype_dealloc (typeobject.c:1004)
==5591==    by 0x43F2C1: list_dealloc (listobject.c:306)
==5591==    by 0x44D5D2: dict_dealloc (dictobject.c:911)
==5591== 
==5591== Conditional jump or move depends on uninitialised value(s)
==5591==    at 0x4542FC: PyObject_Free (obmalloc.c:931)
==5591==    by 0x44D5D2: dict_dealloc (dictobject.c:911)
==5591==    by 0x427DD3: instance_dealloc (classobject.c:668)
==5591==    by 0x44BEE6: insertdict (dictobject.c:459)
==5591==    by 0x44E3F6: PyDict_SetItem (dictobject.c:701)
==5591==    by 0x4502FD: _PyModule_Clear (moduleobject.c:138)
==5591==    by 0x4B8DF6: PyImport_Cleanup (import.c:494)
==5591==    by 0x4C684E: Py_Finalize (pythonrun.c:444)
==5591==    by 0x417B45: Py_Main (main.c:602)
==5591==    by 0x5EEAD8D: (below main) (libc-start.c:226)
==5591== 
==5591== Invalid read of size 4
==5591==    at 0x4542F3: PyObject_Free (obmalloc.c:931)
==5591==    by 0x44C287: PyDict_Fini (dictobject.c:218)
==5591==    by 0x4C68AE: Py_Finalize (pythonrun.c:516)
==5591==    by 0x417B45: Py_Main (main.c:602)
==5591==    by 0x5EEAD8D: (below main) (libc-start.c:226)
==5591==  Address 0xc5e6020 is not stack'd, malloc'd or (recently) free'd
==5591== 
==5591== Invalid read of size 4
==5591==    at 0x4542F3: PyObject_Free (obmalloc.c:931)
==5591==    by 0x522689: PyGrammar_RemoveAccelerators (acceler.c:47)
==5591==    by 0x4C68BE: Py_Finalize (pythonrun.c:529)
==5591==    by 0x417B45: Py_Main (main.c:602)
==5591==    by 0x5EEAD8D: (below main) (libc-start.c:226)
==5591==  Address 0x6425020 is 352 bytes inside a block of size 704 free'd
==5591==    at 0x4C29159: realloc (vg_replace_malloc.c:525)
==5591==    by 0x440C20: PyList_Append (listobject.c:62)
==5591==    by 0x4A0C9F: PyEval_EvalFrameEx (ceval.c:1321)
==5591==    by 0x4A6BD0: PyEval_EvalCodeEx (ceval.c:3000)
==5591==    by 0x4A4D58: PyEval_EvalFrameEx (ceval.c:3846)
==5591==    by 0x4A6BD0: PyEval_EvalCodeEx (ceval.c:3000)
==5591==    by 0x535B4F: function_call (funcobject.c:524)
==5591==    by 0x41C9D6: PyObject_Call (abstract.c:2492)
==5591==    by 0x42570E: instancemethod_call (classobject.c:2579)
==5591==    by 0x41C9D6: PyObject_Call (abstract.c:2492)
==5591==    by 0x47603E: slot_tp_call (typeobject.c:5372)
==5591==    by 0x41C9D6: PyObject_Call (abstract.c:2492)
==5591== 
==5591== 
==5591== HEAP SUMMARY:
==5591==     in use at exit: 18,359,795 bytes in 11,801 blocks
==5591==   total heap usage: 156,956 allocs, 145,155 frees, 91,162,866 bytes allocated
==5591== 
==5591== LEAK SUMMARY:
==5591==    definitely lost: 192 bytes in 6 blocks
==5591==    indirectly lost: 0 bytes in 0 blocks
==5591==      possibly lost: 272,826 bytes in 502 blocks
==5591==    still reachable: 18,086,777 bytes in 11,293 blocks
==5591==         suppressed: 0 bytes in 0 blocks
==5591== Rerun with --leak-check=full to see details of leaked memory
==5591== 
==5591== For counts of detected and suppressed errors, rerun with: -v
==5591== Use --track-origins=yes to see where uninitialised values come from
==5591== ERROR SUMMARY: 9735 errors from 141 contexts (suppressed: 304 from 7)
